
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            TNO1, a TGN-localized SNARE-interacting protein, modulates root skewing in Arabidopsis thaliana - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="8E5918E28E15DE131418E2003DD965B1.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcps">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC5387210/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Plant Biology">
<meta name="citation_title" content="TNO1, a TGN-localized SNARE-interacting protein, modulates root skewing in Arabidopsis thaliana">
<meta name="citation_author" content="Rahul Roy">
<meta name="citation_author_institution" content="Department of Genetics, Development and Cell Biology, 1035B Roy J Carver Co-Lab, 1111 WOI Rd, Iowa State University, Ames, IA 50011 USA">
<meta name="citation_author_institution" content="Interdepartmental Genetics Program, Iowa State University, Ames, IA USA">
<meta name="citation_author_institution" content="Current Address: Department of Plant and Microbial Biology, University of Minnesota, Twin Cities, MN 55108 USA">
<meta name="citation_author" content="Diane C Bassham">
<meta name="citation_author_institution" content="Department of Genetics, Development and Cell Biology, 1035B Roy J Carver Co-Lab, 1111 WOI Rd, Iowa State University, Ames, IA 50011 USA">
<meta name="citation_author_institution" content="Interdepartmental Genetics Program, Iowa State University, Ames, IA USA">
<meta name="citation_author_institution" content="Plant Sciences Institute, Iowa State University, Ames, IA USA">
<meta name="citation_publication_date" content="2017 Apr 11">
<meta name="citation_volume" content="17">
<meta name="citation_firstpage" content="73">
<meta name="citation_doi" content="10.1186/s12870-017-1024-4">
<meta name="citation_pmid" content="28399805">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC5387210/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC5387210/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC5387210/pdf/12870_2017_Article_1024.pdf">
<meta name="description" content="The movement of plant roots within the soil is key to their ability to interact with the environment and maximize anchorage and nutrient acquisition. Directional growth of roots occurs by a combination of sensing external cues, hormonal signaling ...">
<meta name="og:title" content="TNO1, a TGN-localized SNARE-interacting protein, modulates root skewing in Arabidopsis thaliana">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The movement of plant roots within the soil is key to their ability to interact with the environment and maximize anchorage and nutrient acquisition. Directional growth of roots occurs by a combination of sensing external cues, hormonal signaling ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC5387210/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://pmc.ncbi.nlm.nih.gov/search/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="5387210">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12870-017-1024-4"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12870_2017_Article_1024.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC5387210%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/5387210/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/5387210/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC5387210/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcps.png" alt="BMC Plant Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Plant Biology" title="Link to BMC Plant Biology" shape="default" href="https://bmcplantbiol.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Plant Biol</button></div>. 2017 Apr 11;17:73. doi: <a href="https://doi.org/10.1186/s12870-017-1024-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12870-017-1024-4</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://pmc.ncbi.nlm.nih.gov/search/?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Plant%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>TNO1, a TGN-localized SNARE-interacting protein, modulates root skewing in <em>Arabidopsis thaliana</em>
</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roy%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Rahul Roy</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Rahul Roy</span></h3>
<div class="p">
<sup>1</sup>Department of Genetics, Development and Cell Biology, 1035B Roy J Carver Co-Lab, 1111 WOI Rd, Iowa State University, Ames, IA 50011 USA </div>
<div class="p">
<sup>2</sup>Interdepartmental Genetics Program, Iowa State University, Ames, IA USA </div>
<div class="p">
<sup>4</sup>Current Address: Department of Plant and Microbial Biology, University of Minnesota, Twin Cities, MN 55108 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roy%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rahul Roy</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bassham%20DC%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Diane C Bassham</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Diane C Bassham</span></h3>
<div class="p">
<sup>1</sup>Department of Genetics, Development and Cell Biology, 1035B Roy J Carver Co-Lab, 1111 WOI Rd, Iowa State University, Ames, IA 50011 USA </div>
<div class="p">
<sup>2</sup>Interdepartmental Genetics Program, Iowa State University, Ames, IA USA </div>
<div class="p">
<sup>3</sup>Plant Sciences Institute, Iowa State University, Ames, IA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bassham%20DC%22%5BAuthor%5D" class="usa-link"><span class="name western">Diane C Bassham</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Genetics, Development and Cell Biology, 1035B Roy J Carver Co-Lab, 1111 WOI Rd, Iowa State University, Ames, IA 50011 USA </div>
<div id="Aff2">
<sup>2</sup>Interdepartmental Genetics Program, Iowa State University, Ames, IA USA </div>
<div id="Aff3">
<sup>3</sup>Plant Sciences Institute, Iowa State University, Ames, IA USA </div>
<div id="Aff4">
<sup>4</sup>Current Address: Department of Plant and Microbial Biology, University of Minnesota, Twin Cities, MN 55108 USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2016 Dec 14; Accepted 2017 Apr 5; Collection date 2017.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s). 2017</div>
<p>
<strong>Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<a href="http://creativecommons.org/publicdomain/zero/1.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC5387210  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/28399805/" class="usa-link">28399805</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p>The movement of plant roots within the soil is key to their ability to interact with the environment and maximize anchorage and nutrient acquisition. Directional growth of roots occurs by a combination of sensing external cues, hormonal signaling and cytoskeletal changes in the root cells. Roots growing on slanted, impenetrable growth medium display a characteristic waving and skewing, and mutants with deviations in these phenotypes assist in identifying genes required for root movement. Our study identifies a role for a <em>trans</em>-Golgi network-localized protein in root skewing.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p>We found that <em>Arabidopsis thaliana</em> TNO1 (TGN-localized SYP41-interacting protein), a putative tethering factor localized at the <em>trans</em>-Golgi network, affects root skewing. <em>tno1</em> knockout mutants display enhanced root skewing and epidermal cell file rotation. Skewing of <em>tno1</em> roots increases upon microtubule stabilization, but is insensitive to microtubule destabilization. Microtubule destabilization leads to severe defects in cell morphology in <em>tno1</em> seedlings. Microtubule array orientation is unaffected in the mutant roots, suggesting that the increase in cell file rotation is independent of the orientation of microtubule arrays.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p>We conclude that TNO1 modulates root skewing in a mechanism that is dependent on microtubules but is not linked to disruption of the orientation of microtubule arrays. In addition, TNO1 is required for maintenance of cell morphology in mature regions of roots and the base of hypocotyls. The TGN-localized SNARE machinery might therefore be important for appropriate epidermal cell file rotation and cell expansion during root growth.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> <em>t</em>rans-Golgi network, Root movement, Cell file rotation, Tethering factor, Microtubules</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p>Establishment of the root system of a plant is crucial for efficient anchorage and nutrient acquisition. Root development and architecture are well-studied [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>] but our understanding of how roots interact with soil components is still incomplete. This is due in part to the difficulty of visualizing roots as they move into the soil, although recent developments such as the GLO-Roots platform aid in imaging this process [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Root growth into the substratum presents multiple cues to the root tip such as mechanical obstacles, moisture and nutrient gradients. These cues are integrated and then signal downstream processes involving hormonal pathways and cell expansion [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. This leads to a cumulative physiological response driving root movement and establishing the root architecture.</p>
<p>Studying and simulating root movement and directional growth in the laboratory on synthetic growth media has revealed distinct movement types, and some of the genes and pathways that control them [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Roots of <em>Arabidopsis thaliana</em> seedlings display various growth behaviors depending on external conditions. When embedded in a homogeneous medium (penetrable agar), roots grow downward in response to gravity and show minimal deviation from the gravity vector. By contrast, when roots are subjected to multiple directional cues, complicated growth patterns occur [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. For example, <em>Arabidopsis</em> roots grown on a slanted impenetrable medium (1.5% agar) show a characteristic deviation from the vertical (skewing) with a periodic wave-like pattern along their trajectory (waving). This arises due to a combination of touch [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>], gravitropism [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>], circumnutation [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>], and physical interaction between the root tip and the growth medium [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>].</p>
<p>Skewing and waving roots display a characteristic twisting of epidermal cell files along the root, referred to as cell file rotation (CFR). When roots of <em>Arabidopsis</em> seedlings skew and wave, the succession of sinusoidal waves alternate between left-handed and right-handed CFRs, which correlates with their rightward and leftward movement respectively [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. According to Rutherford and Masson (1996), skewing is described as rightward or leftward when viewed from the back of the plate [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>] while handedness of the CFR is defined as left-handed or right-handed when viewing the axis of the root pointing shootwards. Roots of <em>Arabidopsis</em> grown on a slanted, hard medium exhibit a dominant left-handed rotation around the growth axis resulting in a predominantly counterclockwise/left-handed epidermal CFR. This usually corresponds to a rightward skewing when seen from the back of the plate [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. CFR also typically correlates with an oblique microtubule (MT) array orientation in the twisting cell files, although exceptions do exist [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>].</p>
<p>Multiple factors such as external cues (e.g. moisture, light or gravity), hormonal pathways, and cytoskeletal and cell wall dynamics influence the direction of root growth [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. External cues signal changes in hormone signaling pathways, including auxin [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>], ethylene [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>], cytokinin [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] and brassinosteroid pathways [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. Downstream of hormonal and environmental perception, changes in the cytoskeleton and in cell wall deposition patterns modulate cell division and cell expansion dynamics, thus mediating root movements. Defects in tubulin structure or activity or in MT-associated proteins result in altered MT dynamics and array orientation. This then leads to changes in cell expansion and affects CFR and skewing [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. Cell wall properties and the trafficking of cell wall components to the plasma membrane are key to root elongation and movement as inferred from the altered root movements observed in mutants defective in cellulose deposition, wall-anchored proteins or crosslinking of cell wall components [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>–<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>].</p>
<p>Sorting of cell wall components occurs at the <em>trans</em>-Golgi network (TGN), a tubulo-vesicular organelle that matures from the two or three <em>trans</em>- most cisternae of the Golgi [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. It acts as an early endosome, receiving endocytosed cargo as well as directing vacuolar and secretory traffic, with distinct subdomains that function in various trafficking routes [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>–<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Specific membrane fusion factors called SNAREs (soluble <em>N</em>-ethylmaleimide–sensitive factor attachment protein receptors) maintain vesicle trafficking fidelity and cargo sorting within cells [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. The TGN-localized SYP4 (41/42/43) SNARE family is required for multiple transport pathways that in turn regulate auxin homeostasis and disease resistance [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. SYP61, another TGN-localized SNARE that interacts with SYP41, functions in vacuolar trafficking and secretion of cell wall enzymes such as cellulose synthase and pectin-polysaccharide-modifying protein. This indicates a role for SYP61 in trafficking of cell wall components and enzymes for their synthesis and modification [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>–<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. TNO1 (TGN-localized SYP41-interacting protein) is a TGN-localized coiled-coil protein that interacts with the SYP41 SNARE machinery. Mutant plants lacking TNO1 (<em>tno1</em>) partially mis-sort vacuolar cargo, mis-localize SYP61 and display altered TGN dynamics and slower gravitropic responses [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>, <a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>].</p>
<p>Trafficking at the TGN plays a role in auxin responses, trafficking of cell wall components, and cell expansion, all of which are important for root growth and movement. In addition, gravitropic bending is delayed in <em>tno1</em> roots, suggesting defects in directional growth of roots. We therefore investigated a potential function for TNO1 in root movement. We report here that TNO1 acts as a negative regulator of root skewing, since <em>tno1</em> mutant roots have enhanced skewing which also correlates with an enhanced CFR.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Plant material and growth conditions</h3>
<p>
<em>Arabidopsis thaliana</em> Col-0 (wild-type, WT) and <em>tno1</em> knockout mutant (SALK_112503) were obtained from the Arabidopsis Biological Resource Center; the complemented <em>tno1</em> mutant was generated previously in our laboratory [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. <em>Arabidopsis</em> seeds were surface-sterilized in 33% bleach, 0.1% (<em>v</em>/v) Triton X-100 for 20 min, rinsed 5 times with sterile water and kept in the dark at 4 °C for at least 2 days before being subjected to the root skewing and waving assays or drug sensitivity assays described below. <em>Arabidopsis</em> hypocotyls were analyzed by plating sterilized seeds on 0.5X solid Murashige-Skoog (MS) medium [(Murashige-Skoog vitamin and salt mixture, Caisson, MSPA0910] with 1% sucrose, 2.4 mM MES (pH 5.7), and 0.8% (<em>w</em>/<em>v</em>) Phytoblend agar (Caisson, PTP01) and growing vertically at 22 °C in the dark.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Root skewing and waving assays</h3>
<p>Skewing and waving assays were performed as described [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. <em>Arabidopsis</em> plants were grown at 22 °C in long-day conditions (16 h light) on 0.5X solid Murashige-Skoog (MS) medium (Murashige-Skoog vitamin and salt mixture, Caisson, MSPA0910) with 1% sucrose, 2.4 mM MES (pH 5.7), and 1.5% (<em>w</em>/<em>v</em>) Phytoblend agar (Caisson, PTP01). Seedlings were grown vertically for 3 days after which the root tip position was marked. The plates were then slanted backwards, 30<sup>o</sup> to the vertical, and grown for another 4 days. Images of the roots were acquired from the back of the plate using a Nikon SMZ1000 light microscope equipped with a Nikon S10 CoolPix camera. Analysis of root parameters was performed using Image J [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. Root tip deviations to the right from the back of the plate were considered positive while deviations to the left were considered negative.</p>
<p>Root morphometric analyses were performed as described [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Images of skewing/waving roots were analyzed with the ImageJ software [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>] for measuring root tip abscissa (Lx) and length of the root (L) followed by calculation of horizontal growth indices (Lx/L).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Drug sensitivity assays</h3>
<p>Seedlings were grown vertically on plates containing either taxol (Paclitaxel, Sigma Cat #T7191) or propyzamide (Sigma-Aldrich, Cat #45645) at the indicated concentrations for 5 days and skewing angles of roots were determined. Dark grown hypocotyl cell phenotypes were analyzed by placing 7-day-old hypocotyls on the surface of warm 3% low melting point agarose on a slide [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>] and then imaging the imprints with a Zeiss Axioplan II light microscope.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Cell file rotation analysis and propidium iodide staining</h3>
<p>The root surface of <em>Arabidopsis</em> seedlings was visualized using a Zeiss macro-zoom microscope at the Microscopy and NanoImaging facility, Iowa State University. Alternatively, roots were stained with propidium iodide (ThermoFisher Scientific Cat #P3566). Seedlings were dipped in a working aqueous solution (10 μg/ml) of propidium iodide for 1 min and then washed twice by dipping in petri dishes filled with water for 30 s. The roots were mounted in water and visualized using a Leica confocal laser scanning microscope (Leica SP5; Leica Microsystems) at the Iowa State University Confocal and Multiphoton Facility. Excitation and emission wavelengths were 488 nm and 617 nm respectively. Laser power, scan frequency and line averaging were optimized and kept constant between samples and replicates. CFR angles of the roots were calculated as described [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>] using ImageJ [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>].</p></section><section id="Sec7"><h3 class="pmc_sec_title">Immunostaining and analysis of microtubules</h3>
<p>Five-day-old <em>Arabidopsis</em> seedlings grown on slanted medium were fixed and immunostained as described [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. Mouse anti-α<strong>-</strong> tubulin antibodies (Sigma-Aldrich, Cat #T6074), diluted 1:100, were used for immunolabeling of MTs followed by AlexaFluor 488-conjugated goat anti-mouse IgG (ThermoFisher Scientific, Cat #A11029) as a secondary antibody, diluted 1:500. Cells within the elongation zone were imaged by confocal microscopy with a Leica confocal laser scanning microscope (Leica SP5; Leica Microsystems) at the Iowa State University Confocal and Multiphoton Facility, using a 40X (1.25 NA) or 63X (1.4 NA) Leica oil immersion objective. Excitation and emission wavelengths were 488 nm and 507 nm. Laser power, scan frequency and line averaging were optimized and kept constant between samples and replicates. Confocal images were analyzed with the freely available software package MicroFilament Analyzer (MFA) [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>], and data was displayed as circular plots as generated by the software. MT angles generated by the software were classified into three classes, 0 to 10, 11 to 30 and 31 to 90 degrees using Microsoft Excel and used for subsequent analysis.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec9"><h3 class="pmc_sec_title">Loss of TNO1 results in exaggerated root skewing</h3>
<p>Mutants lacking TNO1 have normal root elongation, but show a lag in root gravitropic bending upon gravistimulation [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. Positive gravitropism is a key component driving root waving and skewing, and hence we hypothesized that TNO1 might play a role in these processes.</p>
<p>To test this hypothesis, 3-day-old seedlings were grown vertically on the surface of hard agar medium (1.5% agar), the plates tipped over by 30<sup>o</sup>, and the seedlings allowed to grow for an additional 4 days (Fig. <a href="#Fig1" class="usa-link">1a</a>). The slant forces the root tip to press against the impenetrable medium due to positive gravitropism. As the root fails to penetrate the medium, cell expansion and circumnutation drive the root to the side and this alternates between rightward and leftward corrective deviations resulting in characteristic waving and skewing responses [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>, <a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. <em>tno1</em> mutant roots (KO) formed sinusoidal wave-like growth patterns similar to WT (Col-0) roots but had an exaggerated rightward skew compared to the WT roots, when seen from the back of the plate (Fig. <a href="#Fig1" class="usa-link">1b</a>). <em>tno1</em> mutants complemented with transgenic <em>TNO1</em> under the control of its own promoter (COM) [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] had reduced skewing compared to the mutants and were similar to the WT roots (Fig. <a href="#Fig1" class="usa-link">1b</a>). Root images were analyzed using ImageJ [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>] and quantified for deviation of the root tips from their position at the time of slant initiation (β) (Fig. <a href="#Fig1" class="usa-link">1c</a>) [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. <em>tno1</em> mutants had a higher angle of deviation than the WT or complemented roots (Fig. <a href="#Fig1" class="usa-link">1d</a>; <em>P</em> &lt; 0.05). To confirm the difference in phenotype, another morphometric parameter, the horizontal growth index (HGI), was calculated for the skewing roots. HGI is the ratio of the root tip abscissa (Lx) and length of the root (L) (Fig. <a href="#Fig1" class="usa-link">1c</a>) and is a sensitive and robust parameter to quantify lateral directions of growth, independent of root shape [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Root tips skewed to the right yield positive Lx values and negative values when roots skew left. A positive Lx results in a positive HGI value, which then signifies a rightward deviation while a negative value suggests leftward deviation [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. <em>tno1</em> roots had a significantly higher HGI than WT and complemented lines (Fig. <a href="#Fig1" class="usa-link">1e</a>; <em>P</em> &lt; 0.05), confirming that the <em>tno1</em> mutants do indeed have an increased rightward deviation from the vertical.
</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5ec/5387210/9c0655172212/12870_2017_1024_Fig1_HTML.jpg" loading="lazy" id="MO1" height="381" width="567" alt="Fig. 1"></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Loss of TNO1 protein function causes increased rightward root skewing. <strong>a</strong> Side view of the setup for the root skewing assay on growth medium solidified with 1.5% agar. Seedlings were grown vertically for 3 days in long day (LD) conditions, after which the root tip position was marked. The plates were then slanted backwards, 30<sup>o</sup> to the vertical, and grown for another 4 days. <strong>b</strong> Images of WT, <em>tno1</em> mutant (KO) and complemented <em>tno1</em> mutants (COM) displaying skewing of roots away from the vertical. Scale bar =1 cm. <strong>c</strong> Schematic diagram of analysis of a root to determine the angular deviation of the root tip (β), the horizontal root-tip deviation (Lx) and the length of the root (L). A higher value of β and HGI (=Lx/L) indicate increased skewing. <strong>d</strong> Angular deviation of root tips of WT, KO and COM seedlings subjected to the root skewing assay as described in (a). <strong>e</strong> Horizontal growth index (HGI) of roots of WT, KO and COM seedlings subjected to the skewing assay. HGI is the ratio of the horizontal root tip deviation (Lx) and root length (L). All values represent the means of 3 biological replicates with 12–15 seedlings analyzed for each replicate. Error bars indicate standard error. <em>Asterisks</em> indicate a statistically significant difference (<em>P</em> &lt; 0.05) by Student’s <em>t</em>-test</p></figcaption></figure></section><section id="Sec10"><h3 class="pmc_sec_title">Skewing <em>tno1</em> roots display enhanced root epidermal cell file rotation (CFR)</h3>
<p>The cellular basis for CFR is still not completely understood, but has been proposed to be a result of the circumnutation of the root tip [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. Circumnutation is dependent on an internal mechanism that results in an elliptical or circular trajectory around an imaginary axis of growth [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. The helical pattern of cell division at the root tip has been hypothesized to be the basis for epidermal CFR [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. The characteristic spiral cell division patterns of the outer circle of meristematic cells in the root apex causes a spiraling of the cell files, thus resulting in CFR formation [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>], although this fails to explain why CFR initiates at the base of the elongation zone, away from the site of initial cell divisions. A lag in anisotropic expansion rates between the epidermal and the inner cell layers can also cause a twisting of the cell files to compensate for stresses and strains that might otherwise cause tissue breakage [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. CFR, usually preceding root bending and skewing [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>], is visible at the base of the elongation zone of skewing roots. Rightward-skewing mutants usually display a dominant left-handed CFR [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>, <a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. We therefore hypothesized that the enhanced rightward skewing in the <em>tno1</em> mutants would correlate with an increase in left-handed CFR.</p>
<p>Microscopic analysis of mutant roots showed a marked left-handed epidermal CFR initiating in the root elongation zone (black arrow, Fig. <a href="#Fig2" class="usa-link">2a</a>) while WT and complemented roots mainly have a vertical arrangement of cell files with a lack of distinct CFR in this region (Fig. <a href="#Fig2" class="usa-link">2a</a>). Confocal imaging of propidium iodide-stained roots revealed a distinctive left-handed CFR with a higher twist in the elongation zone of <em>tno1</em> lines compared to the WT and complemented lines (Fig. <a href="#Fig2" class="usa-link">2b</a>). Images were analyzed with ImageJ to quantify the left-handed twist of the cell files relative to the longitudinal axis of the root. The <em>tno1</em> CFRs were significantly higher (<em>P</em> &lt; 0.01) than the WT and complemented root CFRs (Fig. <a href="#Fig2" class="usa-link">2c</a>). This may explain the enhanced skew of the mutant roots, since the increased left-handed twist might cause larger deviations from the axis of growth.
</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5ec/5387210/5c77c3640d4f/12870_2017_1024_Fig2_HTML.jpg" loading="lazy" id="MO2" height="381" width="567" alt="Fig. 2"></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Skewing <em>tno1</em> roots display enhanced root epidermal cell file rotation. <strong>a</strong> Images of the surface of root tips of WT, KO and COM lines subjected to the skewing assay, imaged under a macro zoom microscope. The <em>black arrow</em> points to epidermal cell file rotation. Scale bar =100 μm. <strong>b</strong> Confocal microscopic image of propidium iodide-stained elongation zone of WT, KO and COM roots subjected to the skewing assay. Scale bar =100 μm. <strong>c</strong> Cell file rotation angles of skewing roots of WT, KO and COM lines. All values represent the means of 3 biological replicates, with at least 10 cell files from 5 seedlings analyzed for each replicate. Error bars indicate standard error. <em>Asterisks</em> indicate a statistically significant difference (<em>P</em> &lt; 0.01) by Student’s <em>t</em>-test</p></figcaption></figure></section><section id="Sec11"><h3 class="pmc_sec_title">Skewing of <em>tno1</em> roots is enhanced upon MT stabilization by taxol</h3>
<p>CFR can be caused directly by altered cortical MT organization, which influences anisotropic expansion as MT alignment controls the direction of cellulose deposition [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>, <a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. Several mutants defective in MT stability, organization and dynamics have enhanced right- or leftward skewing, with a dominant left or right-handed CFR respectively [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. Many, but not all, also display a handedness of the cortical MT arrays that is opposite to the CFR handedness of the elongation zone of the roots. Some mutants defective in MT-interacting proteins show right-handed CFR, skew to the left, and display left-handed helical cortical MT arrays in the elongation zone [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>, <a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>], while others have skewing defects but normal transverse MT array orientation in the root elongation zone. This suggests that the relationship between MT dynamics, orientation, and skewing is complex. We hypothesized that the enhanced skewing and CFR in <em>tno1</em> mutants might also be due to changes in cortical MT array organization or dynamics. TNO1 is localized to the TGN, and this hypothesis receives support from a recent study in which a TGN-localized protein was shown to activate a kinesin and be involved in root skewing and gravitropism, linking MTs and the TGN in the root skewing process [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>].</p>
<p>The MT-stabilizing chemical taxol causes MT bundling and enhances rightward skewing in WT roots [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. When grown vertically on increasing concentrations of taxol (Fig. <a href="#Fig3" class="usa-link">3a</a>), <em>tno1</em> roots showed significantly higher rightward skewing than the WT and complemented lines (Fig. <a href="#Fig3" class="usa-link">3a, b</a>). The percentage increase in the skewing angle of the mutant roots from 0 μM to 0.2 μM taxol is significantly higher (<em>P</em> &lt; 0.05) than in the WT and complemented line roots (Fig. <a href="#Fig3" class="usa-link">3c</a>). We hypothesized that this increase in skewing was due to an increase in the left-handed twist of the roots. Analysis of confocal images of propidium iodide stained roots in the presence of 0.2 μM taxol reveals a larger left-handed CFR in the <em>tno1</em> roots compared to WT and complemented lines (Fig. <a href="#Fig3" class="usa-link">3d, e</a>). The phenotype is reflective of the effect of taxol on the root growth rather than the effect of a slanted substratum, since the drug assay is conducted on vertically-oriented plates. These results suggest that MT stabilization exaggerates the root skewing and CFR phenotype of the <em>tno1</em> mutants.
</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5ec/5387210/6c52cc00e408/12870_2017_1024_Fig3_HTML.jpg" loading="lazy" id="MO3" height="589" width="778" alt="Fig. 3"></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Skewing of <em>tno1</em> roots is enhanced upon MT stabilization by taxol. <strong>a</strong> Angle of skew with respect to the horizontal for WT, KO and COM roots grown on two different concentrations of taxol. Seedlings were grown vertically in LD conditions for 5 days on growth media solidified with 1.5% agar and either containing taxol or solvent as a control. All values represent the means of three biological replicates with at least 12–15 seedlings for each replicate. Error bars indicate standard error. <em>Asterisks</em> indicate a statistically significant difference (<em>P</em> &lt; 0.05) by Student’s <em>t</em>-test. <strong>b</strong> Light microscopic images of roots of WT, KO and COM lines grown vertically in the presence of 0.2 μM taxol for 5 days. Scale bar =1 cm. <strong>c</strong> Mean root skew of each genotype on taxol compared to the mean on the solvent control was expressed as percentage increase in skew. Values represent analysis of 3 biological replicates with at least 12 seedlings for each replicate. Error bars indicate standard errors. Different letters indicate statistically significant differences (<em>P</em> &lt; 0.05) by Student’s <em>t</em>-test. <strong>d</strong> Confocal images of propidium iodide-stained elongation zones of WT, KO and COM roots displaying epidermal CFR. The roots were grown vertically on 0.2 μM taxol for 5 days. Scale bar = 100 μm. <strong>e</strong> Cell file rotation angles of skewing roots of WT, KO and COM lines grown on 0.2 μM taxol. All values represent the means of 3 biological replicates with at least 10 cell files from 3 seedlings analyzed for each replicate. Error bars indicate standard errors. <em>Asterisks</em> indicate a statistically significant difference (<em>P</em> &lt; 0.01) by Student’s <em>t</em>-test</p></figcaption></figure></section><section id="Sec12"><h3 class="pmc_sec_title">Effect of MT destabilization by propyzamide (PPD) on root skewing in <em>tno1</em> mutants</h3>
<p>The MT-destabilizing chemical propyzamide (PPD) also enhances rightward skewing in WT seedlings [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. Since the effect of MT stabilization on root skewing in <em>tno1</em> seedlings suggests a MT-based role for TNO1 in root movement, we tested the effect of PPD on root skewing in the <em>tno1</em> mutant. WT and complemented roots showed a slight increase in rightward skew at 1 μM PPD and a marked increase at 3 μM PPD. Surprisingly, skewing of <em>tno1</em> mutant roots was insensitive to PPD (Fig. <a href="#Fig4" class="usa-link">4a, b</a>). The percentage increase in skewing observed for WT and complemented roots in the presence of PPD was significantly higher than in <em>tno1</em> (Fig. <a href="#Fig4" class="usa-link">4c</a>). The increase in skewing for the complemented lines was slightly higher than the WT, suggesting that the precise expression pattern of the TNO1 protein may be reflected in the extent of root skewing. While the TNO1 transgene in the complemented lines is expressed under its native promoter and its overall expression levels are similar to that of the endogenous protein, position effects within the genome may cause slight differences in expression pattern in different cell types.
</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5ec/5387210/cfba72659ed1/12870_2017_1024_Fig4_HTML.jpg" loading="lazy" id="MO4" height="557" width="778" alt="Fig. 4"></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effect of MT destabilization by propyzamide (PPD) on root skewing in <em>tno1</em> mutants. <strong>a</strong> Angle of skew with respect to the horizontal for WT, KO and COM roots grown on two different concentrations of PPD. Seedlings were grown vertically in long-day conditions for 5 days on growth media solidified with 1.5% agar and either containing PPD or solvent as a control. All values represent the means of 3 biological replicates with at least 12–15 seedlings for each replicate. Error bars indicate standard error. <em>Asterisks</em> indicate a statistically significant difference from WT (<em>P</em> &lt; 0.05) by Student’s <em>t</em>-test. <strong>b</strong> Light microscopic image of roots of WT, KO and COM lines grown vertically in the presence of 3 μM PPD for 5 days. Scale bar =1 cm. <strong>c</strong> Mean root skew of each genotype on PPD compared to the mean on the solvent control, expressed as percentage inhibition. Values represent analysis of 3 biological replicates with at least 12 seedlings for each set. Error bars indicate standard error. Different letters indicate statistically significant differences (<em>P</em> &lt; 0.05) by Student’s <em>t</em>-test. <strong>d</strong> Representative image of propidium iodide-stained elongation zones of WT, KO and COM roots grown vertically on 3 μM PPD for 5 days, imaged with a confocal microscope. Scale bar = 100 μm. <strong>e</strong> Cell file rotation angles of roots of WT, KO and COM lines grown on 3 μM PPD. All values represent the means of 3 biological replicates with at least 10 cell files from 3 seedlings analyzed for each replicate. Error bars indicate standard errors</p></figcaption></figure><p>We hypothesized that the reduced skewing of <em>tno1</em> roots in the presence of PPD would correlate with a reduced CFR when compared with CFRs of PPD treated-WT and complemented roots. Confocal microscopic analysis of the surface of roots growing on 3 μM PPD reveals a twisting of WT and complemented epidermal cell files. The mutant root epidermal cell files also display considerable left-handed CFR, indistinguishable from WT and complemented lines (Fig. <a href="#Fig4" class="usa-link">4d, e</a>). Hence, the resistance of <em>tno1</em> roots to PPD-induced skewing is independent of PPD-induced CFR. To explain this observation, we hypothesized that the PPD treatment might have other effects on the mutant roots, including effects on root growth or morphology. There were no measurable differences between the root growth rates of the WT, <em>tno1</em> and complemented lines when grown in the presence or absence of PPD (not shown), and it is therefore unlikely that the difference in skewing is due to differences in growth rate.</p>
<p>Surprisingly, mature regions of <em>tno1</em> roots grown on 3 μM PPD had more severe defects in cell morphology when compared to PPD-treated WT and complemented lines, with the mutants displaying malformed cells (Fig. <a href="#Fig5" class="usa-link">5a</a>). Whereas PPD-treated WT and complemented roots have typical long, narrow cells, <em>tno1</em> mutant root cells are shorter and wider, indicating defects in anisotropic growth. This points to a possible defect in cell maturation or cell wall deposition that could potentially interfere with the ability of the <em>tno1</em> roots to skew in the presence of PPD even as the CFR in the elongation zone continues to form normally. It is possible that the defects in the mature zone of the PPD-treated <em>tno1</em> roots could then affect how the growing root interacts with the substrate, which would in turn affect root skewing.
</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5ec/5387210/776337b7bc94/12870_2017_1024_Fig5_HTML.jpg" loading="lazy" id="MO5" height="359" width="472" alt="Fig. 5"></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effect of 3 μM propyzamide (PPD) on cell morphology of the mature region of the root and the base of the hypocotyl. <strong>a</strong> Representative image of propidium iodide-stained mature zone of WT, KO and COM roots grown on solvent as a control or 3 μM PPD for 7 days, imaged with a confocal microscope. Scale bar =100 μm. <strong>b</strong> Representative agarose imprints of the base of WT, KO and COM hypocotyls grown on solvent control or 3 μM PPD for 7 days, imaged with a macro zoom microscope. Scale bar =100 μm</p></figcaption></figure><p>To assess whether other organs and cell types in the mutant might also be sensitive to MT disruption by PPD, we analyzed agarose imprints of control and PPD- treated dark-grown hypocotyls. The base of the hypocotyls in the WT and complemented lines had cells with increased width when compared to their untreated counterparts. <em>tno1</em> hypocotyls were hypersensitive to PPD, with much more severe defects in cell morphology when compared to the untreated <em>tno1</em> seedlings and to PPD-treated WT and complemented seedlings (Fig. <a href="#Fig5" class="usa-link">5b</a>). The PPD-treated <em>tno1</em> seedlings also lacked the spiralization of the cells observed in the treated WT and complemented lines. As observed in the skewing assay, the complemented lines were slightly more sensitive to PPD than WT, which again may be due to position effects. Cell elongation is initiated at the base of dark grown hypocotyls [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>], and the loss of TNO1 may lead to an increased susceptibility of this cell expansion process to MT destabilization by PPD. Treatment with low concentrations of PPD (such as 2 μM) has been reported to cause a decrease in the dynamic instability of cortical MTs, leading to more static MTs that spend more time in the paused state compared to untreated cells [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. Our results suggest that the disruption of MT dynamics by PPD has a more adverse effect on <em>tno1</em> roots than WT. This could be manifest as defects in MT-array organization, anisotropic expansion, or both, resulting in the observed disruption in cell morphology in <em>tno1</em>.</p></section><section id="Sec13"><h3 class="pmc_sec_title">The orientation of MT arrays in the elongating cells of skewing <em>tno1</em> roots is not affected</h3>
<p>MT arrays in elongation zones of roots are usually arranged in a transverse orientation to facilitate anisotropic expansion. Skewing roots of some mutants with marked CFRs have distinct oblique arrays of MTs, with the handedness of a MT array opposite to that of CFR [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. Strong correlations have also been observed between angle of skewing and pitch of the MT arrays [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. Many mutants that are defective in MT-associated processes have a change in the pitch of MT arrays associated with a corresponding change in CFR. However, some mutants with enhanced CFR show no obliqueness of the MT array, but rather have transverse arrangements similar to the central elongation zone cells of the WT roots. For example, the <em>sku6</em> mutant maintains a transverse alignment of MT arrays despite its left-handed CFR [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>] while the <em>mor1–1</em> mutant, impaired in MT polymerization, also displays a left-handed CFR but no bias in MT array orientation [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. These mutant phenotypes suggest that additional factors can also be responsible for a change in CFR, apart from the MT array orientation.</p>
<p>We hypothesized that the dominant left-handed CFR in the <em>tno1</em> roots would correspond to a right-handed MT array in the epidermal cells of the elongation zone compared to WT roots. Four-day-old chemically fixed WT and <em>tno1</em> roots, grown on a slanted impenetrable medium, were immunostained with tubulin antibodies [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>] to detect MTs and imaged by confocal microscopy. The MT arrays of epidermal cells in the mutant roots did not show any distinct handedness or change in pitch compared to WT epidermal MT arrays (Fig. <a href="#Fig6" class="usa-link">6a</a>). Quantitative analyses on collected images were performed utilizing the MATLAB software package MicroFilament Analyzer [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>] to allow comparison of MT orientation in multiple images from different biological replicates. The representative circular output graphs from the analyses suggest a mainly transverse orientation of the MTs in the elongation zone of both WT and <em>tno1</em> roots (blue line along the 0–180 degree axis) (Fig. <a href="#Fig6" class="usa-link">6b</a>). To assess whether a difference in the percentage of oblique MTs between the WT and mutant might explain the higher CFR in the mutants, MT angles of root cells from WT and <em>tno1</em> roots were classified into three classes, 0 to 10, 11 to 30 and 31 to 90 degrees (Fig. <a href="#Fig6" class="usa-link">6c</a>). No significant difference was found in the percentage of MTs in each class between the WT and mutant. Thus we conclude that <em>tno1</em> mutants have normal transverse MT arrays in the skewing roots, suggesting that an alternative mechanism must be responsible for the change in CFR.
</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5ec/5387210/90c1a7357ea7/12870_2017_1024_Fig6_HTML.jpg" loading="lazy" id="MO6" height="422" width="567" alt="Fig. 6"></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The orientation of MT arrays in the elongating cells of skewing mutant roots is not affected. <strong>a</strong> Representative confocal images of the cortical microtubule array of elongating cells of 5-day-old <em>Arabidopsis</em> roots grown on slanted medium, immunostained with anti-α<strong>-</strong> tubulin. The boxed inset indicates the region that was selected for the MT array analysis in 6b. Scale bar =10 μm. <strong>b</strong> Graphical representation of the analyzed MTs as a circular plot derived from analysis of the confocal images in part a with the MicroFilament Analyzer software (MFA). The blue line represents the original data (the MT angles) while the red line represents smoothed data. Green asterisks denote dominant orientations while the yellow line is a weighted average of the detected filaments per angle. <strong>c</strong> Microtubule angle distribution in the elongating cells of WT and KO roots was calculated from the data generated via the MFA software. Values represent analysis of 3 biological replicates with at least 500 MTs analyzed from 3 cells of three different roots for each replicate. Error bars indicate standard errors</p></figcaption></figure></section></section><section id="Sec14"><h2 class="pmc_sec_title">Discussion</h2>
<p>TNO1 plays important roles in post-Golgi trafficking, gravitropism and auxin transport-dependent processes. Here we demonstrate that TNO1 is required for root skewing and CFR formation. Skewing of <em>tno1</em> mutant roots increased to a greater extent upon MT stabilization by taxol when compared to WT roots but was resistant to MT destabilization by PPD. This led us to postulate a role for TNO1 in MT-dependent processes, although MT destabilizing agents have also been reported to induce left-handed CFR without affecting MT arrays [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. In roots, MT arrays transition from a disordered state in the meristematic zone to a transverse orientation in the elongation zone, and this organization is important for facilitating rapid anisotropic growth [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. Defects in tubulin structure/activity and in MT-associated proteins can result in oblique MT arrays in the central elongation zone of roots. Oblique MT orientation in elongating cells causes cells to expand at an angle to the vertical, leading to CFR and root skewing [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>, <a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. Skewing roots of <em>tno1</em> do not show an oblique MT orientation in the elongation zone but rather a transverse one. This suggests that the root movement phenotype in <em>tno1</em> mutants arises independent of the organization of MT arrays in the root elongation zone, although dependent on MT dynamics. It is possible that, despite its normal orientation, the cortical MT array in <em>tno1</em> roots might be compromised in restructuring or function as the root navigates a complex trajectory during skewing. The loss of TNO1 could potentially also affect the function of a MT-associated protein such as a kinesin; as an example, a TGN-localized kinesin-binding separase has recently been reported to affect root skewing [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>].</p>
<p>Certain root skewing mutants have transverse MT arrays or a lack of directional bias in the MT arrays in the root elongation zone but have a distinct CFR [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>, <a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>, <a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. Moreover, CFR can still occur upon MT disruption with MT destabilizing agents such as oryzalin and PPD [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. This suggests that the MT array is not the only factor involved in generating CFR. Owing to anomalies such as these, Wasteneys (2004) proposed that CFR could arise due to torsional handedness inherent to roots, and the correlation between CFR and MT array orientations could result from signaling between the MT array and cell wall machinery. Thus, CFR could arise from the effect of MT arrays on anisotropic expansion in the elongation zone or, alternatively, from the effect of cellulose microfibrils on MT array alignment at the plasma membrane [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>, <a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. Mutants that affect CFR formation but not MT array orientation in the elongation zone support this model, including mutants defective in regulation of anisotropic cell expansion [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. Based on this model, and consistent with the cell morphology phenotypes observed in the presence of PPD, TNO1 might be important for the anisotropic cell expansion process or in cell wall trafficking dynamics.</p>
<p>A lag in the expansion rates of internal cell types compared to the epidermal cells may cause roots to skew to prevent mechanical shearing [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. The mature root cells and hypocotyls of <em>tno1</em> seedlings treated with PPD showed marked disruption of cell morphology, suggesting a role for TNO1 in the cell maturation or expansion process. This model would also explain why <em>tno1</em> roots skew despite the transverse MT arrays in the epidermal cells of the root elongation zone. Vertically-grown <em>tno1</em> seedlings have similar root lengths to WT seedlings, suggesting normal cell expansion during vertical growth. Thus TNO1 might be required specifically for cell expansion during deviation of roots from their growth trajectory.</p>
<p>During cell expansion and cellulose deposition, the cellulose synthase complex traffics via the TGN [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>, <a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. The observed defects in cell morphology and expansion could be due to the role of TNO1 in proper localization of the TGN-localized Q-SNARE SYP61 [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>, <a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Proteomic analysis of SYP61-containing vesicles [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>] revealed the presence of cellulose synthase complex subunits, suggesting a role in trafficking of cellulose synthase subunits, which then drives cellulose deposition. MT array dynamics modulate the deposition of cellulose microfibrils in expanding cells since cellulose synthase runs parallel to the MT array, with CELLULOSE SYNTHASE INTERACTIVE1 acting as a link between the cellulose synthase machinery and cortical MTs [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>–<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. MTs also regulate exocytosis of vesicles containing cellulose synthase complex subunits or complex-associated proteins such as KOR1 and CSI1 [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>–<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]. Disrupting MT arrays therefore affects cellulose synthesis, deposition and orientation, while conversely, inhibiting cellulose synthase activity causes defects in MT arrays [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>, <a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>], which can potentially affect directional growth patterns [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. Inhibitors of cellulose synthase trafficking such as CESTRIN cause MT instability and reduced cellulose content, and increase SYP61 and cellulose synthase co-localization [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]. Since CESTRIN does not affect the bulk secretory and endocytic routes, the SYP61-decorated domain of the TGN and its associated proteins such as TNO1 seem to be crucial for cellulose deposition and, indirectly, MT stability. However, mutations in cellulose synthase components (<em>rsw1–1</em>, <em>any1</em>) or treatment with cellulose synthase inhibitors does not induce CFR [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>, <a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>, <a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>]. Moreover, Sugimoto (2003) also reports that a cellulose synthase mutant can undergo CFR even when the MTs are compromised [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>].</p>
<p>Trafficking of other proteins required for cell expansion and root movement might also be disrupted in <em>tno1</em> mutants. For example, SKU5 is required for enzymatic reactions at the cell wall, and mutants lacking this protein show enhanced root skewing possibly due to altered cell wall composition during cell expansion [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. Our observations suggest that CFR formation involves complex interactions and is not simply dependent on MT array and cellulose deposition patterns. Current models of the mechanisms of helical growth have been challenged by studies on <em>tortifolia2</em> mutants, in which individual freely growing trichomes twist. These mutants therefore uncouple cell division patterns and tissue strain from twisting and suggest that individual twisting of cells can translate to higher-order organ twisting [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. Understanding the exact role of TNO1 at the TGN and the mechanism by which it modulates CFR and skewing will thus be a challenge.</p>
<p>Our results suggest that TNO1 and the TGN play a key role in CFR and root movement, adding another component to the network controlling root movement. Further experiments are necessary to test the exact role of TNO1 and its interacting partners in regulating root skewing and CFR formation. Future studies investigating TNO1’s role in cellulose synthase trafficking, MT stability and cell expansion dynamics in skewing roots will increase our understanding of the function of TNO1. Since TNO1 affects auxin transport [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>], our study now points to the possibility of an impact of TNO1 on auxin transport, MT dynamics and cell wall trafficking at the TGN. Discovering TNO1’s precise role will help in elucidation of the function of the TGN SNARE machinery in plant growth and morphogenesis.</p></section><section id="Sec15"><h2 class="pmc_sec_title">Conclusions</h2>
<p>In conclusion, we have discovered a role for a TGN-localized protein in root skewing on the surface of impenetrable media. This adds to the growing list of endogenous factors that aid root movement. Our data, alongside prior investigations, also suggest a possible link between TNO1, auxin-dependent and MT-associated processes. A better understanding of these mechanisms can lead to information that can be used to manipulate root growth and development patterns in various soil conditions and improve overall root architecture. This could subsequently result in engineering superior crops with better soil penetration and nutrient assimilation traits.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Dr. Patrick Masson and Dr. Kate Baldwin for advice on the skewing assays, Colton McNinch and Dr. Xiaochen Yang for technical help and advice, Dr. Bin Dong and Jennifer Johnson for assistance with the MFA software, Margaret Carter for guidance on confocal imaging and Dr. Divita Mathur for conceptual insights.</p>
<section id="FPar1"><h3 class="pmc_sec_title">Funding</h3>
<p>This work was supported by a grant from the National Aeronautics and Space Administration (Grant no. NNX09AK78G) to DCB.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Availability of data and materials</h3>
<p>All data generated or analyzed during this study are included in this published article. Raw files are available upon request from Dr. Rahul Roy (rroy@umn.edu).</p></section><section id="FPar3"><h3 class="pmc_sec_title">Authors’ contributions</h3>
<p>RR and DCB conceived of the study and designed the experiments. RR collected and analyzed the data. RR and DCB wrote and revised the manuscript for final publication.</p></section><section id="FPar4"><h3 class="pmc_sec_title">Competing interests</h3>
<p>The authors declare that they have no competing interests.</p></section><section id="FPar5"><h3 class="pmc_sec_title">Consent for publication</h3>
<p>Not applicable.</p></section><section id="FPar6"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p>Not applicable.</p></section><section id="FPar7"><h3 class="pmc_sec_title">Publisher’s Note</h3>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></section></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>CFR</dt>
<dd><p>Cell file rotation</p></dd>
<dt>MT(s)</dt>
<dd><p>Microtubule(s)</p></dd>
<dt>PPD</dt>
<dd><p>Propyzamide</p></dd>
<dt>SNARE</dt>
<dd><p>Soluble <em>N</em>-ethylmaleimide–sensitive factor attachment protein receptor</p></dd>
<dt>TGN</dt>
<dd><p>
<em>Trans-</em>Golgi network</p></dd>
<dt>TNO1</dt>
<dd><p>TGN-localized SYP41-interacting protein</p></dd>
</dl></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Rahul Roy, Email: rroy@umn.edu.</p>
<p>Diane C. Bassham, Email: bassham@iastate.edu</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Hodge A, Berta G, Doussan C, Merchan F, Crespi M. Plant root growth, architecture and function. Plant Soil. 2009;321(1–2):153–187. doi: 10.1007/s11104-009-9929-9.</cite> [<a href="https://doi.org/10.1007/s11104-009-9929-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Soil&amp;title=Plant%20root%20growth,%20architecture%20and%20function&amp;author=A%20Hodge&amp;author=G%20Berta&amp;author=C%20Doussan&amp;author=F%20Merchan&amp;author=M%20Crespi&amp;volume=321&amp;issue=1%E2%80%932&amp;publication_year=2009&amp;pages=153-187&amp;doi=10.1007/s11104-009-9929-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Dunbabin VM, Postma JA, Schnepf A, Pages L, Javaux M, Wu LH, Leitner D, Chen YL, Rengel Z, Diggle AJ. Modelling root-soil interactions using three-dimensional models of root growth, architecture and function. Plant Soil. 2013;372(1–2):93–124. doi: 10.1007/s11104-013-1769-y.</cite> [<a href="https://doi.org/10.1007/s11104-013-1769-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Soil&amp;title=Modelling%20root-soil%20interactions%20using%20three-dimensional%20models%20of%20root%20growth,%20architecture%20and%20function&amp;author=VM%20Dunbabin&amp;author=JA%20Postma&amp;author=A%20Schnepf&amp;author=L%20Pages&amp;author=M%20Javaux&amp;volume=372&amp;issue=1%E2%80%932&amp;publication_year=2013&amp;pages=93-124&amp;doi=10.1007/s11104-013-1769-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Rellan-Alvarez R, Lobet G, Lindner H, Pradier PL, Sebastian J, Yee MC, Geng Y, Trontin C, LaRue T, Schrager-Lavelle A, et al. GLO-Roots: an imaging platform enabling multidimensional characterization of soil-grown root systems. elife. 2015;4:e07597.</cite> [<a href="https://doi.org/10.7554/eLife.07597" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4589753/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26287479/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Roy R, Bassham DC. Root growth movements: Waving and skewing. Plant Sci. 2014;221-222C:42–47. doi: 10.1016/j.plantsci.2014.01.007.</cite> [<a href="https://doi.org/10.1016/j.plantsci.2014.01.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24656334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Sci&amp;title=Root%20growth%20movements:%20Waving%20and%20skewing&amp;author=R%20Roy&amp;author=DC%20Bassham&amp;volume=221-222C&amp;publication_year=2014&amp;pages=42-47&amp;pmid=24656334&amp;doi=10.1016/j.plantsci.2014.01.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Migliaccio F, Piconese S. Spiralizations and tropisms in Arabidopsis roots. Trends Plant Sci. 2001;6(12):561–565. doi: 10.1016/S1360-1385(01)02152-5.</cite> [<a href="https://doi.org/10.1016/S1360-1385(01)02152-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11738380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Plant%20Sci&amp;title=Spiralizations%20and%20tropisms%20in%20Arabidopsis%20roots&amp;author=F%20Migliaccio&amp;author=S%20Piconese&amp;volume=6&amp;issue=12&amp;publication_year=2001&amp;pages=561-565&amp;pmid=11738380&amp;doi=10.1016/S1360-1385(01)02152-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Okada KaS Y. Reversible root tip rotation in Arabidopsis seedlings induced by obstacle-touching stimulus. Science. 1990;250:274–276. doi: 10.1126/science.250.4978.274.</cite> [<a href="https://doi.org/10.1126/science.250.4978.274" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17797309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Reversible%20root%20tip%20rotation%20in%20Arabidopsis%20seedlings%20induced%20by%20obstacle-touching%20stimulus&amp;author=Y%20Okada%20KaS&amp;volume=250&amp;publication_year=1990&amp;pages=274-276&amp;pmid=17797309&amp;doi=10.1126/science.250.4978.274&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Rutherford R, Masson PH. Arabidopsis thaliana sku mutant seedlings show exaggerated surface-dependent alteration in root growth vector. Plant Physiol. 1996;111(4):987–998. doi: 10.1104/pp.111.4.987.</cite> [<a href="https://doi.org/10.1104/pp.111.4.987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC160968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8756492/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Arabidopsis%20thaliana%20sku%20mutant%20seedlings%20show%20exaggerated%20surface-dependent%20alteration%20in%20root%20growth%20vector&amp;author=R%20Rutherford&amp;author=PH%20Masson&amp;volume=111&amp;issue=4&amp;publication_year=1996&amp;pages=987-998&amp;pmid=8756492&amp;doi=10.1104/pp.111.4.987&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Buer CS, Wasteneys GO, Masle J. Ethylene modulates root-wave responses in Arabidopsis. Plant Physiol. 2003;132(2):1085–1096. doi: 10.1104/pp.102.019182.</cite> [<a href="https://doi.org/10.1104/pp.102.019182" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC167046/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12805636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Ethylene%20modulates%20root-wave%20responses%20in%20Arabidopsis&amp;author=CS%20Buer&amp;author=GO%20Wasteneys&amp;author=J%20Masle&amp;volume=132&amp;issue=2&amp;publication_year=2003&amp;pages=1085-1096&amp;pmid=12805636&amp;doi=10.1104/pp.102.019182&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Furutani I, Watanabe Y, Prieto R, Masukawa M, Suzuki K, Naoi K, Thitamadee S, Shikanai T, Hashimoto T. The SPIRAL genes are required for directional control of cell elongation in Aarabidopsis thaliana. Development. 2000;127(20):4443–4453. doi: 10.1242/dev.127.20.4443.</cite> [<a href="https://doi.org/10.1242/dev.127.20.4443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11003843/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=The%20SPIRAL%20genes%20are%20required%20for%20directional%20control%20of%20cell%20elongation%20in%20Aarabidopsis%20thaliana&amp;author=I%20Furutani&amp;author=Y%20Watanabe&amp;author=R%20Prieto&amp;author=M%20Masukawa&amp;author=K%20Suzuki&amp;volume=127&amp;issue=20&amp;publication_year=2000&amp;pages=4443-4453&amp;pmid=11003843&amp;doi=10.1242/dev.127.20.4443&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Yuen CYL, Sedbrook JC, Perrin RM, Carroll KL, Masson PH. Loss-of-function mutations of ROOT HAIR DEFECTIVE3 suppress root waving, skewing, and epidermal cell file rotation in Arabidopsis. Plant Physiol. 2005;138(2):701–714. doi: 10.1104/pp.105.059774.</cite> [<a href="https://doi.org/10.1104/pp.105.059774" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1150390/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15908600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Loss-of-function%20mutations%20of%20ROOT%20HAIR%20DEFECTIVE3%20suppress%20root%20waving,%20skewing,%20and%20epidermal%20cell%20file%20rotation%20in%20Arabidopsis&amp;author=CYL%20Yuen&amp;author=JC%20Sedbrook&amp;author=RM%20Perrin&amp;author=KL%20Carroll&amp;author=PH%20Masson&amp;volume=138&amp;issue=2&amp;publication_year=2005&amp;pages=701-714&amp;pmid=15908600&amp;doi=10.1104/pp.105.059774&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Vaughn LM, Masson PH. A QTL Study for Regions Contributing to Arabidopsis thaliana Root Skewing on Tilted Surfaces. G3 (Bethesda) 2011;1(2):105–115. doi: 10.1534/g3.111.000331.</cite> [<a href="https://doi.org/10.1534/g3.111.000331" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3276130/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22384323/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20(Bethesda)&amp;title=A%20QTL%20Study%20for%20Regions%20Contributing%20to%20Arabidopsis%20thaliana%20Root%20Skewing%20on%20Tilted%20Surfaces&amp;author=LM%20Vaughn&amp;author=PH%20Masson&amp;volume=1&amp;issue=2&amp;publication_year=2011&amp;pages=105-115&amp;pmid=22384323&amp;doi=10.1534/g3.111.000331&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Santner AA, Watson JC. The WAG1 and WAG2 protein kinases negatively regulate root waving in Arabidopsis. Plant J. 2006;45(5):752–764. doi: 10.1111/j.1365-313X.2005.02641.x.</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2005.02641.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16460509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=The%20WAG1%20and%20WAG2%20protein%20kinases%20negatively%20regulate%20root%20waving%20in%20Arabidopsis&amp;author=AA%20Santner&amp;author=JC%20Watson&amp;volume=45&amp;issue=5&amp;publication_year=2006&amp;pages=752-764&amp;pmid=16460509&amp;doi=10.1111/j.1365-313X.2005.02641.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Qi B, Zheng H. Modulation of root-skewing responses by KNAT1 in Arabidopsis thaliana. Plant J. 2013;76(3):380–392. doi: 10.1111/tpj.12295.</cite> [<a href="https://doi.org/10.1111/tpj.12295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23889705/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=Modulation%20of%20root-skewing%20responses%20by%20KNAT1%20in%20Arabidopsis%20thaliana&amp;author=B%20Qi&amp;author=H%20Zheng&amp;volume=76&amp;issue=3&amp;publication_year=2013&amp;pages=380-392&amp;pmid=23889705&amp;doi=10.1111/tpj.12295&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Whitford R, Fernandez A, Tejos R, Perez AC, Kleine-Vehn J, Vanneste S, Drozdzecki A, Leitner J, Abas L, Aerts M, et al.  GOLVEN secretory peptides regulate auxin carrier turnover during plant gravitropic responses. Dev Cell. 2012;22(3):678–685. doi: 10.1016/j.devcel.2012.02.002.</cite> [<a href="https://doi.org/10.1016/j.devcel.2012.02.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22421050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev%20Cell&amp;title=GOLVEN%20secretory%20peptides%20regulate%20auxin%20carrier%20turnover%20during%20plant%20gravitropic%20responses&amp;author=R%20Whitford&amp;author=A%20Fernandez&amp;author=R%20Tejos&amp;author=AC%20Perez&amp;author=J%20Kleine-Vehn&amp;volume=22&amp;issue=3&amp;publication_year=2012&amp;pages=678-685&amp;pmid=22421050&amp;doi=10.1016/j.devcel.2012.02.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Santisree P, Nongmaithem S, Sreelakshmi Y, Ivanchenko M, Sharma R. The root as a drill: an ethylene-auxin interaction facilitates root penetration in soil. Plant Signal Behav. 2012;7(2):151–156. doi: 10.4161/psb.18936.</cite> [<a href="https://doi.org/10.4161/psb.18936" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3405696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22415043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Signal%20Behav&amp;title=The%20root%20as%20a%20drill:%20an%20ethylene-auxin%20interaction%20facilitates%20root%20penetration%20in%20soil&amp;author=P%20Santisree&amp;author=S%20Nongmaithem&amp;author=Y%20Sreelakshmi&amp;author=M%20Ivanchenko&amp;author=R%20Sharma&amp;volume=7&amp;issue=2&amp;publication_year=2012&amp;pages=151-156&amp;pmid=22415043&amp;doi=10.4161/psb.18936&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Kushwah S, Laxmi A. The interaction between glucose and cytokinin signal transduction pathway in Arabidopsis thaliana. Plant Cell Environ. 2014;37(1):235–253. doi: 10.1111/pce.12149.</cite> [<a href="https://doi.org/10.1111/pce.12149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23763631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Environ&amp;title=The%20interaction%20between%20glucose%20and%20cytokinin%20signal%20transduction%20pathway%20in%20Arabidopsis%20thaliana&amp;author=S%20Kushwah&amp;author=A%20Laxmi&amp;volume=37&amp;issue=1&amp;publication_year=2014&amp;pages=235-253&amp;pmid=23763631&amp;doi=10.1111/pce.12149&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Lanza M, Garcia-Ponce B, Castrillo G, Catarecha P, Sauer M, Rodriguez-Serrano M, Paez-Garcia A, Sanchez-Bermejo E, T CM. Leo del Puerto Y, et al.  Role of actin cytoskeleton in brassinosteroid signaling and in its integration with the auxin response in plants. Dev Cell. 2012;22(6):1275–1285. doi: 10.1016/j.devcel.2012.04.008.</cite> [<a href="https://doi.org/10.1016/j.devcel.2012.04.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22698285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev%20Cell&amp;title=Role%20of%20actin%20cytoskeleton%20in%20brassinosteroid%20signaling%20and%20in%20its%20integration%20with%20the%20auxin%20response%20in%20plants&amp;author=M%20Lanza&amp;author=B%20Garcia-Ponce&amp;author=G%20Castrillo&amp;author=P%20Catarecha&amp;author=M%20Sauer&amp;volume=22&amp;issue=6&amp;publication_year=2012&amp;pages=1275-1285&amp;pmid=22698285&amp;doi=10.1016/j.devcel.2012.04.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Ishida T, Thitamadee S, Hashimoto T. Twisted growth and organization of cortical microtubules. J Plant Res. 2007;120(1):61–70. doi: 10.1007/s10265-006-0039-y.</cite> [<a href="https://doi.org/10.1007/s10265-006-0039-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17061141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Plant%20Res&amp;title=Twisted%20growth%20and%20organization%20of%20cortical%20microtubules&amp;author=T%20Ishida&amp;author=S%20Thitamadee&amp;author=T%20Hashimoto&amp;volume=120&amp;issue=1&amp;publication_year=2007&amp;pages=61-70&amp;pmid=17061141&amp;doi=10.1007/s10265-006-0039-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Sedbrook JC, Kaloriti D. Microtubules, MAPs and plant directional cell expansion. Trends Plant Sci. 2008;13(6):303–310. doi: 10.1016/j.tplants.2008.04.002.</cite> [<a href="https://doi.org/10.1016/j.tplants.2008.04.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18467155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Plant%20Sci&amp;title=Microtubules,%20MAPs%20and%20plant%20directional%20cell%20expansion&amp;author=JC%20Sedbrook&amp;author=D%20Kaloriti&amp;volume=13&amp;issue=6&amp;publication_year=2008&amp;pages=303-310&amp;pmid=18467155&amp;doi=10.1016/j.tplants.2008.04.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Sedbrook JC, Carroll KL, Hung KF, Masson PH, Somerville CR. The Arabidopsis SKU5 gene encodes an extracellular glycosyl phosphatidylinositol-anchored glycoprotein involved in directional root growth. Plant Cell. 2002;14(7):1635–1648. doi: 10.1105/tpc.002360.</cite> [<a href="https://doi.org/10.1105/tpc.002360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC150712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12119380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20Arabidopsis%20SKU5%20gene%20encodes%20an%20extracellular%20glycosyl%20phosphatidylinositol-anchored%20glycoprotein%20involved%20in%20directional%20root%20growth&amp;author=JC%20Sedbrook&amp;author=KL%20Carroll&amp;author=KF%20Hung&amp;author=PH%20Masson&amp;author=CR%20Somerville&amp;volume=14&amp;issue=7&amp;publication_year=2002&amp;pages=1635-1648&amp;pmid=12119380&amp;doi=10.1105/tpc.002360&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Roudier F, Fernandez AG, Fujita M, Himmelspach R, Borner GH, Schindelman G, Song S, Baskin TI, Dupree P, Wasteneys GO, et al.  COBRA, an Arabidopsis extracellular glycosyl-phosphatidyl inositol-anchored protein, specifically controls highly anisotropic expansion through its involvement in cellulose microfibril orientation. Plant Cell. 2005;17(6):1749–1763. doi: 10.1105/tpc.105.031732.</cite> [<a href="https://doi.org/10.1105/tpc.105.031732" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1143074/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15849274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=COBRA,%20an%20Arabidopsis%20extracellular%20glycosyl-phosphatidyl%20inositol-anchored%20protein,%20specifically%20controls%20highly%20anisotropic%20expansion%20through%20its%20involvement%20in%20cellulose%20microfibril%20orientation&amp;author=F%20Roudier&amp;author=AG%20Fernandez&amp;author=M%20Fujita&amp;author=R%20Himmelspach&amp;author=GH%20Borner&amp;volume=17&amp;issue=6&amp;publication_year=2005&amp;pages=1749-1763&amp;pmid=15849274&amp;doi=10.1105/tpc.105.031732&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Wolf S, Mravec J, Greiner S, Mouille G, Hofte H. Plant cell wall homeostasis is mediated by brassinosteroid feedback signaling. Curr Biol. 2012;22(18):1732–1737. doi: 10.1016/j.cub.2012.07.036.</cite> [<a href="https://doi.org/10.1016/j.cub.2012.07.036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22885061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&amp;title=Plant%20cell%20wall%20homeostasis%20is%20mediated%20by%20brassinosteroid%20feedback%20signaling&amp;author=S%20Wolf&amp;author=J%20Mravec&amp;author=S%20Greiner&amp;author=G%20Mouille&amp;author=H%20Hofte&amp;volume=22&amp;issue=18&amp;publication_year=2012&amp;pages=1732-1737&amp;pmid=22885061&amp;doi=10.1016/j.cub.2012.07.036&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Staehelin LA, Kang BH. Nanoscale architecture of endoplasmic reticulum export sites and of Golgi membranes as determined by electron tomography. Plant Physiol. 2008;147(4):1454–1468. doi: 10.1104/pp.108.120618.</cite> [<a href="https://doi.org/10.1104/pp.108.120618" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2492626/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18678738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Nanoscale%20architecture%20of%20endoplasmic%20reticulum%20export%20sites%20and%20of%20Golgi%20membranes%20as%20determined%20by%20electron%20tomography&amp;author=LA%20Staehelin&amp;author=BH%20Kang&amp;volume=147&amp;issue=4&amp;publication_year=2008&amp;pages=1454-1468&amp;pmid=18678738&amp;doi=10.1104/pp.108.120618&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Reyes FC, Buono R, Otegui MS. Plant endosomal trafficking pathways. Curr Opin Plant Biol. 2011;14(6):666–673. doi: 10.1016/j.pbi.2011.07.009.</cite> [<a href="https://doi.org/10.1016/j.pbi.2011.07.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21821464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Plant%20Biol&amp;title=Plant%20endosomal%20trafficking%20pathways&amp;author=FC%20Reyes&amp;author=R%20Buono&amp;author=MS%20Otegui&amp;volume=14&amp;issue=6&amp;publication_year=2011&amp;pages=666-673&amp;pmid=21821464&amp;doi=10.1016/j.pbi.2011.07.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Contento AL, Bassham DC. Structure and function of endosomes in plant cells. J Cell Sci. 2012;125(Pt 15):3511–3518. doi: 10.1242/jcs.093559.</cite> [<a href="https://doi.org/10.1242/jcs.093559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22935651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&amp;title=Structure%20and%20function%20of%20endosomes%20in%20plant%20cells&amp;author=AL%20Contento&amp;author=DC%20Bassham&amp;volume=125&amp;issue=Pt%2015&amp;publication_year=2012&amp;pages=3511-3518&amp;pmid=22935651&amp;doi=10.1242/jcs.093559&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Gendre D, Jonsson K, Boutte Y, Bhalerao RP. Journey to the cell surface-the central role of the trans-Golgi network in plants. Protoplasma. 2015;252(2):385–98.</cite> [<a href="https://doi.org/10.1007/s00709-014-0693-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25187082/" class="usa-link">PubMed</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Sauer M, Delgadillo AO, Zouhar J, Reynolds GD, Pennington JG, Jiang L, Liljegren SJ, Stierhof YD, De Jaeger G, Otegui MS, et al.  MTV1 and MTV4 Encode Plant-Specific ENTH and ARF GAP Proteins That Mediate Clathrin-Dependent Trafficking of Vacuolar Cargo from the Trans-Golgi Network. Plant Cell. 2013;25(6):2217–2235. doi: 10.1105/tpc.113.111724.</cite> [<a href="https://doi.org/10.1105/tpc.113.111724" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3723622/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23771894/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=MTV1%20and%20MTV4%20Encode%20Plant-Specific%20ENTH%20and%20ARF%20GAP%20Proteins%20That%20Mediate%20Clathrin-Dependent%20Trafficking%20of%20Vacuolar%20Cargo%20from%20the%20Trans-Golgi%20Network&amp;author=M%20Sauer&amp;author=AO%20Delgadillo&amp;author=J%20Zouhar&amp;author=GD%20Reynolds&amp;author=JG%20Pennington&amp;volume=25&amp;issue=6&amp;publication_year=2013&amp;pages=2217-2235&amp;pmid=23771894&amp;doi=10.1105/tpc.113.111724&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Qi XY, Zheng HQ. Rab-A1c GTPase Defines a Population of the Trans-Golgi Network that Is Sensitive to Endosidin1 during Cytokinesis in Arabidopsis. Mol Plant. 2013;6(3):847–859. doi: 10.1093/mp/sss116.</cite> [<a href="https://doi.org/10.1093/mp/sss116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23075992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Plant&amp;title=Rab-A1c%20GTPase%20Defines%20a%20Population%20of%20the%20Trans-Golgi%20Network%20that%20Is%20Sensitive%20to%20Endosidin1%20during%20Cytokinesis%20in%20Arabidopsis&amp;author=XY%20Qi&amp;author=HQ%20Zheng&amp;volume=6&amp;issue=3&amp;publication_year=2013&amp;pages=847-859&amp;pmid=23075992&amp;doi=10.1093/mp/sss116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Feraru E, Feraru MI, Asaoka R, Paciorek T, De Rycke R, Tanaka H, Nakano A, Friml J. BEX5/RabA1b Regulates trans-Golgi Network-to-Plasma Membrane Protein Trafficking in Arabidopsis. Plant Cell. 2012;24(7):3074–3086. doi: 10.1105/tpc.112.098152.</cite> [<a href="https://doi.org/10.1105/tpc.112.098152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3426133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22773752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=BEX5/RabA1b%20Regulates%20trans-Golgi%20Network-to-Plasma%20Membrane%20Protein%20Trafficking%20in%20Arabidopsis&amp;author=E%20Feraru&amp;author=MI%20Feraru&amp;author=R%20Asaoka&amp;author=T%20Paciorek&amp;author=R%20De%20Rycke&amp;volume=24&amp;issue=7&amp;publication_year=2012&amp;pages=3074-3086&amp;pmid=22773752&amp;doi=10.1105/tpc.112.098152&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Uemura T, Kim H, Saito C, Ebine K, Ueda T, Schulze-Lefert P, Nakano A. Qa-SNAREs localized to the trans-Golgi network regulate multiple transport pathways and extracellular disease resistance in plants. Proc Natl Acad Sci U S A. 2012;109(5):1784–1789. doi: 10.1073/pnas.1115146109.</cite> [<a href="https://doi.org/10.1073/pnas.1115146109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3277133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22307646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Qa-SNAREs%20localized%20to%20the%20trans-Golgi%20network%20regulate%20multiple%20transport%20pathways%20and%20extracellular%20disease%20resistance%20in%20plants&amp;author=T%20Uemura&amp;author=H%20Kim&amp;author=C%20Saito&amp;author=K%20Ebine&amp;author=T%20Ueda&amp;volume=109&amp;issue=5&amp;publication_year=2012&amp;pages=1784-1789&amp;pmid=22307646&amp;doi=10.1073/pnas.1115146109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Kim SJ, Brandizzi F. News and Views into the SNARE Complexity in Arabidopsis. Front Plant Sci. 2012;3:28. doi: 10.3389/fpls.2012.00028.</cite> [<a href="https://doi.org/10.3389/fpls.2012.00028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3355637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23018380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Plant%20Sci&amp;title=News%20and%20Views%20into%20the%20SNARE%20Complexity%20in%20Arabidopsis&amp;author=SJ%20Kim&amp;author=F%20Brandizzi&amp;volume=3&amp;publication_year=2012&amp;pages=28&amp;pmid=23018380&amp;doi=10.3389/fpls.2012.00028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Bassham DC, Sanderfoot AA, Kovaleva V, Zheng H, Raikhel NV. AtVPS45 complex formation at the trans-Golgi network. Mol Biol Cell. 2000;11(7):2251–2265. doi: 10.1091/mbc.11.7.2251.</cite> [<a href="https://doi.org/10.1091/mbc.11.7.2251" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC14917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10888666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Cell&amp;title=AtVPS45%20complex%20formation%20at%20the%20trans-Golgi%20network&amp;author=DC%20Bassham&amp;author=AA%20Sanderfoot&amp;author=V%20Kovaleva&amp;author=H%20Zheng&amp;author=NV%20Raikhel&amp;volume=11&amp;issue=7&amp;publication_year=2000&amp;pages=2251-2265&amp;pmid=10888666&amp;doi=10.1091/mbc.11.7.2251&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Zhu J, Gong Z, Zhang C, Song CP, Damsz B, Inan G, Koiwa H, Zhu JK, Hasegawa PM, Bressan RA. OSM1/SYP61: a syntaxin protein in Arabidopsis controls abscisic acid-mediated and non-abscisic acid-mediated responses to abiotic stress. Plant Cell. 2002;14(12):3009–3028. doi: 10.1105/tpc.006981.</cite> [<a href="https://doi.org/10.1105/tpc.006981" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC151199/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12468724/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=OSM1/SYP61:%20a%20syntaxin%20protein%20in%20Arabidopsis%20controls%20abscisic%20acid-mediated%20and%20non-abscisic%20acid-mediated%20responses%20to%20abiotic%20stress&amp;author=J%20Zhu&amp;author=Z%20Gong&amp;author=C%20Zhang&amp;author=CP%20Song&amp;author=B%20Damsz&amp;volume=14&amp;issue=12&amp;publication_year=2002&amp;pages=3009-3028&amp;pmid=12468724&amp;doi=10.1105/tpc.006981&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Drakakaki G, van de Ven W, Pan S, Miao Y, Wang J, Keinath NF, Weatherly B, Jiang L, Schumacher K, Hicks G, et al.  Isolation and proteomic analysis of the SYP61 compartment reveal its role in exocytic trafficking in Arabidopsis. Cell Res. 2012;22(2):413–424. doi: 10.1038/cr.2011.129.</cite> [<a href="https://doi.org/10.1038/cr.2011.129" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3271593/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21826108/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Res&amp;title=Isolation%20and%20proteomic%20analysis%20of%20the%20SYP61%20compartment%20reveal%20its%20role%20in%20exocytic%20trafficking%20in%20Arabidopsis&amp;author=G%20Drakakaki&amp;author=W%20van%20de%20Ven&amp;author=S%20Pan&amp;author=Y%20Miao&amp;author=J%20Wang&amp;volume=22&amp;issue=2&amp;publication_year=2012&amp;pages=413-424&amp;pmid=21826108&amp;doi=10.1038/cr.2011.129&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Kim SJ, Bassham DC. TNO1 is involved in salt tolerance and vacuolar trafficking in Arabidopsis. Plant Physiol. 2011;156(2):514–526. doi: 10.1104/pp.110.168963.</cite> [<a href="https://doi.org/10.1104/pp.110.168963" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3177255/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21521696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=TNO1%20is%20involved%20in%20salt%20tolerance%20and%20vacuolar%20trafficking%20in%20Arabidopsis&amp;author=SJ%20Kim&amp;author=DC%20Bassham&amp;volume=156&amp;issue=2&amp;publication_year=2011&amp;pages=514-526&amp;pmid=21521696&amp;doi=10.1104/pp.110.168963&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Roy R, Bassham DC. Gravitropism and Lateral Root Emergence are Dependent on the Trans-Golgi Network Protein TNO1. Front Plant Sci. 2015;6:969. doi: 10.3389/fpls.2015.00969.</cite> [<a href="https://doi.org/10.3389/fpls.2015.00969" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4642138/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26617617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Plant%20Sci&amp;title=Gravitropism%20and%20Lateral%20Root%20Emergence%20are%20Dependent%20on%20the%20Trans-Golgi%20Network%20Protein%20TNO1&amp;author=R%20Roy&amp;author=DC%20Bassham&amp;volume=6&amp;publication_year=2015&amp;pages=969&amp;pmid=26617617&amp;doi=10.3389/fpls.2015.00969&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Rutherford R, Gallois P, Masson PH. Mutations in Arabidopsis thaliana genes involved in the tryptophan biosynthesis pathway affect root waving on tilted agar surfaces. Plant J. 1998;16(2):145–154. doi: 10.1046/j.1365-313x.1998.00279.x.</cite> [<a href="https://doi.org/10.1046/j.1365-313x.1998.00279.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9839461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=Mutations%20in%20Arabidopsis%20thaliana%20genes%20involved%20in%20the%20tryptophan%20biosynthesis%20pathway%20affect%20root%20waving%20on%20tilted%20agar%20surfaces&amp;author=R%20Rutherford&amp;author=P%20Gallois&amp;author=PH%20Masson&amp;volume=16&amp;issue=2&amp;publication_year=1998&amp;pages=145-154&amp;pmid=9839461&amp;doi=10.1046/j.1365-313x.1998.00279.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Schneider CA, Rasband WS, Eliceiri KW. NIH Image to ImageJ: 25 years of image analysis. Nat Methods. 2012;9(7):671–675. doi: 10.1038/nmeth.2089.</cite> [<a href="https://doi.org/10.1038/nmeth.2089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5554542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=NIH%20Image%20to%20ImageJ:%2025%20years%20of%20image%20analysis&amp;author=CA%20Schneider&amp;author=WS%20Rasband&amp;author=KW%20Eliceiri&amp;volume=9&amp;issue=7&amp;publication_year=2012&amp;pages=671-675&amp;pmid=22930834&amp;doi=10.1038/nmeth.2089&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Grabov A, Ashley MK, Rigas S, Hatzopoulos P, Dolan L, Vicente-Agullo F. Morphometric analysis of root shape. New Phytol. 2005;165(2):641–651. doi: 10.1111/j.1469-8137.2004.01258.x.</cite> [<a href="https://doi.org/10.1111/j.1469-8137.2004.01258.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15720674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol&amp;title=Morphometric%20analysis%20of%20root%20shape&amp;author=A%20Grabov&amp;author=MK%20Ashley&amp;author=S%20Rigas&amp;author=P%20Hatzopoulos&amp;author=L%20Dolan&amp;volume=165&amp;issue=2&amp;publication_year=2005&amp;pages=641-651&amp;pmid=15720674&amp;doi=10.1111/j.1469-8137.2004.01258.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Mathur J, Koncz C. Method for preparation of epidermal imprints using agarose. BioTechniques. 1997;22(2):280–282. doi: 10.2144/97222bm19.</cite> [<a href="https://doi.org/10.2144/97222bm19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9043700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BioTechniques&amp;title=Method%20for%20preparation%20of%20epidermal%20imprints%20using%20agarose&amp;author=J%20Mathur&amp;author=C%20Koncz&amp;volume=22&amp;issue=2&amp;publication_year=1997&amp;pages=280-282&amp;pmid=9043700&amp;doi=10.2144/97222bm19&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Mochizuki S, Harada A, Inada S, Sugimoto-Shirasu K, Stacey N, Wada T, Ishiguro S, Okada K, Sakai T. The Arabidopsis WAVY GROWTH 2 protein modulates root bending in response to environmental stimuli. Plant Cell. 2005;17(2):537–547. doi: 10.1105/tpc.104.028530.</cite> [<a href="https://doi.org/10.1105/tpc.104.028530" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC548824/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15659627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20Arabidopsis%20WAVY%20GROWTH%202%20protein%20modulates%20root%20bending%20in%20response%20to%20environmental%20stimuli&amp;author=S%20Mochizuki&amp;author=A%20Harada&amp;author=S%20Inada&amp;author=K%20Sugimoto-Shirasu&amp;author=N%20Stacey&amp;volume=17&amp;issue=2&amp;publication_year=2005&amp;pages=537-547&amp;pmid=15659627&amp;doi=10.1105/tpc.104.028530&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Sugimoto K, Williamson RE, Wasteneys GO. New techniques enable comparative analysis of microtubule orientation, wall texture, and growth rate in intact roots of Arabidopsis. Plant Physiol. 2000;124(4):1493–1506. doi: 10.1104/pp.124.4.1493.</cite> [<a href="https://doi.org/10.1104/pp.124.4.1493" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1539303/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11115865/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=New%20techniques%20enable%20comparative%20analysis%20of%20microtubule%20orientation,%20wall%20texture,%20and%20growth%20rate%20in%20intact%20roots%20of%20Arabidopsis&amp;author=K%20Sugimoto&amp;author=RE%20Williamson&amp;author=GO%20Wasteneys&amp;volume=124&amp;issue=4&amp;publication_year=2000&amp;pages=1493-1506&amp;pmid=11115865&amp;doi=10.1104/pp.124.4.1493&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Jacques E, Buytaert J, Wells DM, Lewandowski M, Bennett MJ, Dirckx J, Verbelen JP, Vissenberg K. MicroFilament Analyzer, an image analysis tool for quantifying fibrillar orientation, reveals changes in microtubule organization during gravitropism. Plant J. 2013;74(6):1045–1058. doi: 10.1111/tpj.12174.</cite> [<a href="https://doi.org/10.1111/tpj.12174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23489480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=MicroFilament%20Analyzer,%20an%20image%20analysis%20tool%20for%20quantifying%20fibrillar%20orientation,%20reveals%20changes%20in%20microtubule%20organization%20during%20gravitropism&amp;author=E%20Jacques&amp;author=J%20Buytaert&amp;author=DM%20Wells&amp;author=M%20Lewandowski&amp;author=MJ%20Bennett&amp;volume=74&amp;issue=6&amp;publication_year=2013&amp;pages=1045-1058&amp;pmid=23489480&amp;doi=10.1111/tpj.12174&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Migliaccio F, Tassone P, Fortunati A. Circumnutation as an autonomous root movement in plants. Am J Bot. 2013;100(1):4–13. doi: 10.3732/ajb.1200314.</cite> [<a href="https://doi.org/10.3732/ajb.1200314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23243099/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am%20J%20Bot&amp;title=Circumnutation%20as%20an%20autonomous%20root%20movement%20in%20plants&amp;author=F%20Migliaccio&amp;author=P%20Tassone&amp;author=A%20Fortunati&amp;volume=100&amp;issue=1&amp;publication_year=2013&amp;pages=4-13&amp;pmid=23243099&amp;doi=10.3732/ajb.1200314&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Wasteneys G, Collings D. Expanding beyond the great divide: Expanding beyond the great divide: the cytoskeleton and axial growth. In: Hussey PJ, editor. Oxford: Blackwell Publishing Ltd; 2004: p. 83–115.</cite>
</li>
<li id="CR46">
<span class="label">46.</span><cite>Fisher DD, Cyr RJ. Extending the Microtubule/Microfibril paradigm. Cellulose synthesis is required for normal cortical microtubule alignment in elongating cells. Plant Physiol. 1998;116(3):1043–1051. doi: 10.1104/pp.116.3.1043.</cite> [<a href="https://doi.org/10.1104/pp.116.3.1043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC35074/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9501137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Extending%20the%20Microtubule/Microfibril%20paradigm.%20Cellulose%20synthesis%20is%20required%20for%20normal%20cortical%20microtubule%20alignment%20in%20elongating%20cells&amp;author=DD%20Fisher&amp;author=RJ%20Cyr&amp;volume=116&amp;issue=3&amp;publication_year=1998&amp;pages=1043-1051&amp;pmid=9501137&amp;doi=10.1104/pp.116.3.1043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Wasteneys GO. Progress in understanding the role of microtubules in plant cells. Curr Opin Plant Biol. 2004;7(6):651–660. doi: 10.1016/j.pbi.2004.09.008.</cite> [<a href="https://doi.org/10.1016/j.pbi.2004.09.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15491913/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Plant%20Biol&amp;title=Progress%20in%20understanding%20the%20role%20of%20microtubules%20in%20plant%20cells&amp;author=GO%20Wasteneys&amp;volume=7&amp;issue=6&amp;publication_year=2004&amp;pages=651-660&amp;pmid=15491913&amp;doi=10.1016/j.pbi.2004.09.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Buschmann H, Fabri CO, Hauptmann M, Hutzler P, Laux T, Lloyd CW, Schaffner AR. Helical growth of the Arabidopsis mutant tortifolia1 reveals a plant-specific microtubule-associated protein. Curr Biol. 2004;14(16):1515–1521. doi: 10.1016/j.cub.2004.08.033.</cite> [<a href="https://doi.org/10.1016/j.cub.2004.08.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15324671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&amp;title=Helical%20growth%20of%20the%20Arabidopsis%20mutant%20tortifolia1%20reveals%20a%20plant-specific%20microtubule-associated%20protein&amp;author=H%20Buschmann&amp;author=CO%20Fabri&amp;author=M%20Hauptmann&amp;author=P%20Hutzler&amp;author=T%20Laux&amp;volume=14&amp;issue=16&amp;publication_year=2004&amp;pages=1515-1521&amp;pmid=15324671&amp;doi=10.1016/j.cub.2004.08.033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Sedbrook JC. MAPs in plant cells: delineating microtubule growth dynamics and organization. Curr Opin Plant Biol. 2004;7(6):632–640. doi: 10.1016/j.pbi.2004.09.017.</cite> [<a href="https://doi.org/10.1016/j.pbi.2004.09.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15491911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Plant%20Biol&amp;title=MAPs%20in%20plant%20cells:%20delineating%20microtubule%20growth%20dynamics%20and%20organization&amp;author=JC%20Sedbrook&amp;volume=7&amp;issue=6&amp;publication_year=2004&amp;pages=632-640&amp;pmid=15491911&amp;doi=10.1016/j.pbi.2004.09.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Moschou PN, Gutierrez-Beltran E, Bozhkov PV, Smertenko A. Separase Promotes Microtubule Polymerization by Activating CENP-E-Related Kinesin Kin7. Dev Cell. 2016;37(4):350–361. doi: 10.1016/j.devcel.2016.04.015.</cite> [<a href="https://doi.org/10.1016/j.devcel.2016.04.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27219063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev%20Cell&amp;title=Separase%20Promotes%20Microtubule%20Polymerization%20by%20Activating%20CENP-E-Related%20Kinesin%20Kin7&amp;author=PN%20Moschou&amp;author=E%20Gutierrez-Beltran&amp;author=PV%20Bozhkov&amp;author=A%20Smertenko&amp;volume=37&amp;issue=4&amp;publication_year=2016&amp;pages=350-361&amp;pmid=27219063&amp;doi=10.1016/j.devcel.2016.04.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Field JJ, Diaz JF, Miller JH. The binding sites of microtubule-stabilizing agents. Chem Biol. 2013;20(3):301–315. doi: 10.1016/j.chembiol.2013.01.014.</cite> [<a href="https://doi.org/10.1016/j.chembiol.2013.01.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23521789/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem%20Biol&amp;title=The%20binding%20sites%20of%20microtubule-stabilizing%20agents&amp;author=JJ%20Field&amp;author=JF%20Diaz&amp;author=JH%20Miller&amp;volume=20&amp;issue=3&amp;publication_year=2013&amp;pages=301-315&amp;pmid=23521789&amp;doi=10.1016/j.chembiol.2013.01.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Gendreau E, Traas J, Desnos T, Grandjean O, Caboche M, Hofte H. Cellular basis of hypocotyl growth in Arabidopsis thaliana. Plant Physiol. 1997;114(1):295–305. doi: 10.1104/pp.114.1.295.</cite> [<a href="https://doi.org/10.1104/pp.114.1.295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC158305/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9159952/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Cellular%20basis%20of%20hypocotyl%20growth%20in%20Arabidopsis%20thaliana&amp;author=E%20Gendreau&amp;author=J%20Traas&amp;author=T%20Desnos&amp;author=O%20Grandjean&amp;author=M%20Caboche&amp;volume=114&amp;issue=1&amp;publication_year=1997&amp;pages=295-305&amp;pmid=9159952&amp;doi=10.1104/pp.114.1.295&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Nakamura M, Naoi K, Shoji T, Hashimoto T. Low concentrations of propyzamide and oryzalin alter microtubule dynamics in Arabidopsis epidermal cells. Plant Cell Physiol. 2004;45(9):1330–1334. doi: 10.1093/pcp/pch300.</cite> [<a href="https://doi.org/10.1093/pcp/pch300" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15509858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol&amp;title=Low%20concentrations%20of%20propyzamide%20and%20oryzalin%20alter%20microtubule%20dynamics%20in%20Arabidopsis%20epidermal%20cells&amp;author=M%20Nakamura&amp;author=K%20Naoi&amp;author=T%20Shoji&amp;author=T%20Hashimoto&amp;volume=45&amp;issue=9&amp;publication_year=2004&amp;pages=1330-1334&amp;pmid=15509858&amp;doi=10.1093/pcp/pch300&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Sedbrook JC, Ehrhardt DW, Fisher SE, Scheible WR, Somerville CR. The Arabidopsis sku6/spiral1 gene encodes a plus end-localized microtubule-interacting protein involved in directional cell expansion. Plant Cell. 2004;16(6):1506–1520. doi: 10.1105/tpc.020644.</cite> [<a href="https://doi.org/10.1105/tpc.020644" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC490042/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15155883/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20Arabidopsis%20sku6/spiral1%20gene%20encodes%20a%20plus%20end-localized%20microtubule-interacting%20protein%20involved%20in%20directional%20cell%20expansion&amp;author=JC%20Sedbrook&amp;author=DW%20Ehrhardt&amp;author=SE%20Fisher&amp;author=WR%20Scheible&amp;author=CR%20Somerville&amp;volume=16&amp;issue=6&amp;publication_year=2004&amp;pages=1506-1520&amp;pmid=15155883&amp;doi=10.1105/tpc.020644&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Whittington AT, Vugrek O, Wei KJ, Hasenbein NG, Sugimoto K, Rashbrooke MC, Wasteneys GO. MOR1 is essential for organizing cortical microtubules in plants. Nature. 2001;411(6837):610–613. doi: 10.1038/35079128.</cite> [<a href="https://doi.org/10.1038/35079128" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11385579/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=MOR1%20is%20essential%20for%20organizing%20cortical%20microtubules%20in%20plants&amp;author=AT%20Whittington&amp;author=O%20Vugrek&amp;author=KJ%20Wei&amp;author=NG%20Hasenbein&amp;author=K%20Sugimoto&amp;volume=411&amp;issue=6837&amp;publication_year=2001&amp;pages=610-613&amp;pmid=11385579&amp;doi=10.1038/35079128&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Oliva M, Dunand C. Waving and skewing: how gravity and the surface of growth media affect root development in Arabidopsis. New Phytol. 2007;176(1):37–43. doi: 10.1111/j.1469-8137.2007.02184.x.</cite> [<a href="https://doi.org/10.1111/j.1469-8137.2007.02184.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17692076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol&amp;title=Waving%20and%20skewing:%20how%20gravity%20and%20the%20surface%20of%20growth%20media%20affect%20root%20development%20in%20Arabidopsis&amp;author=M%20Oliva&amp;author=C%20Dunand&amp;volume=176&amp;issue=1&amp;publication_year=2007&amp;pages=37-43&amp;pmid=17692076&amp;doi=10.1111/j.1469-8137.2007.02184.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Ishida T, Hashimoto T. An Arabidopsis thaliana tubulin mutant with conditional root-skewing phenotype. J Plant Res. 2007;120(5):635–640. doi: 10.1007/s10265-007-0105-0.</cite> [<a href="https://doi.org/10.1007/s10265-007-0105-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17641820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Plant%20Res&amp;title=An%20Arabidopsis%20thaliana%20tubulin%20mutant%20with%20conditional%20root-skewing%20phenotype&amp;author=T%20Ishida&amp;author=T%20Hashimoto&amp;volume=120&amp;issue=5&amp;publication_year=2007&amp;pages=635-640&amp;pmid=17641820&amp;doi=10.1007/s10265-007-0105-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Sugimoto K, Himmelspach R, Williamson RE, Wasteneys GO. Mutation or Drug-Dependent Microtubule Disruption Causes Radial Swelling without Altering Parallel Cellulose Microfibril Deposition in Arabidopsis Root Cells. Plant Cell. 2003;15(6):1414–1429. doi: 10.1105/tpc.011593.</cite> [<a href="https://doi.org/10.1105/tpc.011593" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC156376/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12782733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Mutation%20or%20Drug-Dependent%20Microtubule%20Disruption%20Causes%20Radial%20Swelling%20without%20Altering%20Parallel%20Cellulose%20Microfibril%20Deposition%20in%20Arabidopsis%20Root%20Cells&amp;author=K%20Sugimoto&amp;author=R%20Himmelspach&amp;author=RE%20Williamson&amp;author=GO%20Wasteneys&amp;volume=15&amp;issue=6&amp;publication_year=2003&amp;pages=1414-1429&amp;pmid=12782733&amp;doi=10.1105/tpc.011593&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Galva C, Kirik V, Lindeboom JJ, Kaloriti D, Rancour DM, Hussey PJ, Bednarek SY, Ehrhardt DW, Sedbrook JC. The microtubule plus-end tracking proteins SPR1 and EB1b interact to maintain polar cell elongation and directional organ growth in Arabidopsis. Plant Cell. 2014;26(11):4409–4425. doi: 10.1105/tpc.114.131482.</cite> [<a href="https://doi.org/10.1105/tpc.114.131482" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4277225/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25415978/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20microtubule%20plus-end%20tracking%20proteins%20SPR1%20and%20EB1b%20interact%20to%20maintain%20polar%20cell%20elongation%20and%20directional%20organ%20growth%20in%20Arabidopsis&amp;author=C%20Galva&amp;author=V%20Kirik&amp;author=JJ%20Lindeboom&amp;author=D%20Kaloriti&amp;author=DM%20Rancour&amp;volume=26&amp;issue=11&amp;publication_year=2014&amp;pages=4409-4425&amp;pmid=25415978&amp;doi=10.1105/tpc.114.131482&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Crowell EF, Bischoff V, Desprez T, Rolland A, Stierhof YD, Schumacher K, Gonneau M, Hofte H, Vernhettes S. Pausing of Golgi bodies on microtubules regulates secretion of cellulose synthase complexes in Arabidopsis. Plant Cell. 2009;21(4):1141–1154. doi: 10.1105/tpc.108.065334.</cite> [<a href="https://doi.org/10.1105/tpc.108.065334" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2685615/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19376932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Pausing%20of%20Golgi%20bodies%20on%20microtubules%20regulates%20secretion%20of%20cellulose%20synthase%20complexes%20in%20Arabidopsis&amp;author=EF%20Crowell&amp;author=V%20Bischoff&amp;author=T%20Desprez&amp;author=A%20Rolland&amp;author=YD%20Stierhof&amp;volume=21&amp;issue=4&amp;publication_year=2009&amp;pages=1141-1154&amp;pmid=19376932&amp;doi=10.1105/tpc.108.065334&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Gutierrez R, Lindeboom JJ, Paredez AR, Emons AM, Ehrhardt DW. Arabidopsis cortical microtubules position cellulose synthase delivery to the plasma membrane and interact with cellulose synthase trafficking compartments. Nat Cell Biol. 2009;11(7):797–806. doi: 10.1038/ncb1886.</cite> [<a href="https://doi.org/10.1038/ncb1886" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19525940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Cell%20Biol&amp;title=Arabidopsis%20cortical%20microtubules%20position%20cellulose%20synthase%20delivery%20to%20the%20plasma%20membrane%20and%20interact%20with%20cellulose%20synthase%20trafficking%20compartments&amp;author=R%20Gutierrez&amp;author=JJ%20Lindeboom&amp;author=AR%20Paredez&amp;author=AM%20Emons&amp;author=DW%20Ehrhardt&amp;volume=11&amp;issue=7&amp;publication_year=2009&amp;pages=797-806&amp;pmid=19525940&amp;doi=10.1038/ncb1886&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Lei L, Li S, Du J, Bashline L, Gu Y. Cellulose synthase INTERACTIVE3 regulates cellulose biosynthesis in both a microtubule-dependent and microtubule-independent manner in Arabidopsis. Plant Cell. 2013;25(12):4912–4923. doi: 10.1105/tpc.113.116715.</cite> [<a href="https://doi.org/10.1105/tpc.113.116715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3903995/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24368796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Cellulose%20synthase%20INTERACTIVE3%20regulates%20cellulose%20biosynthesis%20in%20both%20a%20microtubule-dependent%20and%20microtubule-independent%20manner%20in%20Arabidopsis&amp;author=L%20Lei&amp;author=S%20Li&amp;author=J%20Du&amp;author=L%20Bashline&amp;author=Y%20Gu&amp;volume=25&amp;issue=12&amp;publication_year=2013&amp;pages=4912-4923&amp;pmid=24368796&amp;doi=10.1105/tpc.113.116715&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Bashline L, Li S, Gu Y. The trafficking of the cellulose synthase complex in higher plants. Ann Bot. 2014;114(6):1059–1067. doi: 10.1093/aob/mcu040.</cite> [<a href="https://doi.org/10.1093/aob/mcu040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4195546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24651373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann%20Bot&amp;title=The%20trafficking%20of%20the%20cellulose%20synthase%20complex%20in%20higher%20plants&amp;author=L%20Bashline&amp;author=S%20Li&amp;author=Y%20Gu&amp;volume=114&amp;issue=6&amp;publication_year=2014&amp;pages=1059-1067&amp;pmid=24651373&amp;doi=10.1093/aob/mcu040&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Bashline L, Lei L, Li S, Gu Y. Cell wall, cytoskeleton, and cell expansion in higher plants. Mol Plant. 2014;7(4):586–600. doi: 10.1093/mp/ssu018.</cite> [<a href="https://doi.org/10.1093/mp/ssu018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24557922/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Plant&amp;title=Cell%20wall,%20cytoskeleton,%20and%20cell%20expansion%20in%20higher%20plants&amp;author=L%20Bashline&amp;author=L%20Lei&amp;author=S%20Li&amp;author=Y%20Gu&amp;volume=7&amp;issue=4&amp;publication_year=2014&amp;pages=586-600&amp;pmid=24557922&amp;doi=10.1093/mp/ssu018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Lei L, Singh A, Bashline L, Li S, Yingling YG, Gu Y. CELLULOSE SYNTHASE INTERACTIVE1 Is Required for Fast Recycling of Cellulose Synthase Complexes to the Plasma Membrane in Arabidopsis. Plant Cell. 2015;27(10):2926–2940. doi: 10.1105/tpc.15.00442.</cite> [<a href="https://doi.org/10.1105/tpc.15.00442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4682321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26443667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=CELLULOSE%20SYNTHASE%20INTERACTIVE1%20Is%20Required%20for%20Fast%20Recycling%20of%20Cellulose%20Synthase%20Complexes%20to%20the%20Plasma%20Membrane%20in%20Arabidopsis&amp;author=L%20Lei&amp;author=A%20Singh&amp;author=L%20Bashline&amp;author=S%20Li&amp;author=YG%20Yingling&amp;volume=27&amp;issue=10&amp;publication_year=2015&amp;pages=2926-2940&amp;pmid=26443667&amp;doi=10.1105/tpc.15.00442&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Robert S, Bichet A, Grandjean O, Kierzkowski D, Satiat-Jeunemaitre B, Pelletier S, Hauser MT, Hofte H, Vernhettes S. An Arabidopsis endo-1,4-beta-D-glucanase involved in cellulose synthesis undergoes regulated intracellular cycling. Plant Cell. 2005;17(12):3378–3389. doi: 10.1105/tpc.105.036228.</cite> [<a href="https://doi.org/10.1105/tpc.105.036228" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1315376/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16284310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=An%20Arabidopsis%20endo-1,4-beta-D-glucanase%20involved%20in%20cellulose%20synthesis%20undergoes%20regulated%20intracellular%20cycling&amp;author=S%20Robert&amp;author=A%20Bichet&amp;author=O%20Grandjean&amp;author=D%20Kierzkowski&amp;author=B%20Satiat-Jeunemaitre&amp;volume=17&amp;issue=12&amp;publication_year=2005&amp;pages=3378-3389&amp;pmid=16284310&amp;doi=10.1105/tpc.105.036228&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Li S, Bashline L, Lei L, Gu Y. Cellulose synthesis and its regulation. Arabidopsis Book. 2014;12:e0169. doi: 10.1199/tab.0169.</cite> [<a href="https://doi.org/10.1199/tab.0169" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3894906/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24465174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Arabidopsis%20Book&amp;title=Cellulose%20synthesis%20and%20its%20regulation&amp;author=S%20Li&amp;author=L%20Bashline&amp;author=L%20Lei&amp;author=Y%20Gu&amp;volume=12&amp;publication_year=2014&amp;pages=e0169&amp;pmid=24465174&amp;doi=10.1199/tab.0169&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Worden N, Wilkop TE, Esteve VE, Jeannotte R, Lathe R, Vernhettes S, Weimer B, Hicks G, Alonso J, Labavitch J, et al.  CESA TRAFFICKING INHIBITOR inhibits cellulose deposition and interferes with the trafficking of cellulose synthase complexes and their associated proteins KORRIGAN1 and POM2/CELLULOSE SYNTHASE INTERACTIVE PROTEIN1. Plant Physiol. 2015;167(2):381–393. doi: 10.1104/pp.114.249003.</cite> [<a href="https://doi.org/10.1104/pp.114.249003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4326758/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25535279/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=CESA%20TRAFFICKING%20INHIBITOR%20inhibits%20cellulose%20deposition%20and%20interferes%20with%20the%20trafficking%20of%20cellulose%20synthase%20complexes%20and%20their%20associated%20proteins%20KORRIGAN1%20and%20POM2/CELLULOSE%20SYNTHASE%20INTERACTIVE%20PROTEIN1&amp;author=N%20Worden&amp;author=TE%20Wilkop&amp;author=VE%20Esteve&amp;author=R%20Jeannotte&amp;author=R%20Lathe&amp;volume=167&amp;issue=2&amp;publication_year=2015&amp;pages=381-393&amp;pmid=25535279&amp;doi=10.1104/pp.114.249003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Burk DH, Ye ZH. Alteration of oriented deposition of cellulose microfibrils by mutation of a katanin-like microtubule-severing protein. Plant Cell. 2002;14(9):2145–2160. doi: 10.1105/tpc.003947.</cite> [<a href="https://doi.org/10.1105/tpc.003947" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC150762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12215512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Alteration%20of%20oriented%20deposition%20of%20cellulose%20microfibrils%20by%20mutation%20of%20a%20katanin-like%20microtubule-severing%20protein&amp;author=DH%20Burk&amp;author=ZH%20Ye&amp;volume=14&amp;issue=9&amp;publication_year=2002&amp;pages=2145-2160&amp;pmid=12215512&amp;doi=10.1105/tpc.003947&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Paredez AR, Somerville CR, Ehrhardt DW. Visualization of Cellulose Synthase Demonstrates Functional Association with Microtubules. Science. 2006;312(5779):1491–1495. doi: 10.1126/science.1126551.</cite> [<a href="https://doi.org/10.1126/science.1126551" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16627697/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Visualization%20of%20Cellulose%20Synthase%20Demonstrates%20Functional%20Association%20with%20Microtubules&amp;author=AR%20Paredez&amp;author=CR%20Somerville&amp;author=DW%20Ehrhardt&amp;volume=312&amp;issue=5779&amp;publication_year=2006&amp;pages=1491-1495&amp;pmid=16627697&amp;doi=10.1126/science.1126551&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Sugimoto K, Williamson RE, Wasteneys GO. Wall architecture in the cellulose-deficientrsw1 mutant Ofarabidopsis thaliana: Microfibrils but not microtubules lose their transverse alignment before microfibrils become unrecognizable in the mitotic and elongation zones of roots. Protoplasma. 2001;215(1):172–183. doi: 10.1007/BF01280312.</cite> [<a href="https://doi.org/10.1007/BF01280312" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11732056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protoplasma&amp;title=Wall%20architecture%20in%20the%20cellulose-deficientrsw1%20mutant%20Ofarabidopsis%20thaliana:%20Microfibrils%20but%20not%20microtubules%20lose%20their%20transverse%20alignment%20before%20microfibrils%20become%20unrecognizable%20in%20the%20mitotic%20and%20elongation%20zones%20of%20roots&amp;author=K%20Sugimoto&amp;author=RE%20Williamson&amp;author=GO%20Wasteneys&amp;volume=215&amp;issue=1&amp;publication_year=2001&amp;pages=172-183&amp;pmid=11732056&amp;doi=10.1007/BF01280312&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Fujita M, Himmelspach R, Ward J, Whittington A, Hasenbein N, Liu C, Truong TT, Galway ME, Mansfield SD, Hocart CH, et al.  The anisotropy1 D604N mutation in the Arabidopsis cellulose synthase1 catalytic domain reduces cell wall crystallinity and the velocity of cellulose synthase complexes. Plant Physiol. 2013;162(1):74–85. doi: 10.1104/pp.112.211565.</cite> [<a href="https://doi.org/10.1104/pp.112.211565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3641231/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23532584/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=The%20anisotropy1%20D604N%20mutation%20in%20the%20Arabidopsis%20cellulose%20synthase1%20catalytic%20domain%20reduces%20cell%20wall%20crystallinity%20and%20the%20velocity%20of%20cellulose%20synthase%20complexes&amp;author=M%20Fujita&amp;author=R%20Himmelspach&amp;author=J%20Ward&amp;author=A%20Whittington&amp;author=N%20Hasenbein&amp;volume=162&amp;issue=1&amp;publication_year=2013&amp;pages=74-85&amp;pmid=23532584&amp;doi=10.1104/pp.112.211565&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Buschmann H, Hauptmann M, Niessing D, Lloyd CW, Schäffner AR. Helical growth of the Arabidopsis mutant tortifolia2 does not depend on cell division patterns but involves handed twisting of isolated cells. Plant Cell. 2009;21(7):2090–2106. doi: 10.1105/tpc.108.061242.</cite> [<a href="https://doi.org/10.1105/tpc.108.061242" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2729594/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19638477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Helical%20growth%20of%20the%20Arabidopsis%20mutant%20tortifolia2%20does%20not%20depend%20on%20cell%20division%20patterns%20but%20involves%20handed%20twisting%20of%20isolated%20cells&amp;author=H%20Buschmann&amp;author=M%20Hauptmann&amp;author=D%20Niessing&amp;author=CW%20Lloyd&amp;author=AR%20Sch%C3%A4ffner&amp;volume=21&amp;issue=7&amp;publication_year=2009&amp;pages=2090-2106&amp;pmid=19638477&amp;doi=10.1105/tpc.108.061242&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data generated or analyzed during this study are included in this published article. Raw files are available upon request from Dr. Rahul Roy (rroy@umn.edu).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Plant Biology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12870-017-1024-4"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12870_2017_Article_1024.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/5387210/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/5387210/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC5387210%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC5387210/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC5387210/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC5387210/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/28399805/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC5387210/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/28399805/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC5387210/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/5387210/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="crKBLjcvyEYnStbvS4sBDnBW5Zaswcszn68KtgvH64HjxurN01KnQ6BTcN3sBbpa">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
