
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            FAIRness and Usability for Open-access Omics Data Systems - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="8E591B7F8E1630D3141B7F000337EC9C.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="procamia">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC6371294/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="AMIA Annual Symposium Proceedings">
<meta name="citation_title" content="FAIRness and Usability for Open-access Omics Data Systems">
<meta name="citation_author" content="Daniel C Berrios">
<meta name="citation_author_institution" content="Universities Space Research Association;">
<meta name="citation_author_institution" content="NASA Ames Research Center, Moffett Field, CA">
<meta name="citation_author" content="Afshin Beheshti">
<meta name="citation_author_institution" content="KBRwyle;">
<meta name="citation_author_institution" content="NASA Ames Research Center, Moffett Field, CA">
<meta name="citation_author" content="Sylvain V Costes">
<meta name="citation_author_institution" content="NASA">
<meta name="citation_author_institution" content="NASA Ames Research Center, Moffett Field, CA">
<meta name="citation_publication_date" content="2018 Dec 5">
<meta name="citation_volume" content="2018">
<meta name="citation_firstpage" content="232">
<meta name="citation_pmid" content="30815061">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC6371294/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC6371294/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC6371294/pdf/2975810.pdf">
<meta name="description" content="Omics data sharing is crucial to the biological research community, and the last decade or two has seen a huge rise in collaborative analysis systems, databases, and knowledge bases for omics and other systems biology data. We assessed the ...">
<meta name="og:title" content="FAIRness and Usability for Open-access Omics Data Systems">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Omics data sharing is crucial to the biological research community, and the last decade or two has seen a huge rise in collaborative analysis systems, databases, and knowledge bases for omics and other systems biology data. We assessed the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC6371294/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://pmc.ncbi.nlm.nih.gov/search/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="6371294">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/2975810.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC6371294%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/6371294/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/6371294/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC6371294/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-procamia.gif" alt="AMIA Annual Symposium Proceedings logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to AMIA Annual Symposium Proceedings" title="Link to AMIA Annual Symposium Proceedings" shape="default" href="https://knowledge.amia.org/all-proceedings" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">AMIA Annu Symp Proc</button></div>. 2018 Dec 5;2018:232–241.</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://pmc.ncbi.nlm.nih.gov/search/?term=%22AMIA%20Annu%20Symp%20Proc%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22AMIA%20Annu%20Symp%20Proc%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22AMIA%20Annu%20Symp%20Proc%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22AMIA%20Annu%20Symp%20Proc%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>FAIRness and Usability for Open-access Omics Data Systems</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Berrios%20DC%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Daniel C Berrios</span></a><div hidden="hidden" id="id1">
<h3>
<span class="name western">Daniel C Berrios</span>, <span class="degrees">MD, MPH, PhD</span>
</h3>
<div class="p">
<sup>1</sup>Universities Space Research Association;</div>
<div class="p">
<sup>4</sup>NASA Ames Research Center, Moffett Field, CA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Berrios%20DC%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniel C Berrios</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Beheshti%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Afshin Beheshti</span></a><div hidden="hidden" id="id2">
<h3>
<span class="name western">Afshin Beheshti</span>, <span class="degrees">PhD</span>
</h3>
<div class="p">
<sup>2</sup>KBRwyle;</div>
<div class="p">
<sup>4</sup>NASA Ames Research Center, Moffett Field, CA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Beheshti%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Afshin Beheshti</span></a>
</div>
</div>
<sup>2,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Costes%20SV%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Sylvain V Costes</span></a><div hidden="hidden" id="id3">
<h3>
<span class="name western">Sylvain V Costes</span>, <span class="degrees">PhD</span>
</h3>
<div class="p">
<sup>3</sup>NASA</div>
<div class="p">
<sup>4</sup>NASA Ames Research Center, Moffett Field, CA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Costes%20SV%22%5BAuthor%5D" class="usa-link"><span class="name western">Sylvain V Costes</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="af1-2975810">
<sup>1</sup>Universities Space Research Association;</div>
<div id="af2-2975810">
<sup>2</sup>KBRwyle;</div>
<div id="af3-2975810">
<sup>3</sup>NASA</div>
<div id="af4-2975810">
<sup>4</sup>NASA Ames Research Center, Moffett Field, CA</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Collection date 2018.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>©2018 AMIA - All rights reserved.</div>
<p>This is an Open Access article: verbatim copying and redistribution of this article are permitted in all media for any purpose</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC6371294  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/30815061/" class="usa-link">30815061</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Omics data sharing is crucial to the biological research community, and the last decade or two has seen a huge rise in collaborative analysis systems, databases, and knowledge bases for omics and other systems biology data. We assessed the “FAIRness” of NASA’s GeneLab Data Systems (GLDS) along with four similar kinds of systems in the research omics data domain, using 14 FAIRness metrics. The range of overall FAIRness scores was 6-12 (out of 14), average 10.1, and standard deviation 2.4. The range of Pass ratings for the metrics was 29-79%, Partial Pass 0-21%, and Fail 7-50%. The systems we evaluated performed the best in the areas of data findability and accessibility, and worst in the area of data interoperability. Reusability of metadata, in particular, was frequently not well supported. We relate our experiences implementing semantic integration of omics data from some of the assessed systems for federated querying and retrieval functions, given their shortcomings in data interoperability. Finally, we propose two new principles that Big Data system developers, in particular, should consider for maximizing data accessibility.</p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Over the last decade the vast increase in outputs from “omics” (genomics, transcriptomics, proteomics, and metabolomics) assays, together with decreasing assay and data storage costs, have resulted in an exponentially increasing volume of these “big” data. The high feature-number aspect of these big data has highlighted the importance of data sharing (or reuse) functions of these systems, as data sets often have relevance for investigating a very large number of hypotheses. Thus, developers of big data systems increasingly are focusing on the needs of users to discover, annotate, share, and analyze data. Omics data sharing is especially crucial to the biological research community, and the last decade or two has seen a huge rise in collaborative analysis systems<sup><a href="#r1-2975810" class="usa-link" aria-describedby="r1-2975810">1</a>-<a href="#r3-2975810" class="usa-link" aria-describedby="r3-2975810">3</a></sup>, databases<sup><a href="#r4-2975810" class="usa-link" aria-describedby="r4-2975810">4</a>-<a href="#r14-2975810" class="usa-link" aria-describedby="r14-2975810">14</a></sup>, and knowledge bases<sup><a href="#r15-2975810" class="usa-link" aria-describedby="r15-2975810">15</a>-<a href="#r17-2975810" class="usa-link" aria-describedby="r17-2975810">17</a></sup> for omics and other systems biology data.</p>
<p>The FAIR<sup><a href="#r18-2975810" class="usa-link" aria-describedby="r18-2975810">18</a></sup> principles are a set of guiding principles that are being advocated as foundational in the broad areas of data findability, accessibility, interoperability, and reusability. However, the challenges involved in using current technologies and real-world infrastructures within organizational guidelines to develop data repositories and/or data analytical systems that are “FAIR” are substantial. The FAIR principles deal with aspects of data handling such as identification, representation, and attribution. Metrics to evaluate the “FAIRness” of a data system or its design are also currently being developed<sup><a href="#r19-2975810" class="usa-link" aria-describedby="r19-2975810">19</a></sup>. System developers should consider familiarizing themselves with these metrics, as many of the FAIR principles can have significant implications in terms of feasible design options and operations resource planning.</p>
<p>We assessed the “FAIRness” of NASA’s GeneLab Data Systems (GLDS), along with four systems in the research omics data domain chosen for their similarity to the GLDS, using 14 FAIRness metrics. We relate these assessments in the context of discussion of the FAIRness concepts for the design of Big Data omics systems in particular, relating some of the challenges we faced developing the GLDS as a FAIR data system. In addition to designing a data system to be FAIR, developers of Big Data systems in particular should consider how system design can maximize data usability for its users. We propose two new design principles for FAIR systems that handle Big Data like omics data from our experiences developing the GLDS. By complying with these usability principles and the FAIR principles together, systems that manage Big Data can maximize both the usability and reusability of data.</p></section><section id="sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="sec2_1"><h3 class="pmc_sec_title">NASA GeneLab</h3>
<p>GeneLab (<a href="http://genelab.nasa.gov" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://genelab. nasa. gov</a>) is a NASA initiative designed to accelerate “open science” biomedical research in support of the human exploration of space and the improvement of life on earth.<sup><a href="#r20-2975810" class="usa-link" aria-describedby="r20-2975810">20</a></sup> GeneLab data have been used to research impacts of space on mice, humans, and plants. Phase I of GeneLab Data Systems (GLDS) project implemented an “omics” (genomics, transcriptomics proteomics, and metabolomics) data repository for biomedical research data conducted in or relevant to space environments (<a href="#f1-2975810" class="usa-link">Figure 1</a>). This initial phase developed processes and systems for data submission, curation, indexing, search, and retrieval. In Phase II GeneLab implemented federated search and retrieval capabilities for data hosted by GeneLab and three other open-access, omics data systems, in order to facilitate data integration and biological meta-investigation (<a href="#f1-2975810" class="usa-link">Figure 1</a>a).<sup><a href="#r21-2975810" class="usa-link" aria-describedby="r21-2975810">21</a></sup> (Note that these targets for search federation were selected independently and using different criteria than the data systems evaluated in this paper). Such meta-investigations are key to corroborating findings from different studies and/or kinds of bioassay data, and translating them into systems biology knowledge and, eventually, therapeutics<sup><a href="#r22-2975810" class="usa-link" aria-describedby="r22-2975810">22</a></sup>. In Phase III, GeneLab has begun to deploy an omics data analysis platform as part of the GLDS. Using this platform, investigators can design, execute and share in silico experimentation of omics data sets (<a href="#f1-2975810" class="usa-link">Figures 1</a>b and <a href="#f1-2975810" class="usa-link">1</a>c). At the end of Phase III, users with results of these kinds of experiments will be able to store them directly into the GLDS repository, linked to originating data and to the analysis workflows that generated them.</p>
<figure class="fig xbox font-sm" id="f1-2975810"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6371294_2975810f1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/876e/6371294/1368a790dfe7/2975810f1.jpg" loading="lazy" height="836" width="764" alt="Figure 1."></a></p>
<div class="p text-right font-secondary"><a href="figure/f1-2975810/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The GeneLab Data Systems (GLDS). (a) The GeneLab Data Repository. The federated search interface shown includes not only search of GLDS repository data, but of the extramural data sources Gene Expression Omnibus (“GEO”<sup><a href="#r4-2975810" class="usa-link" aria-describedby="r4-2975810">4</a></sup>), PRoteomics IDEntifications repository (“EBI PRIDE”<sup><a href="#r5-2975810" class="usa-link" aria-describedby="r5-2975810">5</a></sup>), and the Metagenomics Analysis server (“ANL MG-RAST”<sup><a href="#r6-2975810" class="usa-link" aria-describedby="r6-2975810">6</a></sup>). (b) The GLDS Workspace. (c) GLDS Tools for omics data analysis (part of a Galaxy server<sup><a href="#r1-2975810" class="usa-link" aria-describedby="r1-2975810">1</a></sup>).</p></figcaption></figure><p>Even though the inception of the GLDS predated publication of the FAIR principles, many of the concepts behind the principles were discussed extensively prior to its design. These discussions included substantial input from the National Academies of Sciences and a steering committee of outside experts in omics biological data, and had significant impacts on selection of technological approaches and system architectures, required resources for development and operations, and policies developed for operating the GLDS. In the absence of best practices from the community in certain areas of design and policy (<a href="#f2-2975810" class="usa-link">Figure 2</a>), we wondered how we might assess the impact on FAIRness of the design and policy choices we faced, and how the GLDS would compare in FAIRness to other open-access, Big Data systems.</p>
<figure class="fig xbox font-sm" id="f2-2975810"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6371294_2975810f2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/876e/6371294/402a007044f3/2975810f2.jpg" loading="lazy" height="496" width="769" alt="Figure 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/f2-2975810/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The FAIR principles<sup><a href="#r18-2975810" class="usa-link" aria-describedby="r18-2975810">18</a></sup>, and areas of design and operations of FAIR data systems (white boxes) for which best practices have yet to be defined.</p></figcaption></figure></section><section id="sec2_2"><h3 class="pmc_sec_title">FAIRness of Omics Data</h3>
<p>The FAIR principles were designed to guide the management of data so that it is maximally findable, accessible, interoperable, and reusable. They are not precise prescriptions for FAIR system design specifications; rather they are aspirational guidelines with some references to exemplar standards and technologies. Among the findability principles are requirements that data be assigned a persistent, global unique identifier (GUID), be described by rich metadata, and be indexed by a system for retrieving the data. Accessibility requires the use of standardized, open (available for implementation to all) communications protocols for retrieval of data by GUID. Interoperability is supported through the use of machine-actionable metadata and data. Reusability requ<strong>i</strong>res systems to be compliant in the management of their data objects with the first three principles, specifically so that data and metadata can be linked and integrated across FAIR systems.</p>
<p>Each of the authors assessed compliance of four open-access, omics data systems and the GLDS for FAIRness using the current working draft of the FAIRness metrics developed by the GO FAIR Metrics group<sup><a href="#r23-2975810" class="usa-link" aria-describedby="r23-2975810">23</a></sup> (Table 1). The four systems selected were the Gene Expression Omnibus (GEO), the European Nucleotide Archive (ENA), the Metagenomics Analysis Server (MG-RAST), and Metabolights. We chose the systems for their similarity to the GLDS; all are open-access, government-operated or government-funded, research lab-developed omics data systems. As the FAIR principles themselves are only aspirational, the assessment of the systems can only be at most semi-quantitative, and they would likely be affected by further clarifications or details in the principles and/or metrics which are almost certain to come. We rated systems with Pass when we had no evidence of any failure of a test by the metric. Systems were rated as Partial Pass, if the metric had multiple steps or components, and the system passed some, but not all; or if we had evidence that some data inputs for the metric yielded a pass, while others did not. We chose Fail only if we could find no evidence the system was compliant with any part of the metric, for any inputs we tried. Individually-assessed ratings were then combined into the consensus ratings shown in <a href="#t1-2975810" class="usa-link">Table 1</a> through dialogue among the raters. Finally, we developed an overall FAIRness Score by assigning a score to each rating of 1 for each Pass, 0.5 for each Partial Pass, and 0 for each Fail and totaling these score for each system.</p>
<section class="tw xbox font-sm" id="t1-2975810"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>The FAIR principles, corresponding draft FAIRness metrics, and semi-quantitative FAIRness ratings for select omics data systems. Metrics were those developed by the GO FAIR Metrics<sup><a href="#r23-2975810" class="usa-link" aria-describedby="r23-2975810">23</a></sup> group.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="box" rules="none"><tbody><tr><td valign="top" align="left" rowspan="1" colspan="1"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/876e/6371294/56f23a3d8fc1/2975810t1.jpg" loading="lazy" alt="graphic file with name 2975810t1.jpg"></td></tr></tbody></table></div>
<div class="p text-right font-secondary"><a href="table/t1-2975810/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec2_3"><h3 class="pmc_sec_title">Summary of FAIRness Assessments</h3>
<p>There is quite some variability in FAIRness among the omics data systems we assessed using the 14 metrics. The range of Pass ratings was 29-79% of metrics, Partial Pass 0-21%, and Fail 7-50%. The range of overall FAIRness scores was 6-12, with average of 10.1 and standard deviation, 2.4. All the systems were rated best in the areas of data accessibility and reusability, slightly worse in data findability, and far worse in data interoperability.</p></section><section id="sec2_4"><h3 class="pmc_sec_title">Findability</h3>
<p>The first findability principle, F1 (<a href="#t1-2975810" class="usa-link">Table 1</a>), states that systems should identify data using globally unique and persistent identifiers; the FM-F1A metric focuses on identifier uniqueness, and FM-F1B on identifier persistence. The objective is to make data findable, but the implementation of F1 also supports data accessibility (FAIR principle A1). The qualifier “global” should be interpreted as global within the domain of conceivable use of the data. Thus, it is not sufficient to use locally (system-specific) controlled processes to manage data identifiers<sup><a href="#r24-2975810" class="usa-link" aria-describedby="r24-2975810">24</a></sup>, as this approach can never absolutely guarantee against overlap with other systems’ local identifier schemes. In addition, F1 specifies the persistence of issued identifiers; identification systems that employ Uniform Resource Locators (which rely on alterable domain name resolution) cannot guarantee persistence of identifiers.</p>
<p>Among the schemes for providing data identifiers, the most reliable over time are likely to be those that use third-party resolution (versus those that rely on in-house redirection). Third-party resolution identifiers include Digital Object Identifiers (DOIs) and those provided by identifers.org, among others<sup><a href="#r25-2975810" class="usa-link" aria-describedby="r25-2975810">25</a></sup>. All but one of the systems we assessed passed the metric for F1 metric by employing persistent, globally unique identifiers with third party resolution. (MG-RAST only offers such identifiers as an option during data submission, and so was assessed as Partial Pass.) In the biomedical research and clinical practice realms, publishers have now widely selected the DOI scheme for identification and citation of literature; more than 80% of 2015 articles indexed by PubMed have assigned DOIs<sup><a href="#r26-2975810" class="usa-link" aria-describedby="r26-2975810">26</a></sup>. As a consequence, the biomedical community is already (and continues to become more and more) familiar with using these identifiers to retrieve and cite literature. Yet to date only a few omics data repositories have similarly implemented DOIs for data publications. Consequently, actual findability and accessibility of data in these repositories is less than optimal, even though they are technically in compliance with F1, and will continue to be so, unless and until the biomedical community is trained on how to use other identifying schemes for citing data. Furthermore, metadata from these repositories are not currently indexed in DOI meta-indexes (such as the one produced by DataCite<sup><a href="#r27-2975810" class="usa-link" aria-describedby="r27-2975810">27</a></sup>). In order to be compliant with F1 and A1, the GLDS is implementing methods to issue DOIs for all data publications. Additionally, curated metadata records of the publications are transmitted to DataCite<sup><a href="#r27-2975810" class="usa-link" aria-describedby="r27-2975810">27</a></sup> for meta-indexing, increasing the chances for reuse of GLDS-hosted data.</p>
<p>FAIR data also need to be described “with rich metadata” in order to be findable. The draft metric to assess F2 compliance merely requires a metadata description document that specifies a format for machine-readable metadata. The definition of the term “machine-readable” varies somewhat, but it is generally agreed that it cannot be natural language. (Although F2 refers to reusability principles requiring data “have a plurality of accurate and relevant attributes” (R1), the metric for F2 does not attempt to assess compliance with principle R1 [perhaps it should]). All the systems we assessed passed this metric. Furthermore, all systems passed F3, requiring association of metadata with data it describes. It should be noted that FM-F3 refers only to IRIs and not broadly to any other kinds of acceptable data identifiers; we assume that ultimately FM-F3 will be broadened to allow other kinds of F1-compliant data identifiers. Also, it is difficult to determine if any of the systems truly support IRIs, given that none of them currently appear to use non-ASCII character encodings in their data identifying schemes.</p>
<p>The FAIRness metric FM-F4 is rather ill-defined currently. It requires submission of an identifier of FAIR data and its metadata to search engines (which precisely, it does not yet specify, but some common search engines are mentioned as examples in the comments for the metric). A Pass should be given if the returned search results include links to the published data (again, it does not yet specify where in the returned results, although among both the top 10 and top 40 are mentioned in metric comments). Some of the systems we assessed failed this metric, and it seems their data records are likely not indexed by at least some search engines. For example, search of the MG-RAST data record identified by “<a href="http://metagenomics.anl.gov/linkin.cgi?metagenome=mgm4447971.3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://metagenomics.anl.gov/linkin.cgi?metagenome=mgm4447971.3</a>” or any of its tabular metadata yielded no search results referring to the repository using Google. More of the systems would fail this metric if it required data to be submitted specifically to DOI-based search systems (such as DataCite).</p></section><section id="sec2_5"><h3 class="pmc_sec_title">Accessibility</h3>
<p>All of the systems for which we assessed compliance with FAIR principle A1.1 passed; all use open source communication protocols to access data, which are retrieved by their identifiers. FAIR Principle A1.2 stipulates that any protocol used to retrieve data “allows for an authentication and authorization procedure, where necessary.” All the systems we assessed describe which functions of the system require credentials, and how to obtain the credentials. It should be noted that A1.2 can be applied to the design of all types of systems, from those with highly restricted access controls, to the kinds of “open access” systems we assessed, that typically require minimal authentication and grant the widest possible access to all users. Furthermore, the data in question could be any kind of data, including raw instrument output, shared data analysis workflows or data products, or scientifically-relevant user comments and opinions. Even for open access systems, some systems functions such as user-workspace file management, user commenting and messaging, and user attribution of data always require authentication, because these functions must necessarily leverage user-specific information.</p>
<p>A specific challenge that GeneLab faced when implementing A1.2 for the GLDS was a diversity of credentialing services available to the system’s target users, with no single service common to all users. Furthermore, the goal of GeneLab is to maximize access to GLDS data across all kinds of users, but within the bounds of authentication and credentialing policies of NASA - another challenge. Aside from policy restrictions, the development of any new credentialing and authentication functions or services can require considerable resources.</p>
<p>A solution for organizations that are seeking to develop open-access systems is to consider investing in development of an organization-wide integrated credentialing and authentication architecture (ICAA, <a href="#f3-2975810" class="usa-link">Figure 3</a>), and policies for allowing users to leverage extramural credentials, with appropriate access limitations. ICAAs and attendant policies can provide flexibility to develop and/or deploy data systems with available resources that can meet policy requirements of the organization that support extramural user access. It should be noted that using an ICAA to leverage credentialing systems developed independently from the system requires 1) technical compatibility between the authentication services of credentialing providers and the ICAA; and 2) compatibility of the credentialing policies of credentialing organizations and those of the organization developing the data system. Technical compatibility is facilitated through the use of authentication protocols and standards. Organizational credentialing policy incompatibility is more problematic to resolve. For example, it may be that the organization deploying the data system requires credentials be updated at an interval that is different from that required by one or more extramural credential provider. If these differences cannot be resolved, the only option may be to limit access for some extramural users data system functions requiring authentication.</p>
<figure class="fig xbox font-sm" id="f3-2975810"><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/876e/6371294/d93bd3cb9ffc/2975810f3.jpg" loading="lazy" height="508" width="798" alt="Figure 3."></p>
<div class="p text-right font-secondary"><a href="figure/f3-2975810/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Use of an Integrated Credentialing and Authentication Architecture (ICAA) to comply with FAIR principle A1.2. The ICAA provides services and policies that use information from intra- and extramural credentialing services to authenticate users of a data system.</p></figcaption></figure><p>NASA is developing an ICAA and adopting policies that permit certain intramural data systems to leverage it for user credentials and authentication. These policies stipulates that users within the organization must use the organization’s own credentialing services, while extramural users may authenticate using one from a defined set of externally-managed credentialing systems (currently including authentication by selected “social media” systems, such as Google.com credentials). At NASA (and at most U.S. federal agencies), intramural credentialing is quite a long process involving corroboration of much identifying information. The use of the ICAA, users without NASA credentials can specify a minimum of information (e.g., only a verifiable email address), which they can provide easily through a web browser, and gain almost immediate access to the GLDS. Through use of the ICAA, the GLDS is able to recognize a lower level of confidence when authenticating extramural users, and implement appropriate access controls to some functions for these users (e.g., they may not join system administrator groups). It seems reasonable to suggest that an additional principle be considered when designing FAIR data systems that have at least some functions requiring authentication or authorization (A1.2.1 in <a href="#t2-2975810" class="usa-link">Table 2</a>). This principle, would stipulate that an integrated credentialing architecture, like that shown in <a href="#f3-2975810" class="usa-link">Figure 3</a> be leveraged, in order to offer access to data to the widest number of users possible, when and where data policies permit such access.</p>
<section class="tw xbox font-sm" id="t2-2975810"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Additional principles for optimizing data accessibility</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="box" rules="none"><tbody><tr><td valign="top" align="left" rowspan="1" colspan="1"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/876e/6371294/61da22a426fb/2975810t2.jpg" loading="lazy" alt="graphic file with name 2975810t2.jpg"></td></tr></tbody></table></div>
<div class="p text-right font-secondary"><a href="table/t2-2975810/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>Most of the systems we assessed for FAIR compliance to A2 have some policy to retain at least some metadata when data are removed from the system. It is frequently unclear, however, which metadata were to be retained when data records are removed from the system. Data systems should ensure that policies for retaining metadata are clearly stated, including which metadata and for what period of time.</p>
<p>Another factor that specifically affects accessibility to Big Data is data transport. Data to be used as inputs in Big Data systems, but which are not located in the same environment as executable code of the system, commonly must first be transported (duplicated or moved) to the environment first. This environment typically is a user-specific workspace. The transport process for Big Data can require minutes, hours or even days in some Big Data systems, and represents a significant barrier to efficient access. Designers of Big Data would benefit from a principle that stipulates the collocation within Big Data systems of data and executable code (A3 in <a href="#t2-2975810" class="usa-link">Table 2</a>).</p></section><section id="sec2_6"><h3 class="pmc_sec_title">Interoperability</h3>
<p>I1 stipulates the “use a formal, accessible, etc. language for knowledge representation. The draft metric for this principle requires that the language for data or metadata be available by URL, and have a BNF representation. The comments on the metric suggest additional required language qualities, including that the language must have semantics (“[‘vanilla’] xml and json ..should fail”). While current knowledge representation languages such as RDF and OWL<sup><a href="#r28-2975810" class="usa-link" aria-describedby="r28-2975810">28</a></sup> are available to generate metadata specifications with inherent semantics, tools that leverage such languages require some sophistication in their use. All the repositories we assessed have metadata specifications that do not have inherent semantics, and thus would not yield a Pass for this metric. Metabolights and the GLDS both use the ISATab metadata specification, which can be converted to RDF, and so were rated with Partial Pass.</p>
<p>The metric for principle I2 has a related requirement: that data systems use open, community-developed vocabularies for data and metadata, and that such vocabularies be more than simple keywords (i.e., include semantics in some form). As the ISATab specification supported ontological lookup from such vocabularies, we again only rated Metabolights and the GLDS with Partial Pass, and the rest of the systems as FAIL.</p>
<p>In addition to lack of experience with knowledge representation technologies like RDF and OWL, another likely reason for lack of compliance of the systems with I1 and I2 is a general lack of available semantic resources developed by the community to generate metadata specifications and vocabularies, particularly in more recent domains like genomics, proteomics and metabolomics. We have ourselves sought such resources for use in the GLDS development, where we are seeking to provide users with omics data analysis capabilities, an omics “workbench” consisting of user-specific workspaces and data access controls, together with access to a suite of omics analysis tools (see <a href="#f1-2975810" class="usa-link">Figure 1</a>b and <a href="#f1-2975810" class="usa-link">Figure 1</a>c). Users of these tools will create data processing (analysis) workflows to transform input data into “higher-order” data products, which they will likely want to share with others and/or associate with the workflows and data inputs. One of the challenges we have observed for making these kinds of processed information FAIR include a lack of <span class="text-underline">semantic resources</span> for creating metadata specifications for such processed data and the processing workflows that generate them. While schemas like the Common Workflow Language<sup><a href="#r29-2975810" class="usa-link" aria-describedby="r29-2975810">29</a></sup> (CWL) specification provide a method to represent and reproduce the mechanics of data processing, they lack domain-specific semantics regarding the nature of these data transformations (e.g., what are the goals of the transformation from a biological perspective? What are the biological data concepts produced in output files? how do these relate to input data elements? etc.). Without these descriptors, systems may not adequately index these processed data and workflows for discovery and retrieval. While sophisticated users (e.g., trained bioinformaticians) who do encounter processed data and workflows may be able to understand and reuse them, others without domain-specific experience and knowledge of omics data processing may not. A challenge for the GLDS will be to help spur the creation of community resources so that it can implement rich metadata for processed omics data and workflows in a timely fashion.</p>
<p>The objective of the metric for principle I3 is to measure how much data in one system are linked semantically to data in other systems. It is not surprising that none of the data sources link data “out” to data in external systems in semantically qualified ways; this is a challenging objective for any data system. (We ignored links to records that were merely duplicated between systems) Manual linking of data requires subject matter expertise and can be time-consuming for systems with many data records. Automated linking requires that source and linked data both be characterized semantically so that links may be inferred. As mentioned above, semantic resources for characterizing omics data are by and large still forthcoming.</p></section><section id="sec2_7"><h3 class="pmc_sec_title">Reusability</h3>
<p>R1 is referred to by the findability FAIR principle F2 (see above); at this time there is no metric specifically for R1. R1.1 requires data systems to provide licenses for data usage (and is curiously intended to facilitate “legal interoperability” rather than reusability<sup><a href="#r30-2975810" class="usa-link" aria-describedby="r30-2975810">30</a></sup>. All the data repositories we assessed are governmental entities, and as such, licensing is traditionally prohibited.</p>
<p>R1.2 and R1.3 require that data are published together with provenance metadata, and that data and metadata conform to community-derived standards, respectively. Such standards include taxonomies, ontologies and other vocabularies for metadata, and data and file formats. The metric for R1.2 involves the demonstration that a system can provide FAIR-compliant citational and contextual provenance metadata. While all the repositories provide such metadata, some do not use FAIR-compliant schemes for these metadata. Similarly, most of the repositories assess and validate submitted data files (but few validate metadata, as discussed above) against FAIR-schemes and formats.</p></section></section><section id="sec3"><h2 class="pmc_sec_title">Discussion</h2>
<p>The systems we assessed for FAIRness clearly have shortcomings, particularly in the area of supporting data interoperability. None of the systems currently use a metadata representation language with sufficient semantics. In addition, none fully leverage FAIR vocabularies for metadata. There are perhaps several causes at work for these observations. There are often myriad vocabularies (ontologies) for metadata (each of which is a <em>de facto</em> community standard) even within a given biological sub-domain, and often much overlap in semantic content between them, creating the need for systems to be able to “normalize” input metadata across them somehow. For example, the concept of zero or very low-gravity experimental conditions appears in at least 5 different, widely-used biomedical vocabularies: SNOMED CT, MeSH, Read Codes Clinical Terms, Psychology Ontology, PLOS Thesaurus, and FAST Topical Facet, with varying language for the concept (“weightlessness” vs. “zero gravity”). True semantic interoperability and data reusability that would permit a user to find all data tagged with this concept require either that all these concept instances be individually associated with the data, or use of a system that reliably maps all the concepts from these ever-changing, ever-expanding set of ontologies. Even if such metadata normalizing mechanisms are developed, their performance and value in generating “rich” metadata is dependent on the quality of input concepts supplied either by original data submitters or biodata curators (typically employed by the system). The expertise that submitters have is critical to shaping metadata that have value for many different kinds of audiences, and that allow associated data to be found and reused. In the biomedical field, submitters to shared data repositories have relatively low motivation to go to any great lengths to provide “rich metadata” (or assist biocurators in generating them). This is likely due, at least in part, to the fact that only a small fraction of shared data appears to be cited in publication references when they are reused.<sup><a href="#r31-2975810" class="usa-link" aria-describedby="r31-2975810">31</a></sup> Without their input, niche repositories and data systems, like the GLDS, with limited resources for acquiring (and maintaining) rich metadata that meet all the criteria above must constantly trade the use of resources to acquire or generate rich metadata with those that could be used for other worthy goals.</p>
<p>The findability and interoperability principles, when implemented in data systems, support data integration functions like multi-source data querying and retrieval. Principles F2, I1, I2, and R1.3 require the use of community-developed metadata standards, formal languages for metadata representation, FAIR vocabularies, and qualified references to other data. These principles are unfortunately not yet widely or fully implemented across biomedical data systems, making it difficult for systems to develop functions that integrate data semantically. To illustrate this, we examine our own experience developing the GLDS to provide federated search and retrieval of omics data from several data sources. We sought to import metadata records from three open-access data systems into our own metadata warehouse: GEO, MG-RAST, and the PRoteomics IDEntifications repository (PRIDE)<sup><a href="#r5-2975810" class="usa-link" aria-describedby="r5-2975810">5</a></sup>. Each of these systems defines metadata for omics data sets differently, but none of them make these schemas available in a formal semantic language format (such as RDF or OWL). Furthermore, there are no existing maps of any of the metadata schemas to each other. As such, we resorted to manually creating semantic maps of these schemas for use in our system; queries generated via the GLDS or its RESTful search API are executed against the mapped schema concepts, and matching records are shown in a common object model representation (<a href="#f4-2975810" class="usa-link">Figure 4</a>). While this approach met our immediate goals of allowing users to query several data sources using our system, it required a good deal of resources and domain expertise to develop a schema mapping. Furthermore, maintaining this mapping over time as each of the data sources inevitably change will require more of our limited resources. The research community should recognize the value of federated system functions like the querying function that the GLDS offers, and urge all open access repositories and data systems to make their metadata schemas available to all according to the aforementioned FAIR principles.</p>
<figure class="fig xbox font-sm" id="f4-2975810"><h3 class="obj_head">Figure 4.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6371294_2975810f4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/876e/6371294/4a2378083d7d/2975810f4.jpg" loading="lazy" height="697" width="763" alt="Figure 4."></a></p>
<div class="p text-right font-secondary"><a href="figure/f4-2975810/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Federated Search of Omics Data in the GLDS. The RESTful search API result shows 65 records from the four data sources searched (GLDS, GEO, PRIDE, and MG-RAST). Each records metadata is mapped using a single schema for display in the GLDS (see <strong><a href="#f1-2975810" class="usa-link">Figure 1</a></strong>a).</p></figcaption></figure></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="r1-2975810">
<span class="label">1.</span><cite>Giardine B, Riemer C, Hardison RC, Burhans R, Elnitski L, Shah P, et al.  Galaxy: a platform for interactive large-scale genome analysis. Genome Res. 2005;15(10):1451–5. doi: 10.1101/gr.4086505.</cite> [<a href="https://doi.org/10.1101/gr.4086505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1240089/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16169926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Giardine%20B,%20Riemer%20C,%20Hardison%20RC,%20Burhans%20R,%20Elnitski%20L,%20Shah%20P,%20et%20al.%20Galaxy:%20a%20platform%20for%20interactive%20large-scale%20genome%20analysis.%20Genome%20Res.%202005;15(10):1451%E2%80%935.%20doi:%2010.1101/gr.4086505." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r2-2975810">
<span class="label">2.</span><cite>Arkin AP, Stevens RL, Cottingham RW, Maslov S, Henry CS, Dehal P, et al.   2016. The DOE Systems Biology Knowledgebase (KBase). bioRxiv.</cite> [<a href="https://scholar.google.com/scholar_lookup?Arkin%20AP,%20Stevens%20RL,%20Cottingham%20RW,%20Maslov%20S,%20Henry%20CS,%20Dehal%20P,%20et%20al.%202016.%20The%20DOE%20Systems%20Biology%20Knowledgebase%20(KBase).%20bioRxiv." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r3-2975810">
<span class="label">3.</span><cite>Goff SA, Vaughn M, McKay S, Lyons E, Stapleton AE, Gessler D, et al.  The iPlant Collaborative: Cyberinfrastructure for Plant Biology. Front Plant Sci. 2011;2:34. doi: 10.3389/fpls.2011.00034.</cite> [<a href="https://doi.org/10.3389/fpls.2011.00034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3355756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22645531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goff%20SA,%20Vaughn%20M,%20McKay%20S,%20Lyons%20E,%20Stapleton%20AE,%20Gessler%20D,%20et%20al.%20The%20iPlant%20Collaborative:%20Cyberinfrastructure%20for%20Plant%20Biology.%20Front%20Plant%20Sci.%202011;2:34.%20doi:%2010.3389/fpls.2011.00034." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r4-2975810">
<span class="label">4.</span><cite>Edgar R, Domrachev M, Lash AE. Gene Expression Omnibus: NCBI gene expression and hybridization array data repository. Nucleic Acids Res. 2002;30(1):207–10. doi: 10.1093/nar/30.1.207.</cite> [<a href="https://doi.org/10.1093/nar/30.1.207" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC99122/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11752295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Edgar%20R,%20Domrachev%20M,%20Lash%20AE.%20Gene%20Expression%20Omnibus:%20NCBI%20gene%20expression%20and%20hybridization%20array%20data%20repository.%20Nucleic%20Acids%20Res.%202002;30(1):207%E2%80%9310.%20doi:%2010.1093/nar/30.1.207." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r5-2975810">
<span class="label">5.</span><cite>Jones P, Cote RG, Martens L, Quinn AF, Taylor CF, Derache W, et al.  PRIDE: a public repository of protein and peptide identifications for the proteomics community. Nucleic Acids Res. 2006;34((Database issue)):D659–63. doi: 10.1093/nar/gkj138.</cite> [<a href="https://doi.org/10.1093/nar/gkj138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1347500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16381953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jones%20P,%20Cote%20RG,%20Martens%20L,%20Quinn%20AF,%20Taylor%20CF,%20Derache%20W,%20et%20al.%20PRIDE:%20a%20public%20repository%20of%20protein%20and%20peptide%20identifications%20for%20the%20proteomics%20community.%20Nucleic%20Acids%20Res.%202006;34((Database%20issue)):D659%E2%80%9363.%20doi:%2010.1093/nar/gkj138." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r6-2975810">
<span class="label">6.</span><cite>Meyer F, Paarmann D, D’Souza M, Olson R, Glass EM, Kubal M, et al.  The metagenomics RAST server – a public resource for the automatic phylogenetic and functional analysis of metagenomes. BMC Bioinformatics. 2008;9(1):386. doi: 10.1186/1471-2105-9-386.</cite> [<a href="https://doi.org/10.1186/1471-2105-9-386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2563014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18803844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meyer%20F,%20Paarmann%20D,%20D%E2%80%99Souza%20M,%20Olson%20R,%20Glass%20EM,%20Kubal%20M,%20et%20al.%20The%20metagenomics%20RAST%20server%20%E2%80%93%20a%20public%20resource%20for%20the%20automatic%20phylogenetic%20and%20functional%20analysis%20of%20metagenomes.%20BMC%20Bioinformatics.%202008;9(1):386.%20doi:%2010.1186/1471-2105-9-386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r7-2975810">
<span class="label">7.</span><cite>Leinonen R, Akhtar R, Birney E, Bower L, Cerdeno-Tarraga A, Cheng Y, et al.  The European Nucleotide Archive. Nucleic Acids Res. 2011;39((Database issue)):D28–31. doi: 10.1093/nar/gkq967.</cite> [<a href="https://doi.org/10.1093/nar/gkq967" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3013801/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20972220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leinonen%20R,%20Akhtar%20R,%20Birney%20E,%20Bower%20L,%20Cerdeno-Tarraga%20A,%20Cheng%20Y,%20et%20al.%20The%20European%20Nucleotide%20Archive.%20Nucleic%20Acids%20Res.%202011;39((Database%20issue)):D28%E2%80%9331.%20doi:%2010.1093/nar/gkq967." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r8-2975810">
<span class="label">8.</span><cite>Haug K, Salek RM, Conesa P, Hastings J, de Matos P, Rijnbeek M, et al.  MetaboLights--an open-access general-purpose repository for metabolomics studies and associated meta-data. Nucleic Acids Res. 2013;41((Database issue)):D781–6. doi: 10.1093/nar/gks1004.</cite> [<a href="https://doi.org/10.1093/nar/gks1004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3531110/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23109552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haug%20K,%20Salek%20RM,%20Conesa%20P,%20Hastings%20J,%20de%20Matos%20P,%20Rijnbeek%20M,%20et%20al.%20MetaboLights--an%20open-access%20general-purpose%20repository%20for%20metabolomics%20studies%20and%20associated%20meta-data.%20Nucleic%20Acids%20Res.%202013;41((Database%20issue)):D781%E2%80%936.%20doi:%2010.1093/nar/gks1004." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r9-2975810">
<span class="label">9.</span><cite>Motenko H, Neuhauser SB, O’Keefe M, Richardson JE. MouseMine: a new data warehouse for MGI. Mamm Genome. 2015;26(7-8):325–30. doi: 10.1007/s00335-015-9573-z.</cite> [<a href="https://doi.org/10.1007/s00335-015-9573-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4534495/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26092688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Motenko%20H,%20Neuhauser%20SB,%20O%E2%80%99Keefe%20M,%20Richardson%20JE.%20MouseMine:%20a%20new%20data%20warehouse%20for%20MGI.%20Mamm%20Genome.%202015;26(7-8):325%E2%80%9330.%20doi:%2010.1007/s00335-015-9573-z." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r10-2975810">
<span class="label">10.</span><cite>Stein L, Sternberg P, Durbin R, Thierry-Mieg J, Spieth J. WormBase: network access to the genome and biology of Caenorhabditis elegans. Nucleic Acids Res. 2001;29(1):82–6. doi: 10.1093/nar/29.1.82.</cite> [<a href="https://doi.org/10.1093/nar/29.1.82" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC29781/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11125056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stein%20L,%20Sternberg%20P,%20Durbin%20R,%20Thierry-Mieg%20J,%20Spieth%20J.%20WormBase:%20network%20access%20to%20the%20genome%20and%20biology%20of%20Caenorhabditis%20elegans.%20Nucleic%20Acids%20Res.%202001;29(1):82%E2%80%936.%20doi:%2010.1093/nar/29.1.82." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r11-2975810">
<span class="label">11.</span><cite>Arend D, Junker A, Scholz U, Schuler D, Wylie J, Lange M.  2016 2016. PGP repository: a plant phenomics and genomics data publication infrastructure. Database (Oxford)</cite> [<a href="https://doi.org/10.1093/database/baw033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4834206/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27087305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arend%20D,%20Junker%20A,%20Scholz%20U,%20Schuler%20D,%20Wylie%20J,%20Lange%20M.%202016%202016.%20PGP%20repository:%20a%20plant%20phenomics%20and%20genomics%20data%20publication%20infrastructure.%20Database%20(Oxford)" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r12-2975810">
<span class="label">12.</span><cite>Leinonen R, Sugawara H, Shumway M. The sequence read archive. Nucleic Acids Res. 2011;39((Database issue)):D19–21. doi: 10.1093/nar/gkq1019.</cite> [<a href="https://doi.org/10.1093/nar/gkq1019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3013647/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21062823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leinonen%20R,%20Sugawara%20H,%20Shumway%20M.%20The%20sequence%20read%20archive.%20Nucleic%20Acids%20Res.%202011;39((Database%20issue)):D19%E2%80%9321.%20doi:%2010.1093/nar/gkq1019." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r13-2975810">
<span class="label">13.</span><cite>Huala E, Dickerman AW, Garcia-Hernandez M, Weems D, Reiser L, LaFond F, et al.  The Arabidopsis Information Resource (TAIR): a comprehensive database and web-based information retrieval, analysis, and visualization system for a model plant. Nucleic Acids Res. 2001;29(1):102–5. doi: 10.1093/nar/29.1.102.</cite> [<a href="https://doi.org/10.1093/nar/29.1.102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC29827/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11125061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huala%20E,%20Dickerman%20AW,%20Garcia-Hernandez%20M,%20Weems%20D,%20Reiser%20L,%20LaFond%20F,%20et%20al.%20The%20Arabidopsis%20Information%20Resource%20(TAIR):%20a%20comprehensive%20database%20and%20web-based%20information%20retrieval,%20analysis,%20and%20visualization%20system%20for%20a%20model%20plant.%20Nucleic%20Acids%20Res.%202001;29(1):102%E2%80%935.%20doi:%2010.1093/nar/29.1.102." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r14-2975810">
<span class="label">14.</span><cite>FlyBase: the Drosophila database. Nucleic Acids Res. 1996;24(1):53–6. doi: 10.1093/nar/24.1.53.</cite> [<a href="https://doi.org/10.1093/nar/24.1.53" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC145580/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8594600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?FlyBase:%20the%20Drosophila%20database.%20Nucleic%20Acids%20Res.%201996;24(1):53%E2%80%936.%20doi:%2010.1093/nar/24.1.53." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r15-2975810">
<span class="label">15.</span><cite>Joshi-Tope G, Gillespie M, Vastrik I, D’Eustachio P, Schmidt E, de Bono B, et al.  Reactome: a knowledgebase of biological pathways. Nucleic Acids Res. 2005;33((Database issue)):D428–32. doi: 10.1093/nar/gki072.</cite> [<a href="https://doi.org/10.1093/nar/gki072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC540026/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15608231/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Joshi-Tope%20G,%20Gillespie%20M,%20Vastrik%20I,%20D%E2%80%99Eustachio%20P,%20Schmidt%20E,%20de%20Bono%20B,%20et%20al.%20Reactome:%20a%20knowledgebase%20of%20biological%20pathways.%20Nucleic%20Acids%20Res.%202005;33((Database%20issue)):D428%E2%80%9332.%20doi:%2010.1093/nar/gki072." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r16-2975810">
<span class="label">16.</span><cite>Ogata H, Goto S, Sato K, Fujibuchi W, Bono H, Kanehisa M. KEGG: Kyoto Encyclopedia of Genes and Genomes. Nucleic Acids Res. 1999;27(1):29–34. doi: 10.1093/nar/27.1.29.</cite> [<a href="https://doi.org/10.1093/nar/27.1.29" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC148090/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9847135/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ogata%20H,%20Goto%20S,%20Sato%20K,%20Fujibuchi%20W,%20Bono%20H,%20Kanehisa%20M.%20KEGG:%20Kyoto%20Encyclopedia%20of%20Genes%20and%20Genomes.%20Nucleic%20Acids%20Res.%201999;27(1):29%E2%80%9334.%20doi:%2010.1093/nar/27.1.29." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r17-2975810">
<span class="label">17.</span><cite>Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, et al.  Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. Nat Genet. 2000;25(1):25–9. doi: 10.1038/75556.</cite> [<a href="https://doi.org/10.1038/75556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3037419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10802651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ashburner%20M,%20Ball%20CA,%20Blake%20JA,%20Botstein%20D,%20Butler%20H,%20Cherry%20JM,%20et%20al.%20Gene%20ontology:%20tool%20for%20the%20unification%20of%20biology.%20The%20Gene%20Ontology%20Consortium.%20Nat%20Genet.%202000;25(1):25%E2%80%939.%20doi:%2010.1038/75556." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r18-2975810">
<span class="label">18.</span><cite>Wilkinson MD, Dumontier M, Aalbersberg IJ, Appleton G, Axton M, Baak A, et al.  The FAIR Guiding Principles for scientific data management and stewardship. Sci Data. 2016;3:160018. doi: 10.1038/sdata.2016.18.</cite> [<a href="https://doi.org/10.1038/sdata.2016.18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4792175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26978244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilkinson%20MD,%20Dumontier%20M,%20Aalbersberg%20IJ,%20Appleton%20G,%20Axton%20M,%20Baak%20A,%20et%20al.%20The%20FAIR%20Guiding%20Principles%20for%20scientific%20data%20management%20and%20stewardship.%20Sci%20Data.%202016;3:160018.%20doi:%2010.1038/sdata.2016.18." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r19-2975810">
<span class="label">19.</span><cite>Wilkinson MD, Sansone S-A, Schultes E, Doorn P, Bonino da Silva, Santos LO, Dumontier M.  2017. A design framework and exemplar metrics for FAIRness. bioRxiv.</cite> [<a href="https://doi.org/10.1038/sdata.2018.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6018520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29944145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilkinson%20MD,%20Sansone%20S-A,%20Schultes%20E,%20Doorn%20P,%20Bonino%20da%20Silva,%20Santos%20LO,%20Dumontier%20M.%202017.%20A%20design%20framework%20and%20exemplar%20metrics%20for%20FAIRness.%20bioRxiv." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r20-2975810">
<span class="label">20.</span><cite>Berrios D, Thompson T, Fogle H, Rask J, Coughlan J.  San Francisco: Nov 14-18, GeneLab: NASA’s Open Access, Collaborative Platform for Systems Biology and Space Medicine. Poster presented at: 2015 AMIA Annual Symposium; p. CA2015.</cite> [<a href="https://scholar.google.com/scholar_lookup?Berrios%20D,%20Thompson%20T,%20Fogle%20H,%20Rask%20J,%20Coughlan%20J.%20San%20Francisco:%20Nov%2014-18,%20GeneLab:%20NASA%E2%80%99s%20Open%20Access,%20Collaborative%20Platform%20for%20Systems%20Biology%20and%20Space%20Medicine.%20Poster%20presented%20at:%202015%20AMIA%20Annual%20Symposium;%20p.%20CA2015." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r21-2975810">
<span class="label">21.</span><cite>Berrios D, Welch J, Fogle H, Skidmore M, Marcu O.  Chicago: GeneLab: NASA’s GeneLab: Phase I Results and Plans. Poster presented at: 2016 AMIA Annual Symposium; p. IL2016.</cite> [<a href="https://scholar.google.com/scholar_lookup?Berrios%20D,%20Welch%20J,%20Fogle%20H,%20Skidmore%20M,%20Marcu%20O.%20Chicago:%20GeneLab:%20NASA%E2%80%99s%20GeneLab:%20Phase%20I%20Results%20and%20Plans.%20Poster%20presented%20at:%202016%20AMIA%20Annual%20Symposium;%20p.%20IL2016." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r22-2975810">
<span class="label">22.</span><cite>Beheshti A, Cekanaviciute E, Smith DJ, Costes SV. Global transcriptomic analysis suggests carbon dioxide as an environmental stressor in spaceflight: a systems biology GeneLab case study. Sci Rep. 2018;8(1):4191. doi: 10.1038/s41598-018-22613-1.</cite> [<a href="https://doi.org/10.1038/s41598-018-22613-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5843582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29520055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beheshti%20A,%20Cekanaviciute%20E,%20Smith%20DJ,%20Costes%20SV.%20Global%20transcriptomic%20analysis%20suggests%20carbon%20dioxide%20as%20an%20environmental%20stressor%20in%20spaceflight:%20a%20systems%20biology%20GeneLab%20case%20study.%20Sci%20Rep.%202018;8(1):4191.%20doi:%2010.1038/s41598-018-22613-1." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r23-2975810">
<span class="label">23.</span><cite>FAIR Metrics ALL: GO FAIR Metrics Group. 2018.  [updated January 11, 2018. Available from: </cite> [<a href="https://doi.org/10.5281/zenodo.1065973" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="r24-2975810">
<span class="label">24.</span><cite>McMurry JA, Juty N, Blomberg N, Burdett T, Conlin T, Conte N, et al.  Identifiers for the 21st century: How to design, provision, and reuse persistent identifiers to maximize utility and impact of life science data. PLoS Biol. 2017;15(6):e2001414. doi: 10.1371/journal.pbio.2001414.</cite> [<a href="https://doi.org/10.1371/journal.pbio.2001414" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5490878/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28662064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McMurry%20JA,%20Juty%20N,%20Blomberg%20N,%20Burdett%20T,%20Conlin%20T,%20Conte%20N,%20et%20al.%20Identifiers%20for%20the%2021st%20century:%20How%20to%20design,%20provision,%20and%20reuse%20persistent%20identifiers%20to%20maximize%20utility%20and%20impact%20of%20life%20science%20data.%20PLoS%20Biol.%202017;15(6):e2001414.%20doi:%2010.1371/journal.pbio.2001414." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r25-2975810">
<span class="label">25.</span><cite>Starr J, Castro E, Crosas M, Dumontier M, Downs RR, Duerr R, et al.   2015. Achieving human and machine accessibility of cited data in scholarly publications. PeerJ Comput Sci; p. 1.</cite> [<a href="https://doi.org/10.7717/peerj-cs.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4498574/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26167542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Starr%20J,%20Castro%20E,%20Crosas%20M,%20Dumontier%20M,%20Downs%20RR,%20Duerr%20R,%20et%20al.%202015.%20Achieving%20human%20and%20machine%20accessibility%20of%20cited%20data%20in%20scholarly%20publications.%20PeerJ%20Comput%20Sci;%20p.%201." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r26-2975810">
<span class="label">26.</span><cite>Boudry C, Chartron G. Availability of digital object identifiers in publications archived by PubMed. Scientometrics. 2017;110(3):1453–69.</cite> [<a href="https://scholar.google.com/scholar_lookup?Boudry%20C,%20Chartron%20G.%20Availability%20of%20digital%20object%20identifiers%20in%20publications%20archived%20by%20PubMed.%20Scientometrics.%202017;110(3):1453%E2%80%9369." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r27-2975810">
<span class="label">27.</span><cite>Neumann J, Brase J. DataCite and DOI names for research data. J Comput Aided Mol Des. 2014;28(10):1035–41. doi: 10.1007/s10822-014-9776-5.</cite> [<a href="https://doi.org/10.1007/s10822-014-9776-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25038897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Neumann%20J,%20Brase%20J.%20DataCite%20and%20DOI%20names%20for%20research%20data.%20J%20Comput%20Aided%20Mol%20Des.%202014;28(10):1035%E2%80%9341.%20doi:%2010.1007/s10822-014-9776-5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r28-2975810">
<span class="label">28.</span><cite>Patel-Schneider PF, Hayes P, Horrocks I. OWL Web Ontology Language: Semantics and Abstract Syntax. 2004;W3C</cite> [<a href="https://scholar.google.com/scholar_lookup?Patel-Schneider%20PF,%20Hayes%20P,%20Horrocks%20I.%20OWL%20Web%20Ontology%20Language:%20Semantics%20and%20Abstract%20Syntax.%202004;W3C" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r29-2975810">
<span class="label">29.</span><cite>Peter A, Michael R. C, Nebojša T, Brad C, John C, Michael H, et al.  Common Workflow Language. v1:02016.</cite> [<a href="https://scholar.google.com/scholar_lookup?Peter%20A,%20Michael%20R.%20C,%20Neboj%C5%A1a%20T,%20Brad%20C,%20John%20C,%20Michael%20H,%20et%20al.%20Common%20Workflow%20Language.%20v1:02016." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="r30-2975810">
<span class="label">30.</span><cite>R1.1: (Meta)data are released with a clear and accessible data usage license: GO FAIR Initiative.  [Available from:  <a href="https://www.go-fair.org/fair-principles/r1-1-metadata-released-clear-accessible-data-usage-license/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.go-fair.org/fair-principles/r1-1-metadata-released-clear-accessible-data-usage-license/</a></cite>
</li>
<li id="r31-2975810">
<span class="label">31.</span><cite>Kratz JE, Strasser C. Researcher perspectives on publication and peer review of data. PLoS One. 2015;10(2):e0117619. doi: 10.1371/journal.pone.0117619.</cite> [<a href="https://doi.org/10.1371/journal.pone.0117619" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4338305/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25706992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kratz%20JE,%20Strasser%20C.%20Researcher%20perspectives%20on%20publication%20and%20peer%20review%20of%20data.%20PLoS%20One.%202015;10(2):e0117619.%20doi:%2010.1371/journal.pone.0117619." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from AMIA Annual Symposium Proceedings are provided here courtesy of <strong>American Medical Informatics Association</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               
               <li>
                    <a
                            href="pdf/2975810.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/6371294/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/6371294/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC6371294%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC6371294/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC6371294/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC6371294/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/30815061/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC6371294/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/30815061/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC6371294/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/6371294/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="17Wm34SLhOUEZsxvHQdNx3AmJDK5GFcgdkHJRoZI0uu8IWmOYDNr5MEAjwlklrS9">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
