
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            S. aureus MscL Is a Pentamer In Vivo but of Variable Stoichiometries In Vitro: Implications for Detergent-Solubilized Membrane Proteins - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="DB2B52E58E161CE31452E5001BA8A9DD.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plosbiol">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC2998437/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLoS Biology">
<meta name="citation_title" content="S. aureus MscL Is a Pentamer In Vivo but of Variable Stoichiometries In Vitro: Implications for Detergent-Solubilized Membrane Proteins">
<meta name="citation_author" content="Michael R Dorwart">
<meta name="citation_author_institution" content="Department of Physiology, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America">
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America">
<meta name="citation_author" content="Robin Wray">
<meta name="citation_author_institution" content="Department of Physiology, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America">
<meta name="citation_author" content="Chad A Brautigam">
<meta name="citation_author_institution" content="Department of Biochemistry, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America">
<meta name="citation_author" content="Youxing Jiang">
<meta name="citation_author_institution" content="Department of Physiology, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America">
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America">
<meta name="citation_author" content="Paul Blount">
<meta name="citation_author_institution" content="Department of Physiology, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America">
<meta name="citation_publication_date" content="2010 Dec 7">
<meta name="citation_volume" content="8">
<meta name="citation_issue" content="12">
<meta name="citation_firstpage" content="e1000555">
<meta name="citation_doi" content="10.1371/journal.pbio.1000555">
<meta name="citation_pmid" content="21151884">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC2998437/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC2998437/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC2998437/pdf/pbio.1000555.pdf">
<meta name="description" content="Detergent-induced rearrangements of membrane-protein subunits explain why two MscL channel stoichiometries have been resolved by X-ray crystallography - but S. aureus MscL is truly a pentamer in vivo.">
<meta name="og:title" content="S. aureus MscL Is a Pentamer In Vivo but of Variable Stoichiometries In Vitro: Implications for Detergent-Solubilized Membrane Proteins">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Detergent-induced rearrangements of membrane-protein subunits explain why two MscL channel stoichiometries have been resolved by X-ray crystallography - but S. aureus MscL is truly a pentamer in vivo.">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC2998437/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://pmc.ncbi.nlm.nih.gov/search/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="2998437">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pbio.1000555"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pbio.1000555.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC2998437%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/2998437/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/2998437/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC2998437/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plosbiol.png" alt="PLOS Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS Biology" title="Link to PLOS Biology" shape="default" href="https://doi.org/10.1371/journal.pbio.1000555" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS Biol</button></div>. 2010 Dec 7;8(12):e1000555. doi: <a href="https://doi.org/10.1371/journal.pbio.1000555" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pbio.1000555</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://pmc.ncbi.nlm.nih.gov/search/?term=%22PLoS%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>
<em>S. aureus</em> MscL Is a Pentamer In Vivo but of Variable Stoichiometries In Vitro: Implications for Detergent-Solubilized Membrane Proteins</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dorwart%20MR%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Michael R Dorwart</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Michael R Dorwart</span></h3>
<div class="p">
<sup>1</sup>Department of Physiology, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div class="p">
<sup>2</sup>Howard Hughes Medical Institute, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dorwart%20MR%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael R Dorwart</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wray%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Robin Wray</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Robin Wray</span></h3>
<div class="p">
<sup>1</sup>Department of Physiology, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wray%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Robin Wray</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brautigam%20CA%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Chad A Brautigam</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Chad A Brautigam</span></h3>
<div class="p">
<sup>3</sup>Department of Biochemistry, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brautigam%20CA%22%5BAuthor%5D" class="usa-link"><span class="name western">Chad A Brautigam</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Youxing Jiang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Youxing Jiang</span></h3>
<div class="p">
<sup>1</sup>Department of Physiology, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div class="p">
<sup>2</sup>Howard Hughes Medical Institute, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Youxing Jiang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blount%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Paul Blount</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Paul Blount</span></h3>
<div class="p">
<sup>1</sup>Department of Physiology, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blount%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Paul Blount</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Gregory A Petsko</span><sup>4</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>Department of Physiology, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div id="aff2">
<sup>2</sup>Howard Hughes Medical Institute, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div id="aff3">
<sup>3</sup>Department of Biochemistry, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div id="edit1">
<sup>4</sup>Brandeis University, United States of America</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>✉</sup><p class="display-inline">* E-mail: <span>Paul.Blount@UTSouthwestern.edu</span></p>
</div>
<div class="fn" id="fn1"><p>The author(s) have made the following declarations about their contributions: Conceived and designed the experiments: MRD RW CAB YJ PB. Performed the experiments: MRD RW CAB. Analyzed the data: MRD RW CAB PB. Wrote the paper: MRD PB.</p></div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Gregory A Petsko</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2010 Jul 9; Accepted 2010 Oct 27; Collection date 2010 Dec.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Dorwart et al.</div>
<p>This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC2998437  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/21151884/" class="usa-link">21151884</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><hr class="headless">
<p>Detergent-induced rearrangements of membrane-protein subunits explain why two MscL channel stoichiometries have been resolved by X-ray crystallography - but <em>S. aureus</em> MscL is truly a pentamer in vivo.</p></section><section class="abstract" id="abstract2"><h2>Abstract</h2>
<p>While the bacterial mechanosensitive channel of large conductance (MscL) is the best studied biological mechanosensor and serves as a paradigm for how a protein can sense and respond to membrane tension, the simple matter of its oligomeric state has led to debate, with models ranging from tetramers to hexamers. Indeed, two different oligomeric states of the bacterial mechanosensitive channel MscL have been resolved by X-ray crystallography: The <em>M. tuberculosis</em> channel (MtMscL) is a pentamer, while the <em>S. aureus</em> protein (SaMscL) forms a tetramer. Because several studies suggest that, like MtMscL, the <em>E. coli</em> MscL (EcoMscL) is a pentamer, we re-investigated the oligomeric state of SaMscL. To determine the structural organization of MscL in the cell membrane we developed a disulfide-trapping approach. Surprisingly, we found that virtually all SaMscL channels in vivo are pentameric, indicating this as the physiologically relevant and functional oligomeric state. Complementing our in vivo results, we purified SaMscL and assessed its oligomeric state using three independent approaches (sedimentation equilibrium centrifugation, crosslinking, and light scattering) and established that SaMscL is a pentamer when solubilized in Triton X-100 and C<sub>8</sub>E<sub>5</sub> detergents. However, performing similar experiments on SaMscL solubilized in LDAO, the detergent used in the crystallographic study, confirmed the tetrameric oligomerization resolved by X-ray crystallography. We further demonstrate that this stoichiometric shift is reversible by conventional detergent exchange experiments. Our results firmly establish the pentameric organization of SaMscL in vivo. Furthermore they demonstrate that detergents can alter the subunit stoichiometry of membrane protein complexes in vitro; thus, in vivo assays are necessary to firmly establish a membrane protein's true functionally relevant oligomeric state.</p></section><section class="abstract" id="abstract3"><h2>Author Summary</h2>
<p>The ability to detect mechanical forces is at the basis of not only the senses of touch hearing and balance but also cardiovascular and osmotic regulation. One of the primary ways that organisms detect forces is through mechanosensitive channels, and mechanosensation is so vital that essentially all organisms have at least one such sensor. Indeed, the best-studied mechanosensitive channel is from bacteria, and because relatively little is known of mechanosensors from higher organisms, these channels are a model for how a protein can sense and respond to mechanical forces. Although the bacterial mechanosensitive channel MscL has been well studied, the simple issue of how many subunits it has is hotly debated. There are even two published crystal structures showing either tetrameric or pentameric complexes. Here we show that the channel is actually pentameric in vivo and that the detergent used to solubilize the protein can rearrange the complexes from pentamers to tetramers. The finding that detergents can have such a profound effect on structure may have broad implications for the study of other membrane proteins.</p></section><section id="s1"><h2 class="pmc_sec_title">Introduction</h2>
<p>The bacterial mechanosensitive channel MscL serves as a biological “emergency release valve,” allowing rapid loss of solutes in response to a sudden decrease in the osmolarity of a bacterium's environment <a href="#pbio.1000555-Levina1" class="usa-link" aria-describedby="pbio.1000555-Levina1">[1]</a>. It is perhaps the best characterized mechanosensor <a href="#pbio.1000555-Blount1" class="usa-link" aria-describedby="pbio.1000555-Blount1">[2]</a>, thus serving as a paradigm of how a membrane protein can detect and respond to mechanical forces <a href="#pbio.1000555-Blount2" class="usa-link" aria-describedby="pbio.1000555-Blount2">[3]</a>. Ironically, something as simple as the stoichiometry of the MscL complex has plagued the field with debate since its inception.</p>
<p>The original model for the <em>E. coli</em> MscL (EcoMscL) stoichiometry was a homo-hexameric organization, which was suggested by crosslinking and the study of tandem subunits <a href="#pbio.1000555-Blount3" class="usa-link" aria-describedby="pbio.1000555-Blount3">[4]</a>. This model then appeared to be supported by low-resolution two-dimensional crystallization of EcoMscL <a href="#pbio.1000555-Saint1" class="usa-link" aria-describedby="pbio.1000555-Saint1">[5]</a>. But the subsequent elucidation of the <em>M. tuberculosis</em> channel (MtMscL) by X-ray crystallography <a href="#pbio.1000555-Chang1" class="usa-link" aria-describedby="pbio.1000555-Chang1">[6]</a> then suggested a pentameric organization, at least for this orthologue. This result led to a re-evaluation of EcoMscL stoichiometry <a href="#pbio.1000555-Chang1" class="usa-link" aria-describedby="pbio.1000555-Chang1">[6]</a>,<a href="#pbio.1000555-Sukharev1" class="usa-link" aria-describedby="pbio.1000555-Sukharev1">[7]</a>, which supported a pentameric organization and brought into question whether the two-dimensional crystallization data could be fit by 5-fold as well as 6-fold symmetry. Thus, the field transiently seemed to have settled that MscL was most likely a pentamer. However, the recent crystallographic structure of the <em>S. aureus</em> homolog (SaMscL) reveals a tetramer variant <a href="#pbio.1000555-Liu1" class="usa-link" aria-describedby="pbio.1000555-Liu1">[8]</a>. This latter finding has again raised questions regarding the true oligomeric state of MscL and evokes the possibilities either that MscL from different species assemble into complexes with different stoichiometries or that the channel exists as multiple functional oligomeric complexes in the cell membrane. Therefore, we set out to identify the MscL oligomeric state in the cell membrane and to understand how the SaMscL channel, which shares approximately 40% sequence identity with EcoMscL and MtMscL <a href="#pbio.1000555-Liu1" class="usa-link" aria-describedby="pbio.1000555-Liu1">[8]</a>, could exist in the non-pentameric subunit organization resolved by X-ray crystallography. We found not only that the true in vivo oligomeric state of SaMscL is a pentamer but also that at least one detergent, LDAO, artificially but reversibly reorganizes this structure into a tetrameric stoichiometry.</p></section><section id="s2"><h2 class="pmc_sec_title">Results</h2>
<p>With multiple oligomeric states identified in vitro, we devised a disulfide-trapping strategy to identify the oligomeric state of MscL in bacterial membranes. Using this approach, we were able to directly measure the subunit stoichiometry of SaMscL in vivo by generating a series of double-cysteine mutants in regions predicted to be in close proximity from both existing crystal structures (<a href="#pbio-1000555-g001" class="usa-link">Figure 1A</a>). These modifications allow crosslinking in the cell membrane via disulfide bonds as previously described <a href="#pbio.1000555-Iscla1" class="usa-link" aria-describedby="pbio.1000555-Iscla1">[9]</a>,<a href="#pbio.1000555-Li1" class="usa-link" aria-describedby="pbio.1000555-Li1">[10]</a>. Briefly, the cells were osmotically shocked in the presence of the oxidizing agent copper-phenanthroline, centrifuged, and then immediately resuspended in SDS loading buffer to avoid unnecessary manipulation of the proteins. As we previously observed <a href="#pbio.1000555-Iscla1" class="usa-link" aria-describedby="pbio.1000555-Iscla1">[9]</a>, osmotic shock increased disulfide bridging, presumably because reductants including glutathione and thioredoxin are released from the cell <a href="#pbio.1000555-Ajouz1" class="usa-link" aria-describedby="pbio.1000555-Ajouz1">[11]</a>, thus decreasing the reducing potential of the cytosol. The protein content of the lysed cells was evaluated using SDS-PAGE and molecular weight standards. For the L10C/L89C or L10C/M91C SaMscL channels, we were able to observe efficient in vivo disulfide trapping (<a href="#pbio-1000555-g001" class="usa-link">Figure 1B</a>). For the L10C/L89C protein, a range of oligomeric states was observed, a pentamer being the maximal state. However, for the L10C/M91C protein, one primary band was observed that reflects a pentameric complex. These data demonstrate that the pentamer is by far the major species within the cell membrane and thus appears to be the true structural and functional unit in vivo.</p>
<figure class="fig xbox font-sm" id="pbio-1000555-g001"><h3 class="obj_head">Figure 1. 
<em>S. aureus</em> MscL is a pentamer in the cell membrane.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2998437_pbio.1000555.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/fe2ef5a72403/pbio.1000555.g001.jpg" loading="lazy" height="432" width="656" alt="Figure 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pbio-1000555-g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Two orientations of the SaMscL and MtMscL crystal structures are shown with the molecules viewed from the periplasmic side (top) and from the lipid bilayer (bottom). The locations of the residues used to perform the in vivo disulfide crosslinking experiments are highlighted on the structures in surface representation: blue represents L10 (SaMscL) and A10 (MtMscL), orange represents T89 (SaMscL) and T96 (MtMscL), and red represents M91 (SaMscL) and R98 (MtMscL). (B) Western blot analysis of L10C/L89C and L10C/M91C SaMscL channels after in vivo disulfide bond trapping using copper-phenanthroline in both osmotically shocked (S+) and non-shocked (NS) samples.</p></figcaption></figure><p>With SaMscL pentamers being the predominant oligomeric state in the cell membrane, we wanted to understand why SaMscL crystallized as a tetramer. We began by confirming the previously observed pentameric and tetrameric states of MscL by performing crosslinking experiments of purified EcoMscL and SaMscL, visualizing their oligomeric state by Western blot analysis (<a href="#pbio-1000555-g002" class="usa-link">Figure 2</a>). Five crosslinked EcoMscL products were observed, consistent with previous reports showing that EcoMscL is a pentamer <a href="#pbio.1000555-Chang1" class="usa-link" aria-describedby="pbio.1000555-Chang1">[6]</a>,<a href="#pbio.1000555-Sukharev1" class="usa-link" aria-describedby="pbio.1000555-Sukharev1">[7]</a>. In contrast with previous findings <a href="#pbio.1000555-Liu1" class="usa-link" aria-describedby="pbio.1000555-Liu1">[8]</a>, SaMscL also presented five crosslinked products, suggesting that SaMscL also exists in a pentameric oligomeric state. No difference was observed between proteins expressed by a high-expressing plasmid (pET) and those encoded a moderate-expressing plasmid (pB10), demonstrating that expressions levels have no effect on MscL oligomerization. To substantiate these results, sedimentation equilibrium analytical ultracentrifugation (SE) was performed to directly measure the mass of the SaMscL channel. The results for SaMscL were fit to a single-species model and yielded a molar mass of 71.2 kDa, consistent with a pentameric subunit organization (SaMscL theoretical pentameric molar mass is 72.2 kDa) (<a href="#pbio-1000555-g003" class="usa-link">Figure 3A, B</a>). As a third independent method to directly measure the mass of the SaMscL channel, we performed size exclusion chromatography multi-angle light scattering (SEC-MALS) experiments <a href="#pbio.1000555-Slotboom1" class="usa-link" aria-describedby="pbio.1000555-Slotboom1">[12]</a>. Analysis of nine independent SEC-MALS experiments resulted in an average calculated SaMscL mass of 72.8±0.8 kDa (Ave ± SD, <em>n</em> = 9), again consistent with a pentameric subunit organization of SaMscL (representative experiment shown in <a href="#pbio-1000555-g003" class="usa-link">Figure 3C</a>).</p>
<figure class="fig xbox font-sm" id="pbio-1000555-g002"><h3 class="obj_head">Figure 2. 
<em>E. coli</em> and <em>S. aureus</em> MscL are pentameric multimers by crosslinking.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2998437_pbio.1000555.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/7c1037cd54d4/pbio.1000555.g002.jpg" loading="lazy" height="361" width="408" alt="Figure 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pbio-1000555-g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Western blot analysis of DSS-treated <em>E. coli</em>-MscL (Eco) and <em>S. aureus</em>-MscL (Sa) expressed with either the pET21a (pET) or pB10 vector show five distinct bands at both expression levels. Molecular weight markers are shown to the left of the blot and the approximate location of monomers (1×) to pentamers (5×) is shown between the blots. The right-hand blot shows a separate experiment where the majority of the EcoMscL and the SaMscL protein are in the pentameric and monomeric forms.</p></figcaption></figure><figure class="fig xbox font-sm" id="pbio-1000555-g003"><h3 class="obj_head">Figure 3. Purified <em>S. aureus</em> MscL is pentameric by equilibrium sedimentation centrifugation and SEC-MALS.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2998437_pbio.1000555.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/8de34c1e57ce/pbio.1000555.g003.jpg" loading="lazy" height="657" width="476" alt="Figure 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pbio-1000555-g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Sedimentation equilibrium centrifugation was performed on 40 µM SaMscL in the neutrally buoyant detergent C<sub>8</sub>E<sub>5</sub>. The upper two panels in (A) and (B) show scans from three different rotor speeds monitored at either 250 nm (A) or 280 nm (B). The lines represent fits to the data of a single species model that yield a calculated protein mass of 71.2 kDa with a local rmsd of 0.005799 (A) and 0.005059 (B) (SaMscL monomer mass is 14.4 kDa; pentamer mass is 72.2 kDa). The lower two panels in (A) and (B) show the residuals from the data fitting. A representative SEC-MALS experiment of SaMscL in C<sub>8</sub>E<sub>5</sub> is shown in (C) with the UV chromatogram colored blue and the 90° light scattering chromatogram colored red. The total computed mass is shown as the black line across the elution peak, with the SaMscL and C<sub>8</sub>E<sub>5</sub> mass shown in green and orange, respectively.</p></figcaption></figure><p>The data presented in <a href="#pbio-1000555-g002" class="usa-link">Figures 2</a> and <a href="#pbio-1000555-g003" class="usa-link">3</a> show three independent approaches that all confirm the pentameric oligomeric state of SaMscL, in agreement with our in vivo crosslinking results. These data are also consistent with the MtMscL crystal structure <a href="#pbio.1000555-Chang1" class="usa-link" aria-describedby="pbio.1000555-Chang1">[6]</a>,<a href="#pbio.1000555-Steinbacher1" class="usa-link" aria-describedby="pbio.1000555-Steinbacher1">[13]</a> and biochemical studies of EcoMscL <a href="#pbio.1000555-Sukharev1" class="usa-link" aria-describedby="pbio.1000555-Sukharev1">[7]</a> but inconsistent with the subunit organization in the SaMscL crystal structure <a href="#pbio.1000555-Liu1" class="usa-link" aria-describedby="pbio.1000555-Liu1">[8]</a>. The major difference in the protein handling between our experiments and those previously published for the SaMscL channel <a href="#pbio.1000555-Liu1" class="usa-link" aria-describedby="pbio.1000555-Liu1">[8]</a> was the choice of solubilizing detergent. Our measurements were made in the presence of either Triton X-100 (crosslinking) or pentaethylene glycol monooctyl ether (C<sub>8</sub>E<sub>5</sub>) detergent (SE and SEC-MALS), not Lauryldimethylamine-oxide (LDAO), the detergent used previously to crystallize SaMscL and evaluate its oligomeric state <a href="#pbio.1000555-Liu1" class="usa-link" aria-describedby="pbio.1000555-Liu1">[8]</a>. To determine if the solubilizing detergent could account for the discrepancies in the data, we tested whether LDAO may affect the oligomeric state of SaMscL.</p>
<p>We performed SaMscL crosslinking experiments in the presence of LDAO and found essentially no crosslinked pentamer and a substantial increase in tetramers when compared to the experiments performed in Triton X-100 (<a href="#pbio-1000555-g004" class="usa-link">Figure 4A</a>). To confirm that LDAO was altering the oligomeric state of the SaMscL channel rather than simply affecting our crosslinking efficiency, we directly measured the mass of the SaMscL protein in the presence of LDAO by SEC-MALS (representative experiment shown in <a href="#pbio-1000555-g004" class="usa-link">Figure 4B</a>). The results show that the average protein mass in the presence of LDAO is 60.0±2.8 kDa (Ave ± SD, <em>n</em> = 8), a mass that is dramatically reduced compared to the C<sub>8</sub>E<sub>5</sub> experiments (72.8±0.8 kDa) and that is consistent with a tetrameric oligomeric state (theoretical SaMscL tetramer is 57.8 kDa). As a third independent approach to assess the oligomeric state of the protein, we performed sedimentation velocity experiments on the SaMscL protein in LDAO. Two different methods were used to calculate the molar mass of the protein from these data (see <a href="#s4" class="usa-link">Methods</a> for details). Both resulted in similar values: 62.3±1.3 kDa (“<em>s</em> and <em>D</em>” method, <a href="#pbio-1000555-g004" class="usa-link">Figure 4C</a>) and 64±3 kDa (“Stokes radius” method) (Weighted Ave ± 1σ). These results again show a protein molar mass well below that of a pentamer, consistent with LDAO affecting the oligomeric state of the SaMscL channel.</p>
<figure class="fig xbox font-sm" id="pbio-1000555-g004"><h3 class="obj_head">Figure 4. LDAO stabilizes a MscL tetrameric oligomeric state as measured by crosslinking, SEC-MALS, and Sedimentation Velocity Centrifugation.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2998437_pbio.1000555.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/6f7302a3692b/pbio.1000555.g004.jpg" loading="lazy" height="740" width="656" alt="Figure 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pbio-1000555-g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Western blot analysis of DSS-treated SaMscL solubilized with either Triton-X100 or LDAO yield different quantities of tetramers and pentamers. (B) A representative SEC-MALS experiment of SaMscL in LDAO is shown with the UV chromatogram colored blue and the 90° light scattering chromatogram colored red. The total, SaMscL, and LDAO masses are shown as black, green, and orange lines across the elution peak, respectively. The mass of the SaMscL monomer is 14.4 kDa and the tetramer mass is 57.8 kDa. (C) SaMscL in 4 mM LDAO was subjected to sedimentation velocity centrifugation at 50,000 rpm, and the data were analyzed using the noninteracting “species model” of SEDPHAT. Four species, including the detergent micelles, were analyzed. The molar mass of the dominant sedimenting species was 62.5 kDa. In the <em>upper part</em>, the individual data points are depicted as circles, and the best-fit model to those data is shown as lines. The data and fit lines are color coded by color: Violet for the earliest scans, then progressing through indigo, blue, green, yellow, orange, and red as the scans go further forward in time. For clarity only every other scan used in the data analysis is shown. In the <em>lower part</em>, the residuals between the data points and the fitted line are shown and color coded as above.</p></figcaption></figure><p>We also tested the reversibility of this oligomeric state reorganization by using SaMscL purified in LDAO and then exchanged the LDAO detergent for C<sub>8</sub>E<sub>5</sub>. This protocol resulted in the SaMscL mass increasing from 61.4±0.7 kDa (Ave ± SD, <em>n</em> = 3) to 72.0±2.6 kDa (Ave ± SD, <em>n</em> = 4) as measured by SEC-MALS (<a href="#pbio-1000555-g005" class="usa-link">Figure 5</a>). The choice of initial detergent made no difference in the reversibility, as purifying SaMscL in C<sub>8</sub>E<sub>5</sub> and then exchanging the detergent for LDAO showed a mass decrease from 72.2±0.5 kDa in C<sub>8</sub>E<sub>5</sub> (Ave ± SD, <em>n</em> = 5) to 58.7±0.7 kDa in LDAO (Ave ± SD, <em>n</em> = 3). Hence, the oligomeric rearrangement is reversible and is completely dependent upon the detergent solubilizing the MscL protein.</p>
<figure class="fig xbox font-sm" id="pbio-1000555-g005"><h3 class="obj_head">Figure 5. The <em>S. aureus</em> MscL stoichiometry change is reversible.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2998437_pbio.1000555.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/e493ae495bef/pbio.1000555.g005.jpg" loading="lazy" height="672" width="467" alt="Figure 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/pbio-1000555-g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) The oligomeric state of SaMscL in 4 mM LDAO was determined using SEC-MALS, which measured a protein mass of 61.3 kDa, consistent with a tetrameric channel (top chromatogram). (B) The SaMscL-LDAO sample was exchanged into C<sub>8</sub>E<sub>5</sub>, and the following day the oligomeric state of SaMscL-C<sub>8</sub>E<sub>5</sub> sample was measured by SEC-MALS. Analysis of the SEC-MALS results for the SaMscL-C<sub>8</sub>E<sub>5</sub> sample showed a protein mass of 74.4 kDa, consistent with the channel in a pentameric state.</p></figcaption></figure></section><section id="s3"><h2 class="pmc_sec_title">Discussion</h2>
<p>Essentially all previous structural studies on MscL, as with most membrane proteins, have been performed on channels that have been solubilized, under the assumption that detergents do not alter the subunit stoichiometry. Our data examining the effect of different detergents on the oligomeric state of SaMscL have far-reaching implications for membrane protein research, as we now clearly demonstrate that changes in oligomeric state can and do occur upon detergent solubilization, and these alterations are not easily detected by SEC (see <a href="#pbio-1000555-g003" class="usa-link">Figures 3C</a> and <a href="#pbio-1000555-g004" class="usa-link">4B</a>). To circumvent these artifacts associated with solubilization, we have developed an in vivo cross-linking assay. Although the pentamer is by far the major product observed in <a href="#pbio-1000555-g001" class="usa-link">Figure 1B</a>, very small amounts of smaller oligomers can be resolved upon overloading of protein gels and long Western blot exposures. The small amount of tetramers and smaller complexes observed upon long exposure could be incompletely assembled protein or may simply be due to the lack of disulfide bridging, which is rarely observed to be this efficient (e.g. see <a href="#pbio.1000555-Iscla1" class="usa-link" aria-describedby="pbio.1000555-Iscla1">[9]</a>,<a href="#pbio.1000555-Li1" class="usa-link" aria-describedby="pbio.1000555-Li1">[10]</a>). In addition, we cannot completely rule out the possibility that oligomerization is a dynamic and even reversible process in vivo. Regardless, it seems unlikely that oligomeric species smaller than pentamers have a physiological role given that there are tens of channels per cell <a href="#pbio.1000555-Blount4" class="usa-link" aria-describedby="pbio.1000555-Blount4">[14]</a>, and the smaller species compose only a few percent of the total channels. Our data definitively show that the vast majority of the SaMscL channel adopts a pentameric structure in the cell membrane, consistent with the MtMscL crystal structure <a href="#pbio.1000555-Chang1" class="usa-link" aria-describedby="pbio.1000555-Chang1">[6]</a>, but incompatible with the SaMscL crystal structure <a href="#pbio.1000555-Liu1" class="usa-link" aria-describedby="pbio.1000555-Liu1">[8]</a>.</p>
<p>A close inspection of the recent tetrameric SaMscL structure, which was speculated to be in an expanded gating intermediate state, reveals structural interactions between residues in TM1 and TM2 residues of a neighboring subunit similar to those in the closed MtMscL structure. This result was surprising because several studies using various techniques have suggested TM1 undergoes a significant clockwise rotation (as viewed from the periplasmic side) upon channel gating. The approaches supporting this interpretation include: spin labeling combined with electron paramagnetic resonance (EPR) studies <a href="#pbio.1000555-Perozo1" class="usa-link" aria-describedby="pbio.1000555-Perozo1">[15]</a>, accessibility of sulfhydryl reagents to cysteine mutants in TM1 upon gating <a href="#pbio.1000555-Bartlett1" class="usa-link" aria-describedby="pbio.1000555-Bartlett1">[16]</a>,<a href="#pbio.1000555-Bartlett2" class="usa-link" aria-describedby="pbio.1000555-Bartlett2">[17]</a>, accessibility of heavy metals to engineered binding sites in the pore <a href="#pbio.1000555-Iscla2" class="usa-link" aria-describedby="pbio.1000555-Iscla2">[18]</a>, suppression mutagenesis demonstrating interactions between TM1 and TM2 upon gating <a href="#pbio.1000555-Li2" class="usa-link" aria-describedby="pbio.1000555-Li2">[19]</a>, and confirmation of multiple interaction sites between TM1 and TM2 upon gating by using an electrostatic repulsion approach as well as disulfide trapping <a href="#pbio.1000555-Li1" class="usa-link" aria-describedby="pbio.1000555-Li1">[10]</a>. Both the EPR studies as well as the heavy-metal binding in the pore strongly suggest that this rotation occurs quite early in the gating process, before ion permeation. By this model, V21 and G24 (V23 and G26 in EcoMscL) should rotate away from the pore constriction, allowing V22 and the positively charged K29 (I24 and K31 in EcoMscL) to line the open pore. The observation that this rearrangement has not occurred strongly suggests that the crystal SaMscL structure actually reflects a strained closed state due to improper oligomeriazation, rather than an expanded intermediate state.</p>
<p>The observed stoichiometry changes in detergents also evoke questions regarding how membrane protein subunits assemble into physiological multimeric complexes. A previous report on EcoMscL demonstrated that when the protein is translated in vitro in the absence of any lipid or detergent and is subsequently purified in the presence of Triton X-100, the channel spontaneously assembles into pentamers <a href="#pbio.1000555-Berrier1" class="usa-link" aria-describedby="pbio.1000555-Berrier1">[20]</a>. A separate study demonstrated that even synthetically synthesized EcoMscL will oligomerize into functional channels when incorporated into lipid membranes <a href="#pbio.1000555-Clayton1" class="usa-link" aria-describedby="pbio.1000555-Clayton1">[21]</a>. These studies demonstrate that the EcoMscL channel has the ability to self-assemble into the functional oligomer in vitro. Similarly, it appears that C<sub>8</sub>E<sub>5</sub> and Triton X-100 allow SaMscL to properly fold and assemble into a pentameric structure, which correlates with the observed oligomeric state in vivo. In contrast, LDAO reorganizes SaMscL into tetramers that do not appear to exist in significant quantities in vivo, suggesting that they are not physiologically relevant. The observed self-assembly properties of EcoMscL under many conditions, combined with the reversible nature of the detergent-induced oligomeric state changes for SaMscL, may explain why Liu et al. observed channel function for the LDAO solubilized SaMscL tetramers <a href="#pbio.1000555-Liu1" class="usa-link" aria-describedby="pbio.1000555-Liu1">[8]</a>; presumably, once the tetrameric LDAO-solubilized SaMscL is reconstituted into lipids, it rearranges back into functional pentameric structures.</p>
<p>The crystal structure of the SaMscL was derived from a truncation mutant in which several amino acids at the C-terminus had been removed <a href="#pbio.1000555-Liu1" class="usa-link" aria-describedby="pbio.1000555-Liu1">[8]</a>. One previous study suggested that at least some C-terminus deletions of EcoMscL could lead to the formation of larger aggregates when translated in a cell-free system <a href="#pbio.1000555-Yoshimura1" class="usa-link" aria-describedby="pbio.1000555-Yoshimura1">[22]</a>. While we cannot rule out the possibility that the SaMscL deletion contributed to the stabilization of the tetramer in the crystal, our data do demonstrate that detergents alone can stabilize alternative oligomeric states of the full-length SaMscL in solution.</p>
<p>Our observations make it tempting to speculate that larger, more hydrated head groups (i.e. the head groups of C<sub>8</sub>E<sub>5</sub> and Triton X-100) promote pentamer formation, while smaller head groups (i.e. LDAO) promote tetramer formation. However, with such a small number of detergents tested thus far, additional experiments will be required to identify what chemical properties of the detergents cause the oligomeric state change and what effect, if any, lipids may have on the stabilization of specific oligomeric states.</p>
<p>In sum, our data demonstrate that the physiologically active SaMscL oligomer in vivo is a pentamer and that purification in detergents can cause a reorganization of the SaMscL channel's oligomeric state. Our findings also show that SEC alone is insufficient to evaluate the usefulness of a particular detergent for the study of membrane proteins; ideally, SEC-MALS and/or analytical ultracentrifugation should be correlated with in vivo experiments that can more confidently determine the functional and physiologically relevant oligomeric state of membrane proteins.</p></section><section id="s4"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="s4a"><h3 class="pmc_sec_title">Purification</h3>
<p>SaMscL (accession number: <a href="https://www.ncbi.nlm.nih.gov/protein/CAG43065" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CAG43065</a>) was expressed using the pET21a vector in PB116 <em>E. coli</em> cells grown to O.D.<sub>600</sub> 0.6–0.8 and induced with 1 mM IPTG for 2–3 h at 37°C. Cells were harvested and stored at −80°C until needed. Cells containing SaMscL were lysed in 50 mM NaPi pH 8.0, 300 mM NaCl, 0.5 µg/ml DNAse, 1 mg/ml lysozyme, and 150 µl protease inhibitors (Sigma) using a French press at 16K PSI at 4°C. The cell lysate was incubated with 40 mM n-Decyl-β-D-Maltopyranoside (DM) for 1 h at 4°C and then centrifuged at 24,000× g for 30 min at 4°C to clarify the lysate. 4 mM imidazol was added to the resulting supernatant and then bound to 4 ml of Ni-NTA slury pre-equilibrated with 50 mM NaPi pH 8.0, 300 mM NaCl, 5 mM imidazol, and 40 mM DM for 1 h at 4°C. Standard metal affinity chromatography was performed with 10 column volumes of wash buffer containing 50 mM NaPi pH 8.0, 300 mM NaCl, 30 mM imidazol, 4 mM DM, and SaMscL was eluted with 50 mM NaPi pH 8.0, 300 mM NaCl, 300 mM imidazol, and 4 mM DM. The purest protein fractions were combined and concentrated using a 10,000 molecular weight cutoff filter (Amicon). Detergent exchange was accomplished by adding 40 mM Lauryldimethylamine-oxide (LDAO) or 5% (w/v) pentaethylene glycol monooctyl ether (C<sub>8</sub>E<sub>5</sub>) to the concentrated SaMscL and allowing the protein-detergent complex to equilibrate overnight at room temperature. The following day the protein was subjected to size exclusion chromatography using a superdex 200 column (GE Healthcare) in 20 mM NaPi pH 7.5, 170 mM NaCl, and 4 mM LDAO or 0.5% (w/v) C<sub>8</sub>E<sub>5</sub>. The eluting peak was collected and concentrated using a 50,000 molecular weight cutoff filter (Amicon) and used for either sedimentation equilibrium experiments or size exclusion chromatography-multiangle light scattering experiments (SEC-MALS).</p></section><section id="s4b"><h3 class="pmc_sec_title">Crosslinking</h3>
<p>SaMscL and EcoMscL were cloned into pB10 and pET21a vectors and transformed into PB104 or PB116 <em>E. coli</em> and purified as previously described <a href="#pbio.1000555-Li1" class="usa-link" aria-describedby="pbio.1000555-Li1">[10]</a>. 2% Triton X-100 or 44 mM LDAO was used for MscL extraction with a final concentration of 0.2% and 2.1 mM, respectively, after purification. For crosslinking reactions 5 µg of protein was incubated with 2 mM DSS (Thermo Scientific) on ice for 5 min. The reaction was quenched with a final concentration of 100 mM Tris pH 7.5 for 15 min at room temperature as recommended by the manufacturer. Samples were brought up in non-reducing sample buffer, were loaded on a Criterion 4–20% gel, and were subjected to Western as previously described <a href="#pbio.1000555-Li1" class="usa-link" aria-describedby="pbio.1000555-Li1">[10]</a>. The primary antibody anti-Penta His (Qiagen) was used at 1∶4,000 and the secondary Goat anti-Mouse HRP (Bio-Rad) at 1∶100,000. Blots were developed using HRP substrate (Millipore) and exposed to film.</p></section><section id="s4c"><h3 class="pmc_sec_title">In Vivo Disulfide Trapping</h3>
<p>Overnight cultures were diluted 1∶100 and grown 1 h at 37°C in LB. LB with 1 M NaCl was then added for a final concentration of 0.5 M. Cultures were then induced with 1 mM IPTG for 1 h when an OD 600 of 0.2 was reached. Cultures were either Mock shocked (0.5 M NaCl LB) or shocked (water with 1.5 µM copper phenanthroline) at a 1∶20 dilution for 15 min at 37°C. Samples were spun down at 4,000 g for 20 min and immediately re-suspended in non-reducing sample buffer, adjusted for final OD, and run on a 4%–20% gel (Bio-Rad) for Western blot analysis <a href="#pbio.1000555-Li1" class="usa-link" aria-describedby="pbio.1000555-Li1">[10]</a>. The primary antibody anti-Penta His (Qiagen) was used at 1∶4,000 and the secondary Goat anti-Mouse HRP (Bio-Rad) at 1∶40,000. Blots were developed using HRP substrate (Millipore) and exposed to film.</p></section><section id="s4d"><h3 class="pmc_sec_title">SEC-MALS</h3>
<p>A Shimadzu Prominence HPLC system equipped with a Shodex KW-803 SEC column was connected in line with a miniDAWN-TREOS light scattering instrument (Wyatt Technologies) and Optilab rEX refractometer (Wyatt Technologies). The light scattering and refractive index instruments were calibrated following the manufacturer's guidelines, and 40 ml of running buffer (20 mM NaPi pH 7.5, 170 mM NaCl and 4 mM LDAO or 0.5% (w/v) C<sub>8</sub>E<sub>5</sub>) was used to equilibrate the SEC column and establish stable baselines for the light scattering and refractive index instruments. 25–75 µg of purified protein was used for each SEC-MALS experiment, and data were collected using ASTRA V software (Wyatt Technologies) and processed following the manufacturer's guidelines. LDAO and C<sub>8</sub>E<sub>5</sub> dn/dc values (0.1490 and 0.1240, respectively) were measured using the Optilab rEX following the manufacturer's guidelines and the SaMscL extinction coefficient was calculated from the amino acid sequence.</p></section><section id="s4e"><h3 class="pmc_sec_title">Analytical Ultracentrifugation</h3>
<p>Buffer density-matching experiments were performed in a Beckman Coulter XL-I Analytical Ultracentrifuge to identify the appropriate solution density to match the density of the C<sub>8</sub>E<sub>5</sub> detergent. Volumes of 180 µL of solutions containing 20 mM NaPi pH 7.5, 0.5% (w/v) C<sub>8</sub>E<sub>5</sub>, and various concentrations of NaCl were placed in the sample sector of a dual-sectored, charcoal-filled Epon centerpiece (Beckman-Coulter; all AUC experiments were carried out in such centerpieces) that was sandwiched between sapphire windows. Identical buffers without detergent were placed in the reference sector. The centrifugation cells were centrifuged at a temperature of 25°C and a rotor speed of 50,000 rpm; an An-50Ti rotor was used for all centrifugation experiments. The formation of any detergent concentration gradient was monitored using interference optics. C<sub>8</sub>E<sub>5</sub> was found to be neutrally buoyant in 20 mM NaPi pH 7.5, 170 mM NaCl, which was then used for all sedimentation equilibrium centrifugation experiments with 0.5% C<sub>8</sub>E<sub>5</sub> at 25°C. 4, 8, 40, and 80 µM SaMscL samples were prepared and loaded into the assembled ultracentrifuge cells; buffer with detergent was deposited in the reference sector. Long solution columns (180 µL) were also used in this experiment. Centrifugation was performed at 25°C at 9,500, 14,000, and 17,500 rpm. The data were analyzed using SEDFIT to determine if the samples had come to equilibrium, and SEDPHAT <a href="#pbio.1000555-Vistica1" class="usa-link" aria-describedby="pbio.1000555-Vistica1">[23]</a> was used to fit the sedimentation equilibrium data to a single species model. Multispeed analysis with mass-conservation constraints was performed, allowing the time-invariant and radially invariant noise elements to be calculated and subtracted from the data <a href="#pbio.1000555-Vistica1" class="usa-link" aria-describedby="pbio.1000555-Vistica1">[23]</a>. A 68.3% confidence interval was calculated as previously described <a href="#pbio.1000555-Bevington1" class="usa-link" aria-describedby="pbio.1000555-Bevington1">[24]</a>,<a href="#pbio.1000555-Houtman1" class="usa-link" aria-describedby="pbio.1000555-Houtman1">[25]</a>. The error number reported is the average of the positive and negative deviations from best mass value. For the determination of the amount of detergent bound per gram of protein (δ<em><sub>D</sub></em>), buffer without detergent was used in the reference sector, and two different concentrations (41 and 66 µM) of MscL were placed in the sample sector. The volume of each sample and reference was 390 µl. The rotor speed used was 50,000 rpm, and the temperature setting was 20°C. The data were analyzed in SEDFIT using the non-interacting discrete species model <a href="#pbio.1000555-Brown1" class="usa-link" aria-describedby="pbio.1000555-Brown1">[26]</a>,<a href="#pbio.1000555-Schuck1" class="usa-link" aria-describedby="pbio.1000555-Schuck1">[27]</a>. The flotation of the detergent micelles (the density of LDAO is less than that of water) was explicitly accounted for by giving that species a negative sedimentation coefficient. The amounts of absorbance units and interference fringes present for the main sedimenting species were derived from this analysis. Using the calculated extinction coefficient of the protein and the known refractive index increments of the detergent and the protein, <em>δ<sub>D</sub></em> was calculated as in le Maire et al. <a href="#pbio.1000555-leMaire1" class="usa-link" aria-describedby="pbio.1000555-leMaire1">[28]</a>. Subsequent sedimentation velocity experiments were performed with 20, 32, and 64 µM SaMscL samples in 4 mM LDAO at 50,000 rpm at 20°C. The volume centrifuged was 390 µL. In these cases, the reference buffer contained the same [LDAO] as the sample. The absorbance data (interference data were not collected) were initially analyzed using the <em>c</em>(<em>s</em>) model in SEDFIT <a href="#pbio.1000555-Brown1" class="usa-link" aria-describedby="pbio.1000555-Brown1">[26]</a>. Although LDAO has a very low extinction coefficient at 280 nm, its floatation was detectable; the range of data near to the meniscus was therefore eliminated for this analysis. In these analyses, small quantities (2%–3% of the total signal) of large species were detected. Using this information, all species, including the floating detergent, were modeled in the “species analysis” model in SEDPHAT <a href="#pbio.1000555-Brown1" class="usa-link" aria-describedby="pbio.1000555-Brown1">[26]</a>. This approach is referred to as the “<em>s</em> and <em>D</em>” method in the text, because both the sedimentation coefficient and the diffusion coefficient are refined to arrive at the buoyant molar mass (<em>M<sub>b</sub></em>). <em>M<sub>b</sub></em> was transformed to <em>M</em> (the molar mass of the protein) using the formula</p>
<table class="disp-formula p" id="disp-formula1"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/e54a207998f9/pbio.1000555.e001.jpg" loading="lazy" height="41" width="106" alt="graphic file with name pbio.1000555.e001.jpg"></td>
<td class="label">(1)</td>
</tr></table>
<p>where</p>
<table class="disp-formula p" id="disp-formula2"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/8485b0298955/pbio.1000555.e002.jpg" loading="lazy" height="18" width="243" alt="graphic file with name pbio.1000555.e002.jpg"></td>
<td class="label">(2)</td>
</tr></table>
<p>
<a href="#pbio.1000555-Tanford1" class="usa-link" aria-describedby="pbio.1000555-Tanford1">[29]</a>
</p>
<p>The symbol <em>ρ</em> is the solution density, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/c2f09937b05d/pbio.1000555.e003.jpg" loading="lazy" alt="Inline graphic"></span> is the partial specific volume of the protein, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/cf3ff1e4b67e/pbio.1000555.e004.jpg" loading="lazy" alt="Inline graphic"></span> is the partial specific volume of the detergent. Any lipids bound to SaMscL were neglected in these calculations. <em>M<sub>b</sub></em> was also calculated using the relationship</p>
<table class="disp-formula p" id="disp-formula3"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/27f2bd8fd046/pbio.1000555.e005.jpg" loading="lazy" height="17" width="113" alt="graphic file with name pbio.1000555.e005.jpg"></td>
<td class="label">(3)</td>
</tr></table>
<p>where <em>N</em>
<sub>0</sub> is Avogadro's number, <em>η</em> is the solution viscosity, <em>R<sub>s</sub></em> is the Stokes radius of the protein/detergent complex, and <em>s</em> is the sedimentation coefficient. The Stokes radius of the protein was obtained by calibrating the SEC column described above with proteins of known Stokes radius. <em>R<sub>s</sub></em> for SaMscL in LDAO was 4.2 nm. Again, <em>M<sub>b</sub></em> was transformed to <em>M</em> using Eq. 1. Values for <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/c2f09937b05d/pbio.1000555.e006.jpg" loading="lazy" alt="Inline graphic"></span>, <em>η</em>, and <em>ρ</em> were estimated using the program SEDNTERP <a href="#pbio.1000555-Laue1" class="usa-link" aria-describedby="pbio.1000555-Laue1">[30]</a>. The value of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/cf3ff1e4b67e/pbio.1000555.e007.jpg" loading="lazy" alt="Inline graphic"></span> was taken as 1.13 mL/g <a href="#pbio.1000555-leMaire2" class="usa-link" aria-describedby="pbio.1000555-leMaire2">[31]</a>. Confidence intervals of <em>s</em> and <em>M<sub>b</sub></em> were calculated as detailed above. A confidence interval for the value of Stokes radius, which was derived from a linear regression, was:</p>
<table class="disp-formula p" id="disp-formula4"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/6fc22f801b74/pbio.1000555.e008.jpg" loading="lazy" height="51" width="223" alt="graphic file with name pbio.1000555.e008.jpg"></td></tr></table>
<p>where <em>t</em> is the <em>t</em> statistic, <em>s<sub>s</sub></em> is the estimate of <em>σ</em> for the regression line, <em>V<sub>e</sub></em> is the elution volume of MscL from the SEC column, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/348af5018125/pbio.1000555.e009.jpg" loading="lazy" alt="Inline graphic"></span> is the average elution volume of the standards, <em>n</em> is the number of standards, and <em>S<sub>VV</sub></em> is the sum of squared deviations of the individual standard compared to <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/348af5018125/pbio.1000555.e010.jpg" loading="lazy" alt="Inline graphic"></span>. In cases of asymmetric error intervals, the <em>σ</em> was taken as the average of the difference between the boundary value and the best-fitted value. Because multiple experiments, each having different errors, were performed, the value of <em>M</em> reported in the main text is the weighted mean <em>μ<sub>w</sub></em> ± the weighted <em>σ</em> (<em>σ</em>
<sub>w</sub>), calculated using the formulas:</p>
<table class="disp-formula p" id="disp-formula5"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/55f98b81bbb7/pbio.1000555.e011.jpg" loading="lazy" height="66" width="138" alt="graphic file with name pbio.1000555.e011.jpg"></td></tr></table>
<table class="disp-formula p" id="disp-formula6"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/a0dbe1e9d508/pbio.1000555.e012.jpg" loading="lazy" height="51" width="71" alt="graphic file with name pbio.1000555.e012.jpg"></td></tr></table>
<p>for the <em>I</em> experiments preformed (<em>i</em> = 3 in this case).</p></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors thank Christina Eaton and Drs. Irene Iscla, Yuezhou Li, and Limin Yang for helpful discussions.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>C<sub>8</sub>E<sub>5</sub>
</dt>
<dd><p>pentaethylene glycol monooctyl ether</p></dd>
<dt>EcoMscL</dt>
<dd><p>
<em>E. coli</em> MscL</p></dd>
<dt>LDAO</dt>
<dd><p>n-dodecyl-N,N-dimethylamine-N-oxide</p></dd>
<dt>MtMscL</dt>
<dd><p>
<em>M. tuberculosis</em> MscL</p></dd>
<dt>SaMscL</dt>
<dd><p>
<em>S. auresus</em> MscL</p></dd>
<dt>SDS</dt>
<dd><p>sodium dodecyl sulphate</p></dd>
<dt>SE</dt>
<dd><p>sedimentation equilibrium analytical ultracentrifugation</p></dd>
<dt>SEC-MALS</dt>
<dd><p>size exclusion chromatography multi-angle light scattering</p></dd>
</dl></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn2"><p>The authors have declared that no competing interests exist.</p></div>
<div class="fn p" id="fn3"><p>MRD and YJ were supported by the Howard Hughes Medical Institute, the National Institutes of Health (grant GM079179 to YJ) and the Welch Foundation (grant I-1578 to YJ). RW and PB were supported by grant I-1420 of the Welch Foundation, grant NNH08ZTT003N NRA from NASA, grant RP100146 from the Cancer Prevention &amp; Research Institute of Texas (CPRIT), and grant GM61028 and its supplement from the National Institutes of Health. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div>
</div></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pbio.1000555-Levina1">
<span class="label">1.</span><cite>Levina N, Totemeyer S, Stokes N. R, Louis P, Jones M. A, et al.  Protection of Escherichia coli cells against extreme turgor by activation of MscS and MscL mechanosensitive channels: identification of genes required for MscS activity. EMBO Journal. 1999;18:1730–1737. doi: 10.1093/emboj/18.7.1730.</cite> [<a href="https://doi.org/10.1093/emboj/18.7.1730" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1171259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10202137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Journal&amp;title=Protection%20of%20Escherichia%20coli%20cells%20against%20extreme%20turgor%20by%20activation%20of%20MscS%20and%20MscL%20mechanosensitive%20channels:%20identification%20of%20genes%20required%20for%20MscS%20activity.&amp;author=N%20Levina&amp;author=S%20Totemeyer&amp;author=N.%20R%20Stokes&amp;author=P%20Louis&amp;author=M.%20A%20Jones&amp;volume=18&amp;publication_year=1999&amp;pages=1730-1737&amp;pmid=10202137&amp;doi=10.1093/emboj/18.7.1730&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Blount1">
<span class="label">2.</span><cite>Blount P, Iscla I, Moe P. C, Li Y.  MscL: the bacterial mechanosensitive channel of large conductance. In: Hamill O. P, editor. Mechanosensitive ion channels (a volume in the current topics in membranes series) St. Louis, MO: Elsievier Press; 2007. pp. 202–233.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Mechanosensitive%20ion%20channels%20(a%20volume%20in%20the%20current%20topics%20in%20membranes%20series)&amp;author=P%20Blount&amp;author=I%20Iscla&amp;author=P.%20C%20Moe&amp;author=Y%20Li&amp;publication_year=2007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Blount2">
<span class="label">3.</span><cite>Blount P, Li Y, Moe P. C, Iscla I.  Mechanosensitive channels gated by membrane tension: bacteria and beyond. In: Kamkin A, Kiseleva I, editors. Mechanosensitive ion channels (a volume in the mechanosensitivity in cells and tissues, Moscow Academia series) New York: Springer Press; 2008. pp. 71–101.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Mechanosensitive%20ion%20channels%20(a%20volume%20in%20the%20mechanosensitivity%20in%20cells%20and%20tissues,%20Moscow%20Academia%20series)&amp;author=P%20Blount&amp;author=Y%20Li&amp;author=P.%20C%20Moe&amp;author=I%20Iscla&amp;publication_year=2008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Blount3">
<span class="label">4.</span><cite>Blount P, Sukharev S. I, Moe P. C, Schroeder M. J, Guy H. R, et al.  Membrane topology and multimeric structure of a mechanosensitive channel protein of Escherichia coli. EMBO Journal. 1996;15:4798–4805.</cite> [<a href="/articles/PMC452216/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8890153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Journal&amp;title=Membrane%20topology%20and%20multimeric%20structure%20of%20a%20mechanosensitive%20channel%20protein%20of%20Escherichia%20coli.&amp;author=P%20Blount&amp;author=S.%20I%20Sukharev&amp;author=P.%20C%20Moe&amp;author=M.%20J%20Schroeder&amp;author=H.%20R%20Guy&amp;volume=15&amp;publication_year=1996&amp;pages=4798-4805&amp;pmid=8890153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Saint1">
<span class="label">5.</span><cite>Saint N, Lacapere J. J, Gu L. Q, Ghazi A, Martinac B, et al.  A hexameric transmembrane pore revealed by two-dimensional crystallization of the large mechanosensitive ion channel (MscL) of Escherichia coli. Journal of Biological Chemistry. 1998;273:14667–14670. doi: 10.1074/jbc.273.24.14667.</cite> [<a href="https://doi.org/10.1074/jbc.273.24.14667" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9614061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=A%20hexameric%20transmembrane%20pore%20revealed%20by%20two-dimensional%20crystallization%20of%20the%20large%20mechanosensitive%20ion%20channel%20(MscL)%20of%20Escherichia%20coli.&amp;author=N%20Saint&amp;author=J.%20J%20Lacapere&amp;author=L.%20Q%20Gu&amp;author=A%20Ghazi&amp;author=B%20Martinac&amp;volume=273&amp;publication_year=1998&amp;pages=14667-14670&amp;pmid=9614061&amp;doi=10.1074/jbc.273.24.14667&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Chang1">
<span class="label">6.</span><cite>Chang G, Spencer R. H, Lee A. T, Barclay M. T, Rees D. C. Structure of the MscL homolog from Mycobacterium tuberculosis: a gated mechanosensitive ion channel. Science. 1998;282:2220–2226. doi: 10.1126/science.282.5397.2220.</cite> [<a href="https://doi.org/10.1126/science.282.5397.2220" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9856938/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Structure%20of%20the%20MscL%20homolog%20from%20Mycobacterium%20tuberculosis:%20a%20gated%20mechanosensitive%20ion%20channel.&amp;author=G%20Chang&amp;author=R.%20H%20Spencer&amp;author=A.%20T%20Lee&amp;author=M.%20T%20Barclay&amp;author=D.%20C%20Rees&amp;volume=282&amp;publication_year=1998&amp;pages=2220-2226&amp;pmid=9856938&amp;doi=10.1126/science.282.5397.2220&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Sukharev1">
<span class="label">7.</span><cite>Sukharev S. I, Schroeder M. J, McCaslin D. R. Stoichiometry of the large conductance bacterial mechanosensitive channel of E. coli. A biochemical study. Journal of Membrane Biology. 1999;171:183–193. doi: 10.1007/s002329900570.</cite> [<a href="https://doi.org/10.1007/s002329900570" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10501827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Membrane%20Biology&amp;title=Stoichiometry%20of%20the%20large%20conductance%20bacterial%20mechanosensitive%20channel%20of%20E.%20coli.%20A%20biochemical%20study.&amp;author=S.%20I%20Sukharev&amp;author=M.%20J%20Schroeder&amp;author=D.%20R%20McCaslin&amp;volume=171&amp;publication_year=1999&amp;pages=183-193&amp;pmid=10501827&amp;doi=10.1007/s002329900570&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Liu1">
<span class="label">8.</span><cite>Liu Z, Gandhi C. S, Rees D. C. Structure of a tetrameric MscL in an expanded intermediate state. Nature. 2009;461:120–124. doi: 10.1038/nature08277.</cite> [<a href="https://doi.org/10.1038/nature08277" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2737600/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19701184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structure%20of%20a%20tetrameric%20MscL%20in%20an%20expanded%20intermediate%20state.&amp;author=Z%20Liu&amp;author=C.%20S%20Gandhi&amp;author=D.%20C%20Rees&amp;volume=461&amp;publication_year=2009&amp;pages=120-124&amp;pmid=19701184&amp;doi=10.1038/nature08277&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Iscla1">
<span class="label">9.</span><cite>Iscla I, Wray R, Blount P. On the structure of the N-terminal domain of the MscL channel: helical bundle or membrane interface. Biophys J. 2008;95:2283–2291. doi: 10.1529/biophysj.107.127423.</cite> [<a href="https://doi.org/10.1529/biophysj.107.127423" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2517020/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18515388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=On%20the%20structure%20of%20the%20N-terminal%20domain%20of%20the%20MscL%20channel:%20helical%20bundle%20or%20membrane%20interface.&amp;author=I%20Iscla&amp;author=R%20Wray&amp;author=P%20Blount&amp;volume=95&amp;publication_year=2008&amp;pages=2283-2291&amp;pmid=18515388&amp;doi=10.1529/biophysj.107.127423&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Li1">
<span class="label">10.</span><cite>Li Y, Wray R, Eaton C, Blount P. An open-pore structure of the mechanosensitive channel MscL derived by determining transmembrane domain interactions upon gating. FASEB J. 2009;23:2197–2204. doi: 10.1096/fj.09-129296.</cite> [<a href="https://doi.org/10.1096/fj.09-129296" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2704598/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19261722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FASEB%20J&amp;title=An%20open-pore%20structure%20of%20the%20mechanosensitive%20channel%20MscL%20derived%20by%20determining%20transmembrane%20domain%20interactions%20upon%20gating.&amp;author=Y%20Li&amp;author=R%20Wray&amp;author=C%20Eaton&amp;author=P%20Blount&amp;volume=23&amp;publication_year=2009&amp;pages=2197-2204&amp;pmid=19261722&amp;doi=10.1096/fj.09-129296&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Ajouz1">
<span class="label">11.</span><cite>Ajouz B, Berrier C, Garrigues A, Besnard M, Ghazi A. Release of thioredoxin via the mechanosensitive channel MscL during osmotic downshock of Escherichia coli cells. Journal of Biological Chemistry. 1998;273:26670–26674. doi: 10.1074/jbc.273.41.26670.</cite> [<a href="https://doi.org/10.1074/jbc.273.41.26670" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9756908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Release%20of%20thioredoxin%20via%20the%20mechanosensitive%20channel%20MscL%20during%20osmotic%20downshock%20of%20Escherichia%20coli%20cells.&amp;author=B%20Ajouz&amp;author=C%20Berrier&amp;author=A%20Garrigues&amp;author=M%20Besnard&amp;author=A%20Ghazi&amp;volume=273&amp;publication_year=1998&amp;pages=26670-26674&amp;pmid=9756908&amp;doi=10.1074/jbc.273.41.26670&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Slotboom1">
<span class="label">12.</span><cite>Slotboom D. J, Duurkens R. H, Olieman K, Erkens G. B. Static light scattering to characterize membrane proteins in detergent solution. Methods. 2008;46:73–82. doi: 10.1016/j.ymeth.2008.06.012.</cite> [<a href="https://doi.org/10.1016/j.ymeth.2008.06.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18625320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=Static%20light%20scattering%20to%20characterize%20membrane%20proteins%20in%20detergent%20solution.&amp;author=D.%20J%20Slotboom&amp;author=R.%20H%20Duurkens&amp;author=K%20Olieman&amp;author=G.%20B%20Erkens&amp;volume=46&amp;publication_year=2008&amp;pages=73-82&amp;pmid=18625320&amp;doi=10.1016/j.ymeth.2008.06.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Steinbacher1">
<span class="label">13.</span><cite>Steinbacher S, Bass R, Strop P, Rees D. C.  Structures of the prokaryotic mechanosensitive channels MscL and MscS. In: Hamill O. P, editor. Mechanosensitive ion channels (a volume in the Current Topics in Membranes series) St. Louis, MO: Elsievier Press; 2007. pp. 1–20.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Mechanosensitive%20ion%20channels%20(a%20volume%20in%20the%20Current%20Topics%20in%20Membranes%20series)&amp;author=S%20Steinbacher&amp;author=R%20Bass&amp;author=P%20Strop&amp;author=D.%20C%20Rees&amp;publication_year=2007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Blount4">
<span class="label">14.</span><cite>Blount P, Sukharev S. I, Moe P. C, Martinac B, Kung C.  Mechanosensitive channels of bacteria. In: Conn P. M, editor. Methods in enzymology. San Diego, CA: Academic Press; 1999. pp. 458–482.</cite> [<a href="https://doi.org/10.1016/s0076-6879(99)94027-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9916243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20enzymology&amp;author=P%20Blount&amp;author=S.%20I%20Sukharev&amp;author=P.%20C%20Moe&amp;author=B%20Martinac&amp;author=C%20Kung&amp;publication_year=1999&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Perozo1">
<span class="label">15.</span><cite>Perozo E, Cortes D. M, Sompornpisut P, Kloda A, Martinac B. Open channel structure of MscL and the gating mechanism of mechanosensitive channels. Nature. 2002;418:942–948. doi: 10.1038/nature00992.</cite> [<a href="https://doi.org/10.1038/nature00992" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12198539/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Open%20channel%20structure%20of%20MscL%20and%20the%20gating%20mechanism%20of%20mechanosensitive%20channels.&amp;author=E%20Perozo&amp;author=D.%20M%20Cortes&amp;author=P%20Sompornpisut&amp;author=A%20Kloda&amp;author=B%20Martinac&amp;volume=418&amp;publication_year=2002&amp;pages=942-948&amp;pmid=12198539&amp;doi=10.1038/nature00992&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Bartlett1">
<span class="label">16.</span><cite>Bartlett J. L, Levin G, Blount P. An in vivo assay identifies changes in residue accessibility on mechanosensitive channel gating. Proc Natl Acad Sci U S A. 2004;101:10161–10165. doi: 10.1073/pnas.0402040101.</cite> [<a href="https://doi.org/10.1073/pnas.0402040101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC454182/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15226501/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=An%20in%20vivo%20assay%20identifies%20changes%20in%20residue%20accessibility%20on%20mechanosensitive%20channel%20gating.&amp;author=J.%20L%20Bartlett&amp;author=G%20Levin&amp;author=P%20Blount&amp;volume=101&amp;publication_year=2004&amp;pages=10161-10165&amp;pmid=15226501&amp;doi=10.1073/pnas.0402040101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Bartlett2">
<span class="label">17.</span><cite>Bartlett J. L, Li Y, Blount P. Mechanosensitive channel gating transitions resolved by functional changes upon pore modification. Biophys J. 2006;91:3684–3691. doi: 10.1529/biophysj.106.088062.</cite> [<a href="https://doi.org/10.1529/biophysj.106.088062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1630475/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16935962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Mechanosensitive%20channel%20gating%20transitions%20resolved%20by%20functional%20changes%20upon%20pore%20modification.&amp;author=J.%20L%20Bartlett&amp;author=Y%20Li&amp;author=P%20Blount&amp;volume=91&amp;publication_year=2006&amp;pages=3684-3691&amp;pmid=16935962&amp;doi=10.1529/biophysj.106.088062&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Iscla2">
<span class="label">18.</span><cite>Iscla I, Levin G, Wray R, Reynolds R, Blount P. Defining the physical gate of a mechanosensitive channel, MscL, by engineering metal-binding sites. Biophys J. 2004;87(5):3172–3180. doi: 10.1529/biophysj.104.049833.</cite> [<a href="https://doi.org/10.1529/biophysj.104.049833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1304787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15339809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Defining%20the%20physical%20gate%20of%20a%20mechanosensitive%20channel,%20MscL,%20by%20engineering%20metal-binding%20sites.&amp;author=I%20Iscla&amp;author=G%20Levin&amp;author=R%20Wray&amp;author=R%20Reynolds&amp;author=P%20Blount&amp;volume=87&amp;issue=5&amp;publication_year=2004&amp;pages=3172-3180&amp;pmid=15339809&amp;doi=10.1529/biophysj.104.049833&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Li2">
<span class="label">19.</span><cite>Li Y, Wray R, Blount P. Intragenic suppression of gain-of-function mutations in the Escherichia coli mechanosensitive channel, MscL. Mol Microbiol. 2004;53:485–495. doi: 10.1111/j.1365-2958.2004.04150.x.</cite> [<a href="https://doi.org/10.1111/j.1365-2958.2004.04150.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15228529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Microbiol&amp;title=Intragenic%20suppression%20of%20gain-of-function%20mutations%20in%20the%20Escherichia%20coli%20mechanosensitive%20channel,%20MscL.&amp;author=Y%20Li&amp;author=R%20Wray&amp;author=P%20Blount&amp;volume=53&amp;publication_year=2004&amp;pages=485-495&amp;pmid=15228529&amp;doi=10.1111/j.1365-2958.2004.04150.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Berrier1">
<span class="label">20.</span><cite>Berrier C, Park K. H, Abes S, Bibonne A, Betton J. M, et al.  Cell-free synthesis of a functional ion channel in the absence of a membrane and in the presence of detergent. Biochemistry. 2004;43:12585–12591. doi: 10.1021/bi049049y.</cite> [<a href="https://doi.org/10.1021/bi049049y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15449948/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Cell-free%20synthesis%20of%20a%20functional%20ion%20channel%20in%20the%20absence%20of%20a%20membrane%20and%20in%20the%20presence%20of%20detergent.&amp;author=C%20Berrier&amp;author=K.%20H%20Park&amp;author=S%20Abes&amp;author=A%20Bibonne&amp;author=J.%20M%20Betton&amp;volume=43&amp;publication_year=2004&amp;pages=12585-12591&amp;pmid=15449948&amp;doi=10.1021/bi049049y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Clayton1">
<span class="label">21.</span><cite>Clayton D, Shapovalov G, Maurer J. A, Dougherty D. A, Lester H. A, et al.  Total chemical synthesis and electrophysiological characterization of mechanosensitive channels from Escherichia coli and Mycobacterium tuberculosis. Proc Natl Acad Sci U S A. 2004;101:4764–4769. doi: 10.1073/pnas.0305693101.</cite> [<a href="https://doi.org/10.1073/pnas.0305693101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC387322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15041744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Total%20chemical%20synthesis%20and%20electrophysiological%20characterization%20of%20mechanosensitive%20channels%20from%20Escherichia%20coli%20and%20Mycobacterium%20tuberculosis.&amp;author=D%20Clayton&amp;author=G%20Shapovalov&amp;author=J.%20A%20Maurer&amp;author=D.%20A%20Dougherty&amp;author=H.%20A%20Lester&amp;volume=101&amp;publication_year=2004&amp;pages=4764-4769&amp;pmid=15041744&amp;doi=10.1073/pnas.0305693101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Yoshimura1">
<span class="label">22.</span><cite>Yoshimura K, Usukura J, Sokabe M. Gating-associated conformational changes in the mechanosensitive channel MscL. Proc Natl Acad Sci U S A. 2008;105:4033–4038. doi: 10.1073/pnas.0709436105.</cite> [<a href="https://doi.org/10.1073/pnas.0709436105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2268802/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18310324/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Gating-associated%20conformational%20changes%20in%20the%20mechanosensitive%20channel%20MscL.&amp;author=K%20Yoshimura&amp;author=J%20Usukura&amp;author=M%20Sokabe&amp;volume=105&amp;publication_year=2008&amp;pages=4033-4038&amp;pmid=18310324&amp;doi=10.1073/pnas.0709436105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Vistica1">
<span class="label">23.</span><cite>Vistica J, Dam J, Balbo A, Yikilmaz E, Mariuzza R. A, et al.  Sedimentation equilibrium analysis of protein interactions with global implicit mass conservation constraints and systematic noise decomposition. Anal Biochem. 2004;326:234–256. doi: 10.1016/j.ab.2003.12.014.</cite> [<a href="https://doi.org/10.1016/j.ab.2003.12.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15003564/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal%20Biochem&amp;title=Sedimentation%20equilibrium%20analysis%20of%20protein%20interactions%20with%20global%20implicit%20mass%20conservation%20constraints%20and%20systematic%20noise%20decomposition.&amp;author=J%20Vistica&amp;author=J%20Dam&amp;author=A%20Balbo&amp;author=E%20Yikilmaz&amp;author=R.%20A%20Mariuzza&amp;volume=326&amp;publication_year=2004&amp;pages=234-256&amp;pmid=15003564&amp;doi=10.1016/j.ab.2003.12.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Bevington1">
<span class="label">24.</span><cite>Bevington P, Robinson D. K. Data reduction and error analysis for the physical sciences: McGraw Hill. 2002:336.</cite> [<a href="https://scholar.google.com/scholar_lookup?Bevington%20P,%20Robinson%20D.%20K.%20Data%20reduction%20and%20error%20analysis%20for%20the%20physical%20sciences:%20McGraw%20Hill.%202002:336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Houtman1">
<span class="label">25.</span><cite>Houtman J. C, Brown P. H, Bowden B, Yamaguchi H, Appella E, et al.  Studying multisite binary and ternary protein interactions by global analysis of isothermal titration calorimetry data in SEDPHAT: application to adaptor protein complexes in cell signaling. Protein Sci. 2007;16:30–42. doi: 10.1110/ps.062558507.</cite> [<a href="https://doi.org/10.1110/ps.062558507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1794685/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17192587/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Studying%20multisite%20binary%20and%20ternary%20protein%20interactions%20by%20global%20analysis%20of%20isothermal%20titration%20calorimetry%20data%20in%20SEDPHAT:%20application%20to%20adaptor%20protein%20complexes%20in%20cell%20signaling.&amp;author=J.%20C%20Houtman&amp;author=P.%20H%20Brown&amp;author=B%20Bowden&amp;author=H%20Yamaguchi&amp;author=E%20Appella&amp;volume=16&amp;publication_year=2007&amp;pages=30-42&amp;pmid=17192587&amp;doi=10.1110/ps.062558507&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Brown1">
<span class="label">26.</span><cite>Brown P. H, Schuck P. A new adaptive grid-size algorithm for the simulation of sedimentation velocity profiles in analytical ultracentrifugation. Comput Phys Commun. 2008;178:105–120. doi: 10.1016/j.cpc.2007.08.012.</cite> [<a href="https://doi.org/10.1016/j.cpc.2007.08.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2267755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18196178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput%20Phys%20Commun&amp;title=A%20new%20adaptive%20grid-size%20algorithm%20for%20the%20simulation%20of%20sedimentation%20velocity%20profiles%20in%20analytical%20ultracentrifugation.&amp;author=P.%20H%20Brown&amp;author=P%20Schuck&amp;volume=178&amp;publication_year=2008&amp;pages=105-120&amp;pmid=18196178&amp;doi=10.1016/j.cpc.2007.08.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Schuck1">
<span class="label">27.</span><cite>Schuck P. Size-distribution analysis of macromolecules by sedimentation velocity ultracentrifugation and lamm equation modeling. Biophys J. 2000;78:1606–1619. doi: 10.1016/S0006-3495(00)76713-0.</cite> [<a href="https://doi.org/10.1016/S0006-3495(00)76713-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1300758/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10692345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Size-distribution%20analysis%20of%20macromolecules%20by%20sedimentation%20velocity%20ultracentrifugation%20and%20lamm%20equation%20modeling.&amp;author=P%20Schuck&amp;volume=78&amp;publication_year=2000&amp;pages=1606-1619&amp;pmid=10692345&amp;doi=10.1016/S0006-3495(00)76713-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-leMaire1">
<span class="label">28.</span><cite>le Maire M, Arnou B, Olesen C, Georgin D, Ebel C, et al.  Gel chromatography and analytical ultracentrifugation to determine the extent of detergent binding and aggregation, and Stokes radius of membrane proteins using sarcoplasmic reticulum Ca2+-ATPase as an example. Nat Protoc. 2008;3:1782–1795. doi: 10.1038/nprot.2008.177.</cite> [<a href="https://doi.org/10.1038/nprot.2008.177" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18974737/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&amp;title=Gel%20chromatography%20and%20analytical%20ultracentrifugation%20to%20determine%20the%20extent%20of%20detergent%20binding%20and%20aggregation,%20and%20Stokes%20radius%20of%20membrane%20proteins%20using%20sarcoplasmic%20reticulum%20Ca2+-ATPase%20as%20an%20example.&amp;author=M%20le%20Maire&amp;author=B%20Arnou&amp;author=C%20Olesen&amp;author=D%20Georgin&amp;author=C%20Ebel&amp;volume=3&amp;publication_year=2008&amp;pages=1782-1795&amp;pmid=18974737&amp;doi=10.1038/nprot.2008.177&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Tanford1">
<span class="label">29.</span><cite>Tanford C, Reynolds J. A. Characterization of membrane proteins in detergent solutions. Biochim Biophys Acta. 1976;457:133–170. doi: 10.1016/0304-4157(76)90009-5.</cite> [<a href="https://doi.org/10.1016/0304-4157(76)90009-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/135582/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&amp;title=Characterization%20of%20membrane%20proteins%20in%20detergent%20solutions.&amp;author=C%20Tanford&amp;author=J.%20A%20Reynolds&amp;volume=457&amp;publication_year=1976&amp;pages=133-170&amp;pmid=135582&amp;doi=10.1016/0304-4157(76)90009-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-Laue1">
<span class="label">30.</span><cite>Laue T. M, Shah B. D, Ridgeway R. M, Pelletier S. L.  Computer-aided interpretation of analytical sedimentation data for proteins. In: Harding S. E, Rowe A. J, Horton J. C, editors. Analytical ultracentrifugation in biochemistry and polymer science. Cambridge, UK: The Royal Society of Chemistry; 1992. pp. 90–125.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Analytical%20ultracentrifugation%20in%20biochemistry%20and%20polymer%20science&amp;author=T.%20M%20Laue&amp;author=B.%20D%20Shah&amp;author=R.%20M%20Ridgeway&amp;author=S.%20L%20Pelletier&amp;publication_year=1992&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pbio.1000555-leMaire2">
<span class="label">31.</span><cite>le Maire M, Champeil P, Moller J. V. Interaction of membrane proteins and lipids with solubilizing detergents. Biochim Biophys Acta. 2000;1508:86–111. doi: 10.1016/s0304-4157(00)00010-1.</cite> [<a href="https://doi.org/10.1016/s0304-4157(00)00010-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11090820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&amp;title=Interaction%20of%20membrane%20proteins%20and%20lipids%20with%20solubilizing%20detergents.&amp;author=M%20le%20Maire&amp;author=P%20Champeil&amp;author=J.%20V%20Moller&amp;volume=1508&amp;publication_year=2000&amp;pages=86-111&amp;pmid=11090820&amp;doi=10.1016/s0304-4157(00)00010-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLoS Biology are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pbio.1000555"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pbio.1000555.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (666.1 KB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/2998437/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/2998437/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC2998437%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC2998437/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC2998437/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC2998437/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/21151884/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC2998437/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/21151884/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC2998437/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/2998437/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="6sTZgoQGnKDXgR2ek1VaIGKRNQW7q9zswBQgJrjWeVR8WIQ1LyhPrxtezdzhjCZr">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
