
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Reanalysis of the Mars500 experiment reveals common gut microbiome alterations in astronauts induced by long-duration confinement - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="ABCB43E48E1613331543E4002658CC19.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="csbj">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC8099722/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Computational and Structural Biotechnology Journal">
<meta name="citation_title" content="Reanalysis of the Mars500 experiment reveals common gut microbiome alterations in astronauts induced by long-duration confinement">
<meta name="citation_author" content="NJB Brereton">
<meta name="citation_author_institution" content="Institut de Recherche en Biologie Végétale, University of Montreal, Montreal, QC H1X 2B2, Canada">
<meta name="citation_author" content="FE Pitre">
<meta name="citation_author_institution" content="Institut de Recherche en Biologie Végétale, University of Montreal, Montreal, QC H1X 2B2, Canada">
<meta name="citation_author" content="E Gonzalez">
<meta name="citation_author_institution" content="Canadian Centre for Computational Genomics (C3G), Department of Human Genetics, McGill University, 740 Dr. Penfield Avenue, Montréal, QC H3A 0G1, Canada">
<meta name="citation_author_institution" content="Microbiome Research Platform, McGill Interdisciplinary Initiative in Infection and Immunity (MI4), Genome Centre, McGill University, Montréal, QC, Canada">
<meta name="citation_publication_date" content="2021 Apr 23">
<meta name="citation_volume" content="19">
<meta name="citation_firstpage" content="2223">
<meta name="citation_doi" content="10.1016/j.csbj.2021.03.040">
<meta name="citation_pmid" content="33995915">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC8099722/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC8099722/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC8099722/pdf/main.pdf">
<meta name="description" content="Keywords: Microbiome, Mars, Space science, Astronaut health, 16S rRNA gene">
<meta name="og:title" content="Reanalysis of the Mars500 experiment reveals common gut microbiome alterations in astronauts induced by long-duration confinement">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Keywords: Microbiome, Mars, Space science, Astronaut health, 16S rRNA gene">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC8099722/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://pmc.ncbi.nlm.nih.gov/search/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="8099722">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1016/j.csbj.2021.03.040"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/main.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC8099722%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/8099722/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/8099722/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC8099722/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-csbj.gif" alt="Computational and Structural Biotechnology Journal logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Computational and Structural Biotechnology Journal" title="Link to Computational and Structural Biotechnology Journal" shape="default" href="http://csbj.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Comput Struct Biotechnol J</button></div>. 2021 Apr 23;19:2223–2235. doi: <a href="https://doi.org/10.1016/j.csbj.2021.03.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1016/j.csbj.2021.03.040</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://pmc.ncbi.nlm.nih.gov/search/?term=%22Comput%20Struct%20Biotechnol%20J%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Comput%20Struct%20Biotechnol%20J%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Comput%20Struct%20Biotechnol%20J%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Comput%20Struct%20Biotechnol%20J%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Reanalysis of the Mars500 experiment reveals common gut microbiome alterations in astronauts induced by long-duration confinement</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brereton%20NJB%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">NJB Brereton</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">NJB Brereton</span></h3>
<div class="p">
<sup>a</sup>Institut de Recherche en Biologie Végétale, University of Montreal, Montreal, QC H1X 2B2, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brereton%20NJB%22%5BAuthor%5D" class="usa-link"><span class="name western">NJB Brereton</span></a>
</div>
</div>
<sup>a,</sup><sup>⁎,</sup><sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pitre%20FE%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">FE Pitre</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">FE Pitre</span></h3>
<div class="p">
<sup>a</sup>Institut de Recherche en Biologie Végétale, University of Montreal, Montreal, QC H1X 2B2, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pitre%20FE%22%5BAuthor%5D" class="usa-link"><span class="name western">FE Pitre</span></a>
</div>
</div>
<sup>a,</sup><sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gonzalez%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">E Gonzalez</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">E Gonzalez</span></h3>
<div class="p">
<sup>b</sup>Canadian Centre for Computational Genomics (C3G), Department of Human Genetics, McGill University, 740 Dr. Penfield Avenue, Montréal, QC H3A 0G1, Canada</div>
<div class="p">
<sup>c</sup>Microbiome Research Platform, McGill Interdisciplinary Initiative in Infection and Immunity (MI4), Genome Centre, McGill University, Montréal, QC, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gonzalez%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">E Gonzalez</span></a>
</div>
</div>
<sup>b,</sup><sup>c,</sup><sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="af005">
<sup>a</sup>Institut de Recherche en Biologie Végétale, University of Montreal, Montreal, QC H1X 2B2, Canada</div>
<div id="af010">
<sup>b</sup>Canadian Centre for Computational Genomics (C3G), Department of Human Genetics, McGill University, 740 Dr. Penfield Avenue, Montréal, QC H3A 0G1, Canada</div>
<div id="af015">
<sup>c</sup>Microbiome Research Platform, McGill Interdisciplinary Initiative in Infection and Immunity (MI4), Genome Centre, McGill University, Montréal, QC, Canada</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>⁎</sup><p class="display-inline">Corresponding author. <span>Nicholas.brereton@umonteal.ca</span></p>
</div>
<div class="fn" id="fn1">
<sup>1</sup><p class="display-inline" id="np005">All authors contributed equally to this work.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2021 Feb 21; Revised 2021 Mar 24; Accepted 2021 Mar 25; Collection date 2021.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2021 The Author(s)</div>
<p>This is an open access article under the CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC8099722  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/33995915/" class="usa-link">33995915</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="ab005"><h2>Graphical abstract</h2>
<figure class="fig xbox font-sm" id="f0030"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b43c/8099722/8d9da1980d3c/ga1.jpg" loading="lazy" height="723" width="777" alt="graphic file with name ga1.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/f0030/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><section id="kg005" class="kwd-group"><p><strong>Keywords:</strong> Microbiome, Mars, Space science, Astronaut health, 16S rRNA gene</p></section></section><section class="abstract" id="ab010"><h2>Abstract</h2>
<p>Maintaining astronaut health throughout long-duration spaceflight is essential to the feasibility of a manned mission to Mars. The ground-based Mars500 experiment investigated long-duration health by isolating six astronauts for 520 days, the longest controlled human confinement study conducted to date. After 520 days, astronauts had uniform strength and lean body mass losses, and increased fasting plasma glucose, calprotectin, and neutrophil levels characteristic of intestinal inflammation but previous analyses revealed no common significant changes in gut microbiota. This study reanalysed data from early (days 7–45) and late (days 420–520) faecal samples and identified 408 exact sequence variants (ESVs), including 213 shared by all astronauts. Thirty-two ESVs were significantly differentially abundant over time, including depletion of keystone resistant starch degrading, anti-inflammatory and insulin sensitivity-associated species, such as <em>Faecalibacterium prausnitzii</em>, <em>Ruminococcus bromii</em>, <em>Blautia luti</em>, <em>Anaerostipes hadrus</em>, <em>Roseburia faecis</em>, and <em>Lactobacillus rogosae</em>, and enrichment of yet-to-be-cultured bacteria. Additionally, the extraordinary experimental confinement allowed observation of microbiota potentially shared between astronauts and their habitat. Forty-nine species were shared, representing 49% and 12% of the human and environmental microbiome diversity, respectively. These findings reveal the microbiota which significantly altered in relative abundance throughout confinement, including species known to influence inflammation and host glucose homeostasis consistent with astronaut symptoms. Identification of microbiome alterations after 520 days of isolation represents a missing piece connecting Mars500 astronaut physiological studies. Knowledge of the impact of long-term confinement upon the human microbiome helps to improve our understanding of how humans interact with their habitats and is a valuable step forward towards enabling long-duration spaceflight.</p></section><section id="s0005"><h2 class="pmc_sec_title">1. Introduction</h2>
<p id="p0005">The International Space Exploration Coordination Group, which now comprises 20 national space agencies, recently published the third edition of the Global Exploration Roadmap <a href="#b0005" class="usa-link" aria-describedby="b0005">[1]</a> outlining a strategy to expand human presence in the solar system and setting the surface of Mars as a collaborative target for 2030. Safeguarding astronaut health is a critical factor in meeting this ambitious target. The Canadian Space Agency currently considers the major human health risk categories associated to space flight as: musculoskeletal (reduced muscle strength, aerobic capacity and bone fragility), sensorimotor (visual impairment), metabolic (nutritional status and sickness), behavioural health and performance (stress, fatigue, cognitive deterioration and wellbeing), radiation (ultraviolet and ionising), autonomous medical care (medical access), and physiological adaptation to variable gravity and environmental stressors (including human-associated microbial communities) <a href="#b0010" class="usa-link" aria-describedby="b0010">[2]</a>. These challenges are magnified in long-duration space flight, such as during a two–three year mission to Mars, where no resupply of air, water, food or medical supplies is possible, radiation and microgravity exposure will be for longer than has yet been tested, and abortion of a mission due to crew illness will not be possible <a href="#b0015" class="usa-link" aria-describedby="b0015">[3]</a>, <a href="#b0020" class="usa-link" aria-describedby="b0020">[4]</a>.</p>
<p id="p0010">While the human gastrointestinal track (GIT) microbiome is still being explored, it is considered essential for long-term maintenance of many aspects of human health, including healthy nutritional, immune, metabolic and psychophysiological function <a href="#b0025" class="usa-link" aria-describedby="b0025">[5]</a>, <a href="#b0030" class="usa-link" aria-describedby="b0030">[6]</a>, <a href="#b0035" class="usa-link" aria-describedby="b0035">[7]</a>, <a href="#b0040" class="usa-link" aria-describedby="b0040">[8]</a>, <a href="#b0045" class="usa-link" aria-describedby="b0045">[9]</a>, <a href="#b0050" class="usa-link" aria-describedby="b0050">[10]</a>. Long-duration habitat confinement, such as that faced by astronauts during spaceflight, could interrupt the dialog between environmental microbial ecosystems and the human microbiome <a href="#b0020" class="usa-link" aria-describedby="b0020">[4]</a>. Previous research has established that confined habitats subject to strict cleaning procedures on Earth, such as intensive care units and surgical theatres, as well as those in space, can produce a unique surface environment dominated by human GIT microbiome <a href="#b0055" class="usa-link" aria-describedby="b0055">[11]</a>, <a href="#b0060" class="usa-link" aria-describedby="b0060">[12]</a>, <a href="#b0065" class="usa-link" aria-describedby="b0065">[13]</a>, <a href="#b0070" class="usa-link" aria-describedby="b0070">[14]</a>. The influence of confinement upon the microbiome has been studied recently in four astronauts over 60 days by Chen et al <a href="#b0075" class="usa-link" aria-describedby="b0075">[15]</a> using bioregenerative life support systems (confined and self-sufficient ecosystems). Although no common microbiome composition was observed over time they did report a potential non-significant trend of depletion of <em>Faecalibacterium prausnitzii</em> as well as large individual differences between individuals (expected in microbiome studies with very low participant numbers). Whether confinement can influence the GIT microbiome of long-term inhabitants is an important priority for future space missions <a href="#b0080" class="usa-link" aria-describedby="b0080">[16]</a> and was directly addressed in the longest-running human confined habitat experiment conducted to date, the Mars500 experiment <a href="#b0085" class="usa-link" aria-describedby="b0085">[17]</a>.</p>
<p id="p0015">The Mars500 experiment isolated six astronauts within an analogue Mars-surface habitat. After 520 days, the six astronauts emerged as largely healthy, but the long-term confinement had significantly altered some of their physiology. Research conducted by Strollo et al. <a href="#b0090" class="usa-link" aria-describedby="b0090">[18]</a> measured significant decreases in total body mass (−9.3%), BMI (−5.4%) and lean (non-fat) mass (−11.8%) by day 417 of isolation. When considered alongside significant increases in fasting plasma glucose from a healthy median of 4.65 mmol/L at the beginning of the experiment to prediabetic levels of 6.02 mmol/L (impaired fasting glucose) <a href="#b0095" class="usa-link" aria-describedby="b0095">[19]</a>, these findings are indicative of potential disruption in glucose metabolism and insulin sensitivity in astronauts after extensive confinement. This aligned with significant losses of up to 22% of quadriceps/hamstring strength in astronauts, observed by Gaffney et al. <a href="#b0100" class="usa-link" aria-describedby="b0100">[20]</a>. In parallel, Roda et al. <a href="#b0105" class="usa-link" aria-describedby="b0105">[21]</a> observed that the loss of body mass was accompanied by a significant increase in the faecal protein calprotectin, the intestinal inflammation biomarker making up 60% of neutrophils (the cells which characterise intestinal inflammation <a href="#b0110" class="usa-link" aria-describedby="b0110">[22]</a>); from calprotectin negative in astronauts at the beginning of the experiment, towards varying degrees of positive by day 475. Further supporting this general trend towards an onset of intestinal inflammation, Yi et al. <a href="#b0115" class="usa-link" aria-describedby="b0115">[23]</a> directly observed a significant increase in proinflammatory cytokines and neutrophils in the Mars500 astronauts after the 520 days of isolation, symptoms which are commonly observed in astronauts during space flight <a href="#b0120" class="usa-link" aria-describedby="b0120">[24]</a>. While Strollo et al. <a href="#b0090" class="usa-link" aria-describedby="b0090">[18]</a> hypothesised that the observed symptoms accompanied by an increase in calprotectin in later faecal samples could be indicative of common metabolic alterations associated with intestinal inflammation, they discounted any microbiome interactions due to prior research conducted by Turroni et al. <a href="#b0085" class="usa-link" aria-describedby="b0085">[17]</a>, which did not identify significant microbiome changes over time and found only 14 OTUs (operational taxonomic units, representing putative microbiome species) were shared between the six astronauts. Here, the potential for common microbiome change in the Mars500 astronauts over time is reassessed using improved 16S rRNA gene amplicon bioinformatics technology <a href="#b0060" class="usa-link" aria-describedby="b0060">[12]</a> and a common data normalisation strategy for all six astronauts.</p></section><section id="s0010"><h2 class="pmc_sec_title">2. Materials and methods</h2>
<section id="s0015"><h3 class="pmc_sec_title">2.1. Sample collection and microbiome sequencing</h3>
<p id="p0020">The Mars500 experiment was a ground-based human isolation study conducted over 520 days which aimed to simulate a manned mission to Mars <a href="#b0085" class="usa-link" aria-describedby="b0085">[17]</a>, <a href="#b0100" class="usa-link" aria-describedby="b0100">[20]</a>, <a href="#b0125" class="usa-link" aria-describedby="b0125">[25]</a>. The study participants were all male and their ages ranged from 29 to 40. Faecal samples used for analysis here were taken on the days 7, 14, 21, 30 and 45 (early) and days 420, 450, 490, 504, 520 of the Mars500 experiment (<a href="#s0110" class="usa-link">Supplementary file 1</a>). Sampling, DNA extraction, amplification and sequencing approaches were not conducted in this study but are reported in <a href="#b0085" class="usa-link" aria-describedby="b0085">[17]</a>. Briefly, the authors report that total DNA was isolated from faeces using a modified bead-beating and column extraction with QIAamp DNA Stool miniKit. Amplification used 341F (5′-CCTACGGGNGGCWGCAG-3′) and 805R (5′-GACTACHVGGGTATCTAATCC-3′) primers targeting the V3-V4 hypervariable region and amplicons were sequenced using an Illumina MiSeq platform (2 × 300 bp paired-end). Raw sequences available in NCBI SRA BioProject PRJNA358005 or from <a href="https://genelab.nasa.gov/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://genelab.nasa.gov/</a>.</p>
<p id="p0025">Mars500 habitat environmental samples used for analysis were taken from surfaces using swabs on the days 14, 44, 436, 467, 496 to correspond to faecal microbiome sampling and were from the utility, medical or habitat modules (<a href="#s0110" class="usa-link">Supplementary file 1</a>). Sampling, DNA extraction, amplification and sequencing approaches were not conducted in this study but are reported in <a href="#b0125" class="usa-link" aria-describedby="b0125">[25]</a>. Briefly, the authors report that total DNA was isolated from surfaces using a 552C regular ethylene oxide sterilized swab moistened with PCR grade H<sub>2</sub>O. Total DNA was extracted using a FastDNA SPIN Kit. Amplification used 515f (5′-GTGYCAGCMGCCGCGGTAA-3′) and 926r (5′-CCGYCAATTYMTTTRAGTTT-3′) primers targeting the V4 hypervariable region and amplicons were sequenced using an Illumina MiSeq platform (2 × 300 bp paired-end). Raw sequences were deposited in the European Nucleotide Archive accession number: PRJEB21072.</p></section><section id="s0020"><h3 class="pmc_sec_title">2.2. Data processing</h3>
<p id="p0030">Amplicon sequence reads were processed and annotated using Anchor <a href="#b0060" class="usa-link" aria-describedby="b0060">[12]</a>, <a href="#b0130" class="usa-link" aria-describedby="b0130">[26]</a>, <a href="#b0135" class="usa-link" aria-describedby="b0135">[27]</a>, <a href="#b0140" class="usa-link" aria-describedby="b0140">[28]</a>. The amplicon terminology of exact sequence variants (ESV) are used in place of operational taxonomic unit as no 97% similarity clustering is used <a href="#b0145" class="usa-link" aria-describedby="b0145">[29]</a>, <a href="#b0150" class="usa-link" aria-describedby="b0150">[30]</a>, <a href="#b0155" class="usa-link" aria-describedby="b0155">[31]</a>, <a href="#b0160" class="usa-link" aria-describedby="b0160">[32]</a>. Sequences were aligned and dereplicated using Mothur <a href="#b0165" class="usa-link" aria-describedby="b0165">[33]</a> and an ESV count threshold of 18 for the astronaut faecal microbiome dataset and 30 for the Mars500 habitat dataset. Annotation at family, genera or species-level used BLASTn criteria of BLASTn criteria of &gt;99% identity and coverage to the NCBI curated bacterial and Archaea RefSeq and NCBI nr/nt databases (January 2020 versions). All annotation calls are putative despite &gt;99% similarity, as databases are subject to change as discoveries are made and phylogenies revised. Reporting of 100% similar sequence matches to unique species is more accurate than reporting such hits at higher taxon levels <a href="#b0060" class="usa-link" aria-describedby="b0060">[12]</a>; however, all annotation should be considered as putative and interpreted with care. Differentially abundant ESVs annotated from the NCBI nr/nt database were manually assessed for quality. Annotation improvement is possible as databases improve/expand using the sequences provided with each ESV in <a href="#s0110" class="usa-link">Supplementary file 1</a>. When the highest identity/coverage is shared amongst multiple different putative annotation, all annotation is retained and reported. Amplicons with low-counts (&lt;30 or &lt;18) are binned to high-count sequences in a second BLASTn, using a lower threshold of &gt;98% identity/coverage.</p></section><section id="s0025"><h3 class="pmc_sec_title">2.3. Statistical analysis</h3>
<p id="p0035">The normalisation strategy conducted by Turroni et al. <a href="#b0085" class="usa-link" aria-describedby="b0085">[17]</a> was distinct for each astronaut to allow observation of individual microbiome change in steady states <a href="#b0170" class="usa-link" aria-describedby="b0170">[34]</a> over time. While this described some unique elements of each astronaut, the strategy made observation of common OTUs challenging. Here, while the limited biological replication of 6n (astronauts) could not be altered, the variation introduced from single replicate stool sampling <a href="#b0175" class="usa-link" aria-describedby="b0175">[35]</a>, <a href="#b0180" class="usa-link" aria-describedby="b0180">[36]</a>, <a href="#b0185" class="usa-link" aria-describedby="b0185">[37]</a> could be reduced by pooling libraries from the first five samples and from the final five samples to improve representation of early (7–45 days) and late (420–520 days) microbiome states of the each astronaut. Samples selected to represent early and late periods of confinement were defined as the first and last set of five samples taken from all six astronauts and the libraries were pooled <em>in silico</em> per astronaut and period. These timepoints also represent the samples available for all six astronauts which best aligned with the Mars500 physiology studies measuring astronaut body mass <a href="#b0090" class="usa-link" aria-describedby="b0090">[18]</a>, strength <a href="#b0100" class="usa-link" aria-describedby="b0100">[20]</a>, faecal protein calprotectin <a href="#b0105" class="usa-link" aria-describedby="b0105">[21]</a> and immune responses <a href="#b0115" class="usa-link" aria-describedby="b0115">[23]</a>. Differential abundance analysis was performed using DESeq2 <a href="#b0190" class="usa-link" aria-describedby="b0190">[38]</a>, <a href="#b0195" class="usa-link" aria-describedby="b0195">[39]</a>, which performs well with 16S rRNA gene amplicon data and uneven library sizes <a href="#b0200" class="usa-link" aria-describedby="b0200">[40]</a>. A false discovery rate (FDR; Benjamini-Hochberg procedure) &lt; 0.1 was applied <a href="#b0205" class="usa-link" aria-describedby="b0205">[41]</a>. Sparsity and count thresholds were applied whereby an ESV count in a single sample is &lt;90% of the count in all samples and ESV counts must be &gt;0 in at least 3 samples from the same group (early or late) <a href="#b0060" class="usa-link" aria-describedby="b0060">[12]</a>. Constrained ordination analysis was performed using canonical analysis of principal coordinates (CAP; <a href="#f0010" class="usa-link">Fig. 2</a>) with the 1-dimensional constraint (early vs late) represented on the x-axis and the first axis of non-constrained ordination (multidimensional scaling) on the y-axis. This analysis was performed with the Vegan library <a href="#b0210" class="usa-link" aria-describedby="b0210">[42]</a> on R via the capscale function.</p>
<figure class="fig xbox font-sm" id="f0010"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8099722_gr2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b43c/8099722/c20fd5155741/gr2.jpg" loading="lazy" height="785" width="738" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/f0010/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Early and late microbiome of Mars500 astronauts. A) Comparison of alpha diversity indices (Shannon and Inverse Simpson), B) Canonical analysis of principal coordinates (CAP; MDS = multidimensional scaling) of normalised (rlog) ESV abundance of astronaut samples, and C) Differentially abundant ESVs between the early and late samples (n = 6 astronauts). Fold change (FC log<sub>2</sub>) in relative abundance of significantly different (DESeq2 <a href="#b0190" class="usa-link" aria-describedby="b0190">[38]</a>) ESVs between early and late samples. +/− INF (demarcated by the dashed red line) indicates ‘infinite’ fold change, where an ESV had detectable counts in samples from only one condition. The complete ESV table including relative abundance, annotation, count distribution, blast statistics, alternative database hits, and sequences is provided in <a href="#s0110" class="usa-link">Supplementary file 1</a>. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.)</p></figcaption></figure></section><section id="s0030"><h3 class="pmc_sec_title">2.4. Data and code availability</h3>
<p id="p0040">ANCHOR code is available at <a href="https://github.com/gonzalezem/ANCHOR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/gonzalezem/ANCHOR</a>. ESV tables including relative abundance, annotation, count distribution, blast statistics, alternative database hits, and sequences are provided in <a href="#s0110" class="usa-link">Supplementary file 1</a>.</p></section></section><section id="s0035"><h2 class="pmc_sec_title">3. Results</h2>
<section id="s0040"><h3 class="pmc_sec_title">3.1. Mars500 astronaut gastrointestinal track microbiome community overview</h3>
<p id="p0045">A total of 408 exact sequence variants (ESVs) were assembled and captured 71.78% of the total 4,837,317 amplicons sequenced, including 213 ESVs present in all six astronauts in the early (7–45 days) and late (420–520 days) samples alone (<a href="#s0110" class="usa-link">Supplementary file 1</a>). Amplicon lengths ranged between 439 and 466 nt. All ESVs were annotated as &gt;99% similar sequence identity: 162 ESVs as putative species (including 14 which were ambiguous sequences to multiple species), 47 ESVs at genera-level, 34 at family-level, while 165 were poorly classified or unknown (&lt;99% similarity to any well-characterised taxa) (<a href="#f0005" class="usa-link">Fig. 1</a>). ESVs which could be annotated at species-level had an average identity of 99.92% (135 ESVs had 100% sequence similarity) and captured 67% of counts, those annotated at genera-level averaged 99.83% identity and captured 7% of counts, whereas those annotated at family-level averaged 99.92% identity and captured 11% of counts. ESVs identified from the phylum Firmicutes were the most prevalent, making up 64% of counts, while 7% were from Bacteroidetes, 6% from Actinobacteria, 6% from Verrucomicrobia, 2% from Proteobacteria and 15% were from unknown organisms.</p>
<figure class="fig xbox font-sm" id="f0005"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8099722_gr1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b43c/8099722/0fe4054bbb38/gr1.jpg" loading="lazy" height="890" width="778" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/f0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Experimental design and overall microbiome composition of the Mars500 astronauts. A) Sample timepoints with long-term confinement symptoms reported in Strollo et al. <a href="#b0090" class="usa-link" aria-describedby="b0090">[18]</a>, Gaffney et al. <a href="#b0100" class="usa-link" aria-describedby="b0100">[20]</a>, Roda et al. <a href="#b0105" class="usa-link" aria-describedby="b0105">[21]</a> and Yi et al. <a href="#b0115" class="usa-link" aria-describedby="b0115">[23]</a>. B) Flower diagram of all 408 ESVs coloured by phyla. The size of the distal node (i.e. lowest taxonomic level for an ESV) is proportional to the total raw abundance. C) the distribution of counts across taxonomy levels, and D) the number of ESVs annotated at each taxonomy level.</p></figcaption></figure><p id="p0050">The ESVs which could be confidently identified as putative species with the highest relative abundance across all astronauts in the phylum Firmicutes were all Clostridia, including Agathobacter_rectalis_1 with 10.1% of total counts, Gemmiger_formicilis_1 with 3.5%, Faecalibacterium_prausnitzii_3 with 3.3%, Fusicatenibacter_saccharivorans_1 with 2.5% and Blautia_wexlerae_1 with 2.1% (<a href="#s0110" class="usa-link">Supplementary file 1</a>). In Verrucomicrobia, two distinct ESVs had high relative abundance, Akkermansia_muciniphilia_1 with 3.5% of total counts and Akkermansia_muciniphilia_2 with 2.3% (both having 100% similarity to distinct strain groups, ie. JCM30893 and BSH01), whereas Bacteroides_dorei_1 had the highest relative abundance in the phylum Bacteroidetes with 1.2% of total counts. The ESVs with the highest relative abundance from the phyla Proteobacteria and Actinobacteria were both ambiguous, with the amplified sequence identical between multiple species (ESVs labelled as _MS), Enterobacterales_MS_1 (sequence common to <em>Brenneria alni</em>, <em>Escherichia coli</em>, <em>Escherichia fergusonii</em>, <em>Shigella flexneri</em> or <em>Shigella sonnei</em>) and Bifidobacterium_MS_2 (sequence common to <em>Bifidobacterium adolescentis</em>, <em>Bifidobacterium faecale</em> or <em>Bifidobacterium stercoris</em>).</p></section><section id="s0045"><h3 class="pmc_sec_title">3.2. Significant microbiome changes identified across astronauts over time</h3>
<p id="p0055">To determine whether there were common microbiome changes over time, a simplified experimental design was used to compare the first five samples (7–45 days) and the last five samples (420–520 days) taken from all six astronauts (<a href="#f0010" class="usa-link">Fig. 2</a>). When comparing early with late GIT microbiome samples taken from all six astronauts, alpha-diversity (Shannon and inverse Simpson) indices were not significantly different in estimated diversity between groups (t-tests p &gt; 0.05). Canonical analysis of principal coordinates suggests samples separated by early and late groups (<a href="#f0010" class="usa-link">Fig. 2</a>) and multivariate analysis identified significant variance between the groups (PERMANOVA, p &lt; 0.001). To identify which ESVs were significantly altered in relative abundance over time, differential abundance analysis was performed. Thirty-one ESVs were significantly different between timepoints, including 16 significantly enriched and 15 significantly depleted in late samples compared to early samples.</p>
<section id="s0050"><h4 class="pmc_sec_title">3.2.1. ESVs higher in relative abundance in late astronaut samples</h4>
<p id="p0060">The 16 ESVs in significantly higher relative abundance in the astronauts after 420–520 days of confinement were all poorly characterised with the exception of <em>Streptococcus_thermophilus_1</em> (<em>Streptococcus salivarius</em> subsp. <em>thermophilus</em>). Five ESVs have been putatively placed at genera or family levels (<a href="#f0010" class="usa-link">Fig. 2</a>): Kineothrix_1, Lachnospiraceae_10, Ruminococcus_3, Ruminococcaceae_1 and Christensenellaceae_1. The remaining ten differentially abundant ESV are classified here as unknown (&lt;99% similarity to previously identified taxa), although all have previously been observed as unknown bacteria at &gt;99% sequence similarity.</p>
<p id="p0065">Kineothrix_1 and Lachnospiraceae_10 were both most similar to the <em>Kineothrix alysoides</em> strain KNHs209, with 97.50% and 98.41% similar sequence identity, respectively. The ESV Kineothrix_1 did share 100% similarity to a genome assembled from work sequencing difficult to culture (yet-to-be-cultured) microbiome species from the human GIT (uncultured Clostridium sp. isolate 2789STDY5608883) <a href="#b0215" class="usa-link" aria-describedby="b0215">[43]</a>. Ruminococcus_3 is commonly observed at 100% sequence similarity (as an unknown bacteria) in human GIT microbiome studies (for example NCBI:<a href="https://www.ncbi.nlm.nih.gov/nuccore/AM275462" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AM275462</a> <a href="#b0220" class="usa-link" aria-describedby="b0220">[44]</a>) and has previously been classified as <em>Ruminococcus</em> sp. strain 95 (unpublished, but most similar to <em>Ruminococcus bromii</em> YE282 genome <a href="#b0225" class="usa-link" aria-describedby="b0225">[45]</a>) although the most similar well-characterised species was the type strain <em>Ruminococcus bromii</em> ATCC 27255 (97.97% similar). Ruminococcaceae_1 and Christensenellaceae_1, although also commonly observed as unknown bacteria in human faecal samples at 100% sequence similarity, were both highly dissimilar to any known bacterial species (&lt;94% 16S rRNA sequence similarity).</p>
<p id="p0070">The ten unknown sequences identified as significantly differentially abundant and enriched in astronauts after long-term confinement included the three ESVs, Unknown_254, Unknown_268 and Unknown_354, sharing 96–97% similarity to the known species, <em>Anaerocolumna cellulosilytica</em>, <em>Monoglobus pectinilyticus</em> and <em>Intestinimonas butyriciproducens</em>, respectively. The seven remaining unknown ESVs had &lt;90% sequence similarity to any known species but have all been previously observed as unknown bacteria (at &gt;99% sequence similarity) in GIT samples of humans as well as other animals <a href="#b0230" class="usa-link" aria-describedby="b0230">[46]</a>, <a href="#b0235" class="usa-link" aria-describedby="b0235">[47]</a>, <a href="#b0240" class="usa-link" aria-describedby="b0240">[48]</a>, <a href="#b0245" class="usa-link" aria-describedby="b0245">[49]</a>, <a href="#b0250" class="usa-link" aria-describedby="b0250">[50]</a>, <a href="#b0255" class="usa-link" aria-describedby="b0255">[51]</a>, <a href="#b0260" class="usa-link" aria-describedby="b0260">[52]</a>. Two of these ESVs, Unknown_309 (the highest in relative abundance) and Unknown_49, shared 100% sequence similarity with genomes recently assembled by Almeida et al. <a href="#b0265" class="usa-link" aria-describedby="b0265">[53]</a> in breakthrough research targeting yet-to-be-uncultured species in humans with <em>de novo</em> whole genome shotgun techniques (uncultured <em>Clostridium</em> sp. isolates UMGS1238 and UMGS1543) (<a href="#f0015" class="usa-link">Fig. 3</a>). The Unknown_309 sequence has also recently been observed (100% sequence similarity) and classified by Hynönen et al. <a href="#b0270" class="usa-link" aria-describedby="b0270">[54]</a> as the “Ct85 cluster” of uncultured GIT bacteria in mammals (AP10s.319, <a href="https://www.ncbi.nlm.nih.gov/nuccore/LR595742.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">LR595742.1</a> Ct85 type-f), alongside other ESVs present in astronauts but which did not significantly vary over time: Unknown_40 (AP07s.345 Ct85 type-d), Unknown_104 (AP07s.302 Ct85 untyped) and Unknown_38 (AP07s.190 Ct85 type-b).</p>
<figure class="fig xbox font-sm" id="f0015"><h5 class="obj_head">Fig. 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8099722_gr3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b43c/8099722/7a93c40c1cc5/gr3.jpg" loading="lazy" height="978" width="761" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/f0015/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phylogenetic tree of differentially abundant ESVs and <em>Faecalibacterium prausnitzii</em> ESV characterisation by alignment. A) Phylogenetic tree of differentially abundant ESVs and their most similar well-characterised species (if &gt;90% similarity) from the curated 16S rRNA gene NCBI RefSeq database. Genomes from uncultured candidate bacterial species assembled within Almeida et al. <a href="#b0265" class="usa-link" aria-describedby="b0265">[53]</a> were compared to unknown ESVs. Jukes–Cantor model with neighbour-joining with 10,000 replications and branch support is shown. The scale bar represents 1 substitution in 100 bp. B) Distinct <em>Faecalibacterium prausnitzii</em> ESVs were aligned against sequences from representative strains with genome assemblies. Phylogroup assignment derives from Benevides et al. <a href="#b0275" class="usa-link" aria-describedby="b0275">[55]</a> and Fitzgerald et al. <a href="#b0285" class="usa-link" aria-describedby="b0285">[57]</a>.</p></figcaption></figure></section><section id="s0055"><h4 class="pmc_sec_title">3.2.2. ESVs lower in relative abundance in late astronaut samples</h4>
<p id="p0075">After 420–520 days in isolation, 15 ESVs were significantly lower in relative abundance in astronauts. These were better characterised than those enriched over time, including ten ESVs which could be annotated at species level (<a href="#f0010" class="usa-link">Fig. 2</a>). These putative species included four distinct <em>Faecalibacterium prausnitzii</em> ESVs (labelled here as Faecalibacterium_prausnitzii_1, 3, 4 and 5), Ruminococcus_bromii_2, Blautia_luti_2, Anaerostipes_hadrus_1, Roseburia_faecis_1 and Lactobacillus_rogosae_1, as well as the ambiguous ESV Clostridium_MS_1, which shared 100% sequence similarity to both <em>Clostridium disporicum</em> and <em>Clostridium saudiense</em>. Additionally, two EVSs were poorly characterised at genera level, Roseburia_1 and Lachnospira_1, and three were unknown sequences, Unknown_181, Unknown_220 and Unknown_83.</p>
<p id="p0080">Six <em>F. prausnitzii</em> ESVs in total were identified in astronauts, with 4/6 significantly changing in relative abundance, reducing over time in all four cases. All of the ESVs corresponded to known strain groups <a href="#b0275" class="usa-link" aria-describedby="b0275">[55]</a>, <a href="#b0280" class="usa-link" aria-describedby="b0280">[56]</a>, <a href="#b0285" class="usa-link" aria-describedby="b0285">[57]</a> with 100% similarity (<a href="#f0015" class="usa-link">Fig. 3</a>); depleted ESVs were Faecalibacterium_prausnitzii_1 (identical to the type strain <em>F. prausnitzii</em> ATCC27768 in phylogroup I/subgroup-A), Faecalibacterium_prausnitzii_3 (identical to <em>F. prausnitzii</em> CNCM 4546; 4573; 4644; M21/2 in phylogroup I/subgroup-A), Faecalibacterium_prausnitzii_4 (identical to <em>F. prausnitzii</em> CNCM 4544; 4542 in phylogroup IIa/subgroup-C) and Faecalibacterium_prausnitzii_5 (identical to <em>F. prausnitzii</em> APC942/32 in phylogroup IIb/outgroup). The two <em>F. prausnitzii</em> ESVs which didn’t significantly vary between confinement time were Faecalibacterium_prausnitzii_2 (identical to <em>F. prausnitzii</em> CNCM 4543; 4574; A2-165 PacBio phylogroup IIb/subgroup-B) and Faecalibacterium_prausnitzii_6 (identical to <em>F. prausnitzii</em> CNCM 4575 phylogroup IIb/outgroup).</p></section></section><section id="s0060"><h3 class="pmc_sec_title">3.3. A direct comparison between the Mars500 astronaut and habitat microbiome communities</h3>
<p id="p0085">To assess the degree to which the astronauts’ microbiomes was associated with their habitat, the surface microbiome of the Mars500 modules, reported by Schwendner et al. <a href="#b0125" class="usa-link" aria-describedby="b0125">[25]</a>, was directly compared after reanalysis. A total of 1494 ESVs were assembled from 135 habitat samples, capturing 88.02% of the total 23,417,169 amplicons sequenced (<a href="#f0020" class="usa-link">Fig. 4</a>). Amplicon lengths (&gt;0.1% of counts) ranging between 403 and 426 nt. Seven hundred and ninety ESVs could be annotated as putative species (including 341 which were ambiguous sequences to multiple species), 167 ESVs at genera-level, 21 at family-level, while 305 were poorly classified or unknown (&lt;99% similarity to any well-characterised taxa). ESVs which could be annotated at species-level had an average identity of 99.95% (790 ESVs had 100% sequence similarity) and captured 88.39% of counts, those annotated at genera-level averaged 99.85% identity and captured 3.20% of counts, whereas those annotated at family-level averaged 99.88% identity and captured 0.40% of counts.</p>
<figure class="fig xbox font-sm" id="f0020"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8099722_gr4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b43c/8099722/be4543fac798/gr4.jpg" loading="lazy" height="874" width="765" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/f0020/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Overall microbiome composition of the Mars500 habitat. A) Mars500 habitat with the habitation, utility and medical modules highlighted (for further details see <a href="#b0125" class="usa-link" aria-describedby="b0125">[25]</a>). B) Flower diagram of all 1494 Mars500 habitat environment ESVs coloured by phyla (and Plantae). The size of the distal node (i.e. lowest taxonomic level for an ESV) is proportional to the total raw abundance. C) The distribution of counts across taxonomy levels, and D) the number of ESVs annotated at each taxonomy level.</p></figcaption></figure><p id="p0090">The V3-V4 16S rRNA gene primer pair used to assess the astronaut GIT microbiome, 341F and 805R, was distinct to the V4 16S rRNA gene primer pair used to assess the Mars500 habitat microbiome, 515F and 926R <a href="#b0085" class="usa-link" aria-describedby="b0085">[17]</a>, <a href="#b0125" class="usa-link" aria-describedby="b0125">[25]</a>. To directly compare the astronaut and habitat microbiome datasets, ESVs annotated at species level from each experiment were aligned directly. Alignment of each library revealed 79/162 astronaut ESVs had an identical overlapping V4 hypervariable sequence region (252–253 nts) with one or more of 98/790 habitat ESVs (12.4% of total species), 42 of which were single astronaut-habitat ESV pairs unique to a single species. When gene sequences from genome assemblies were used as scaffolds for astronaut-habitat (gut-environment) ESV pairs, a total of 49 Scaffolded Amplicons (ScAmps) aligned as 100% identical to reference genomes (<a href="#f0025" class="usa-link">Fig. 5</a>). Astronaut-habitat ScAmps allowed for improved resolution of distinct 16S rRNA gene sequences and taxonomic annotation. Examples where two distinct taxa could be distinguished using ScAmps but could not from a single primer pair included distinct <em>Anaerostipes hadrus, Bacteroides uniformis</em>, <em>Bacteroides caccae</em> and <em>Eggerthella lenta</em> strain groups. The increased length and hypervariable region coverage of ScAmps, between 557 and 590 nt, resolved ESV annotation ambiguity in 8 cases, identifying two distinct <em>Veillonella parvula</em> strain groups (NCTC11810 and UTDB1-3) in the astronaut microbiome, <em>Bifidobacterium longum</em> subsp. <em>longum</em>, <em>Bacteroides thetaiotaomicron, Streptococcus thermophilus</em> and <em>Lactococcus lactis</em> in the habitat microbiome, and <em>Bifidobacterium pseudocatenulatum</em> and <em>Bifidobacterium adolescentis</em> in both.</p>
<figure class="fig xbox font-sm" id="f0025"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8099722_gr5a.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b43c/8099722/de8bbd3f91e4/gr5a.jpg" loading="lazy" height="407" width="778" alt="Fig. 5"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8099722_gr5b.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b43c/8099722/ee0ee2f485b7/gr5b.jpg" loading="lazy" height="990" width="765" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/f0025/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Astronaut gut-habitat environment connection using sequences from reference genomes. A) Illustration of scaffolded amplicons (ScAmps) construction from paired ESVs and sequences from reference genomes, B) ScAmps shared between Mars500 astronauts and their habitat; gut and environment ESVs aligned as 100% identical to reference genomes.</p></figcaption></figure></section></section><section id="s0065"><h2 class="pmc_sec_title">4. Discussion</h2>
<section id="s0070"><h3 class="pmc_sec_title">4.1. Mars500 astronaut gastrointestinal track microbiome community overview</h3>
<p id="p0095">Recent metagenomic studies indicate that the majority of gastrointestinal track (GIT) microbiome members are still unknown or uncharacterised species <a href="#b0265" class="usa-link" aria-describedby="b0265">[53]</a>. It is therefore not surprising that over half of the ESVs identified across the astronauts were uncharacterised (<a href="#f0005" class="usa-link">Fig. 1</a>). This complexity is further complicated as substantial proportions of the human microbiome are often highly unique to individuals. The highly distinct microbiome compositions of each individual astronaut were well described by the initial Mars500 GIT microbiome research conducted by Turroni et al. <a href="#b0085" class="usa-link" aria-describedby="b0085">[17]</a> and make general patterns of microbiome change challenging to observe. The original analysis did, however, report similar phyla present across astronauts, including: Firmicutes, Bacteroidetes, Actinobacteria, Verrucomicrobia (although present in all astronauts, bacteria from within this group were originally not detected in two astronauts) and Proteobacteria. Instead of using a distinct normalisation strategy for each astronaut to observe change in steady states <a href="#b0085" class="usa-link" aria-describedby="b0085">[17]</a>, <a href="#b0170" class="usa-link" aria-describedby="b0170">[34]</a>, here, libraries were pooled from the first five samples and the last five samples to improve representation of early (7–45 days) and late (420–520 days) microbiome states (reducing technical noise common to single replicate stool sampling <a href="#b0175" class="usa-link" aria-describedby="b0175">[35]</a>, <a href="#b0180" class="usa-link" aria-describedby="b0180">[36]</a>, <a href="#b0185" class="usa-link" aria-describedby="b0185">[37]</a>) before all astronauts were compared directly to make statistical analysis of the common change over time more straightforward.</p></section><section id="s0075"><h3 class="pmc_sec_title">4.2. Enriched taxa in astronauts after long-term confinement</h3>
<p id="p0100">The presence of <em>Streptococcus thermophilus</em> (<em>Streptococcus salivarius subspecies thermophilus</em>) (<a href="#f0010" class="usa-link">Fig. 2</a>) within the Mars500 habitat is not unexpected as the species is commonly used in the dairy industry and is added to yogurts as live culture to market as bestowing health benefits upon consumption (as a probiotic). Subsequent assessment of astronaut dietary reports during the experiment revealed that the Italian company Granarolo provided lyophilised ‘Yomo’ yogurts to the Mars500 project, which do indeed contain live <em>Streptococcus thermophilus</em> when sold commercially <a href="#b0290" class="usa-link" aria-describedby="b0290">[58]</a>. Interestingly, <em>S. thermophilus</em> is known to be particularly resilient to the freeze-drying process <a href="#b0295" class="usa-link" aria-describedby="b0295">[59]</a>, so would likely have been present for consumption as live culture in the habitat. The observed significant increase in relative abundance between early and late confinement times is also coherent as the Italian astronaut reported that the team only ate the lyophilised yogurt in the final 8 months of the experiment (during the simulated “return journey”, reported in a July 2014 interview with astronaut Diego Urbina for the Italian Space Agency’s Outpost 42: Space food in training for Mars). Research suggests that culturing can sometimes capture as low as &lt;1% of a microbial community <a href="#b0300" class="usa-link" aria-describedby="b0300">[60]</a>; however, culture-dependent approaches could have been attempted to confirm viability here. Although bacterial viability was not established, these findings do suggest the introduction of these species to the astronauts has been successfully tracked through to their respective microbiomes.</p>
<p id="p0105">Three ESVs enriched in astronauts after isolation were identified as unknown but shared 97% sequence similarity to well-characterised bacteria <em>A. cellulosilytica</em>, <em>M. pectinilyticus</em> and <em>I. butyriciproducens</em>. <em>A. cellulosilytica</em> and <em>M. pectinilyticus</em> are plant cell wall degrading bacteria with carbohydrate active enzymes (CAZy) suites allowing degradation of cellulose and hemicellulose <a href="#b0305" class="usa-link" aria-describedby="b0305">[61]</a>, and pectin <a href="#b0310" class="usa-link" aria-describedby="b0310">[62]</a>, respectively, alongside corresponding fermentation of xylose, arabinose and galacturonic acid. <em>I. butyriciproducens</em> is a butyrate producing species <a href="#b0315" class="usa-link" aria-describedby="b0315">[63]</a> previously identified as increasing in relative abundance alongside a decrease in other common butyrate producers (such a <em>Faecalibacterium prausnitzii</em>) in patients with chronic pain <a href="#b0130" class="usa-link" aria-describedby="b0130">[26]</a>. While changes in bacteria similar to <em>M. pectinilyticus</em> and <em>A. cellulosilytica</em> could be associated with dietary modifications involving fruit or vegetables, alterations to the butyrate producing community could have important implications for GIT homeostasis when considered in the context of increased muscle loss and intestinal inflammation markers over astronaut confinement. Seven ESVs were dissimilar to any well-characterised species. In complex soil or rhizosphere (plant-soil interface) samples where the majority of microbial species are often yet-to-be-characterised <a href="#b0320" class="usa-link" aria-describedby="b0320">[64]</a>, <a href="#b0325" class="usa-link" aria-describedby="b0325">[65]</a>, difficult to culture groups of bacteria such as the TM7 (group 1) bacteria have previously been observed as differentially abundant <a href="#b0140" class="usa-link" aria-describedby="b0140">[28]</a>. In humans, uncharacterised species often share the most sequence similarity to the TM7 (group 3) bacteria, which are typified by recalcitrance to standard culturing techniques <a href="#b0330" class="usa-link" aria-describedby="b0330">[66]</a> currently thought to stem from their epibiont lifestyle (living on the surface of other microbes) <a href="#b0335" class="usa-link" aria-describedby="b0335">[67]</a>. However, the ubiquitous presence of unknown or poorly characterised microbes in human GIT samples is often not reported, hindering their study. The scale of uncharacterised human microbiome members was recently well-captured by Almeida et al. <a href="#b0265" class="usa-link" aria-describedby="b0265">[53]</a> using <em>de novo</em> whole genome shotgun metagenomics to assemble 1952 uncultured candidate bacterial species genomes from 11,850 human GIT microbiome samples, some of which shared 100% sequence identity with ESVs observed here (<a href="#f0015" class="usa-link">Fig. 3</a>). While reporting unknown or poorly characterised organisms can be challenging, these ESVs are significantly associated with long-term isolation in astronauts. Although little is known as to the role that these microbes might play in health, they do also seem to be consistently present in humans and are here implicated as dynamic members of the human microbiome meriting further study as potentially having novel biological importance.</p></section><section id="s0080"><h3 class="pmc_sec_title">4.3. Depleted taxa in astronauts after long-term confinement</h3>
<p id="p0110">Although the microbiome species enriched after long-term confinement were largely uncharacterised organisms, the majority of the significantly depleted species were well known. <em>F. prausnitzii</em> is one of the best characterised inhabitants of a healthy human GIT and is a butyrate producer with well-documented anti-inflammatory properties whose reduction in relative abundance is consistently associated with intestinal inflammation <a href="#b0340" class="usa-link" aria-describedby="b0340">[68]</a>, <a href="#b0345" class="usa-link" aria-describedby="b0345">[69]</a> and an increase in calprotectin <a href="#b0350" class="usa-link" aria-describedby="b0350">[70]</a>, <a href="#b0355" class="usa-link" aria-describedby="b0355">[71]</a>. <em>F. prausnitzii</em> also almost always negatively associates with type 2 diabetes (4/5 controlled studies) <a href="#b0360" class="usa-link" aria-describedby="b0360">[72]</a>, implicating a putative role in insulin resistance. The significant reduction in <em>F. prausnitzii</em> in astronauts after long-term confinement (<a href="#f0010" class="usa-link">Figs. 2</a> and <a href="#f0015" class="usa-link">3</a>) is therefore consistent with the intestinal inflammation, increased faecal calprotectin levels, moderate insulin resistance and lean body mass loss reported by Roda et al. <a href="#b0105" class="usa-link" aria-describedby="b0105">[21]</a>, Yi et al. <a href="#b0115" class="usa-link" aria-describedby="b0115">[23]</a>, Strollo et al. <a href="#b0090" class="usa-link" aria-describedby="b0090">[18]</a> and Gaffney et al. <a href="#b0100" class="usa-link" aria-describedby="b0100">[20]</a> in Mars500 astronauts over the 520 days of confinement. Interestingly, <em>F. prausnitzii</em> phylogroup II is considered to be the less sensitive marker of intestinal inflammation when depleted <a href="#b0285" class="usa-link" aria-describedby="b0285">[57]</a>, <a href="#b0365" class="usa-link" aria-describedby="b0365">[73]</a>. This aligns to the pattern of depletion in only four putative strain-types observed here, as the two out of six <em>F. prausnitzii</em> ESVs which were not significantly depleted over time shared 100% similarity to members of phylogroup IIb <a href="#b0275" class="usa-link" aria-describedby="b0275">[55]</a>, <a href="#b0280" class="usa-link" aria-describedby="b0280">[56]</a>, <a href="#b0285" class="usa-link" aria-describedby="b0285">[57]</a>. Turroni et al. <a href="#b0085" class="usa-link" aria-describedby="b0085">[17]</a> reported a decreasing trend in OTUs associated to <em>F. prausnitzii</em> over time when first assessing the Mars500 astronaut microbiomes, but were limited in capability to statistically compare OTUs across astronauts owing to tailoring normalisation to each astronaut as well as because low taxonomic resolution would conflate the distinct patterns of relative abundance in these <em>F. prausnitzii</em> ESVs.</p>
<p id="p0115">The five other putative species identified as significantly depleted in relative abundance after long-term confinement, <em>Ruminococcus bromii</em>, <em>Blautia luti</em>, <em>Anaerostipes hadrus</em>, <em>Roseburia faecis</em>, and <em>Lactobacillus rogosae</em> (<a href="#f0010" class="usa-link">Fig. 2</a>), were each consistent with this association with intestinal inflammation and disruption of glucose metabolism. The <em>Ruminococcus bromii</em> ESV was only 99.10% similar to the type strain, <em>Ruminococcus bromii</em> strain ATCC 27255, but was 100% similar to the strain L2-36 (NCBI:GCA_002834165.1) (<a href="#f0015" class="usa-link">Fig. 3</a>) which is considered a ‘keystone’ degrader of resistant starch in the human GIT <a href="#b0370" class="usa-link" aria-describedby="b0370">[74]</a>, <a href="#b0375" class="usa-link" aria-describedby="b0375">[75]</a>. Robinson et al. <a href="#b0380" class="usa-link" aria-describedby="b0380">[76]</a>, <a href="#b0385" class="usa-link" aria-describedby="b0385">[77]</a> first reported a direct relationship between dietary resistant starch and insulin sensitivity improvement but could not explain the association through investigation of host metabolism alone. Although a study in germ-free mice has suggested the impact of resistant starch on insulin resistance could be independent of intestinal bacteria <a href="#b0390" class="usa-link" aria-describedby="b0390">[78]</a>, members of the GIT microbiome have largely been considered to underlie the relationship <a href="#b0395" class="usa-link" aria-describedby="b0395">[79]</a>, <a href="#b0400" class="usa-link" aria-describedby="b0400">[80]</a>, <a href="#b0405" class="usa-link" aria-describedby="b0405">[81]</a>. <em>R. bromii</em> has been more directly associated to insulin resistance through increased relative abundance, alongside <em>F. prausnitzii</em>, in patients with metabolic syndrome after faecal matter transplant from healthy donors which improved insulin sensitivity <a href="#b0410" class="usa-link" aria-describedby="b0410">[82]</a>. Similarly, Benitez-paez et al. <a href="#b0415" class="usa-link" aria-describedby="b0415">[83]</a> recently reported decreases in the relative abundance of <em>B. luti</em> (reclassified from <em>Ruminococcus luti</em>) associated with insulin resistance of obese children, alongside significant increases in proinflammatory cytokines similar to those observed by Yi et al. <a href="#b0115" class="usa-link" aria-describedby="b0115">[23]</a> in Mars500 astronauts after 520 days isolation. Benitez-paez et al. <a href="#b0415" class="usa-link" aria-describedby="b0415">[83]</a> also demonstrated <em>B. luti</em> had anti-inflammatory activity using <em>in vitro</em> cell cultures and speculated that a reduction of <em>B. luti</em> within the GIT may contribute to insulin resistance through increase intestinal inflammation. Both <em>A. hadrus</em> and <em>R. faecis</em> are butyrate producers with the former associated to insulin metabolism, significantly increasing in relative abundance (3-fold) in patients responding to FMT treatment with improved insulin sensitivity and decreased proinflammatory markers <a href="#b0420" class="usa-link" aria-describedby="b0420">[84]</a>, and the latter whose reduction has been associated to increased inflammation alongside <em>F. prausnitzii</em>
<a href="#b0425" class="usa-link" aria-describedby="b0425">[85]</a>. The <em>L. rogosae</em> ESV shares 100% sequence identity to the type-strain <em>L. rogosae</em> strain ATCC 27753. Although less well-studied and potentially misplaced taxonomically <a href="#b0430" class="usa-link" aria-describedby="b0430">[86]</a>, <em>L. rogosae</em> was found to be significantly reduced in relative abundance in patients with systemic inflammatory response syndrome (alongside <em>R. faecis</em> and <em>Faecalibacteria</em> sp.) <a href="#b0435" class="usa-link" aria-describedby="b0435">[87]</a> and in MRSA positive patients with increased serum inflammatory markers (alongside <em>F. prausnitzii</em>) <a href="#b0440" class="usa-link" aria-describedby="b0440">[88]</a> when compared to healthy controls in both cases.</p>
<p id="p0120">Despite the highly unique nature of each astronaut’s microbiome, these findings indicate common changes in specific microbiome species over the 520 days of isolation which are consistent with the reported common significant increases in proinflammatory markers and losses in strength and lean body mass. While the potential involvement of these observed microbiome species is discussed (alongside appropriate evidence), no causal relationship between their relative abundance and astronaut health is established in this work.</p></section><section id="s0085"><h3 class="pmc_sec_title">4.4. Commonalities between the Mars500 astronaut and habitat microbiome communities</h3>
<p id="p0125">Enrichment of in-built environments with human-associated bacteria is thought to occur rapidly <a href="#b0445" class="usa-link" aria-describedby="b0445">[89]</a> and surface microbiome research conducted in hospital environments (intensive care units and operating theatres), clean rooms and the international space station (ISS) revealed that the selective pressure of confinement and cleaning procedures can promote acquisition of broad resistance mechanisms in a microbial community adapted to human habitation <a href="#b0055" class="usa-link" aria-describedby="b0055">[11]</a>, <a href="#b0450" class="usa-link" aria-describedby="b0450">[90]</a>. Other studies into the environmental microbiome of MIR <a href="#b0455" class="usa-link" aria-describedby="b0455">[91]</a> and the ISS <a href="#b0070" class="usa-link" aria-describedby="b0070">[14]</a>, <a href="#b0460" class="usa-link" aria-describedby="b0460">[92]</a>, <a href="#b0465" class="usa-link" aria-describedby="b0465">[93]</a>, <a href="#b0470" class="usa-link" aria-describedby="b0470">[94]</a> have also revealed enrichment of human-associated microbes, in particular within compartments with the highest human inhabitancy (such as the crew quarters <a href="#b0060" class="usa-link" aria-describedby="b0060">[12]</a>). This phenomenon was also reported in the Mars500 confinement experiment, where the environmental microbiome was explored in samples taken from the habitation, utility and medical modules throughout the course of the 520-day experiment. Schwendner et al. <a href="#b0125" class="usa-link" aria-describedby="b0125">[25]</a> reported that the only bacteria detected in all three Mars500 modules were some of the most common human microbiome species: <em>Staphylococcus aureus</em>, <em>Staphylococcus epidermidis</em>, <em>Staphylococcus haemolyticus</em> and <em>Staphylococcus hominis</em>. The potential for opportunistic pathogen selection is of particular concern for long-duration space travel where astronaut health is at risk and medical facilities are limited.</p>
<p id="p0130">Comparisons of 16S rRNA gene OTUs (97% clustered) have previously been performed comparing faecal matter and local water samples to investigate environmental contamination in Kenya <a href="#b0475" class="usa-link" aria-describedby="b0475">[95]</a> using SourceTracker <a href="#b0480" class="usa-link" aria-describedby="b0480">[96]</a>. Improved 16S rRNA gene bioinformatics approaches (ANCHOR <a href="#b0060" class="usa-link" aria-describedby="b0060">[12]</a>, UPARSE <a href="#b0485" class="usa-link" aria-describedby="b0485">[97]</a>, DADA2 <a href="#b0490" class="usa-link" aria-describedby="b0490">[98]</a> or QIIME2 <a href="#b0495" class="usa-link" aria-describedby="b0495">[99]</a>) are able to discern amplicon sequences with more confidence in complex metagenomic samples (<a href="#f0020" class="usa-link">Fig. 4</a>), allowing for sequences to be compared directly between experiments (<a href="#f0025" class="usa-link">Fig. 5</a>). Different primer pairs can have variable 16s rRNA gene amplification rates for different bacteria, potentially confounding direct quantitative integration of amplicon counts across two primer pairs; however, the Mars500 experiment, specifically the high quality datasets shared by Turroni et al. <a href="#b0085" class="usa-link" aria-describedby="b0085">[17]</a> and Schwendner et al. <a href="#b0125" class="usa-link" aria-describedby="b0125">[25]</a>, provided a one-off opportunity for qualitative comparison of the microbiota commonality between a closed system environment and the gut of its entire inhabiting population over 520 days as both of these primer pairs have over 95% coverage of known bacterial species <a href="#b0500" class="usa-link" aria-describedby="b0500">[100]</a>. Direct sequence comparison indicated that 49% and 12% of species were shared between the Mars500 astronaut GIT and environmental habitat microbiomes, respectively, suggesting extensive interactions between the communities.</p></section></section><section id="s0090"><h2 class="pmc_sec_title">5. Conclusions</h2>
<p id="p0135">Significant changes in the relative abundance of astronaut microbiome species occurred over the 520 days of isolation. Some enrichment of known bacteria corresponded to changes in the crew’s diet, although significant enrichment of unknown species highlights the ongoing challenge faced in understanding how complex microbiome interactions influence human health. Depletion of major anti-inflammatory gastrointestinal tract bacteria over time is consistent with symptoms associated with the intestinal inflammation and insulin resistance measured during the Mars500 experiment and sometimes observed in astronauts during long-duration space flight. Sequence comparison of the astronauts’ faecal microbiome to the Mars500 environmental microbiome revealed direct large-scale associations between the gut and environmental microbiomes, an observation made possible due to the extreme experimental confinement. Identification of significant species-level microbiome alterations which align with astronaut symptoms connects the Russian, European, Chinese and now Canadian Space Agency Mars500 studies. This initial evidence of a potentially deleterious impact of long-term confinement upon microbiome health is an important step forward in the current drive to enable long-duration spaceflight.</p></section><section id="s0095"><h2 class="pmc_sec_title">Funding</h2>
<p id="p0140">The project was funded by the Canadian Space Agency, Health and Life Sciences Data and Samples Mining award as well as an NSERC Discovery Grant (FEP RGPIN-2017-05452).</p></section><section id="s0100"><h2 class="pmc_sec_title">Author contributions</h2>
<p id="p0145">NJBB, FEP and EG all analysed and interpreted the data and wrote the manuscript.</p></section><section id="sec21"><h2 class="pmc_sec_title">Declaration of Competing Interest</h2>
<p id="p0150">The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.</p></section><section id="ak005" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p id="p0155">We would like to thank the Canadian Space Agency for funding this research through a Health and Life Sciences Data and Samples Mining award. Funding for FEP was also provided by an NSERC Discovery Grant (FEP RGPIN-2017-05452). Thanks are extended to Silvia Turroni, Simone Rampelli, Elena Biagi, Clarissa Consolandi, Marco Severgnini, Clelia Peano, Sara Quercia, Matteo Soverini, Franck G. Carbonero, Giovanna Bianconi, Petra Rettberg, Francesco Canganella, Patrizia Brigidi and Marco Candela as well as Petra Schwendner, Alexander Mahnert, Kaisa Koskinen, Christine Moissl-Eichinger, Simon Barczyk, Reinhard Wirth, Gabriele Berg and Petra Rettberg for making the data from their research freely available and navigable for other researchers to explore. We hope to have constructively and respectfully built upon their strong research. Special thanks are also given to the incredible Mars500 astronauts: Diego Urbina, Alexander Smoleevskij, Yue Wang, Sukhrob Kamolov, Romain Charles and Alexey Sitev.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="s0105">
<sup>Appendix A</sup><p class="display-inline" id="p0160">Supplementary data to this article can be found online at <a href="https://doi.org/10.1016/j.csbj.2021.03.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.csbj.2021.03.040</a>.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>N.J.B. Brereton, Email: Nicholas.brereton@umonteal.ca.</p>
<p>F.E. Pitre, Email: Frederic.pitre@umotreal.ca.</p>
<p>E. Gonzalez, Email: Emmanuel.gonzalez@mcgill.ca.</p></section><section id="s0110"><h2 class="pmc_sec_title">Appendix A. Supplementary data</h2>
<p id="p0165">The following are the Supplementary data to this article:</p>
<section class="sm xbox font-sm" id="m0005"><div class="caption p"><span>Supplementary Data 1</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/8099722/bin/mmc1.xlsx" data-ga-action="click_feat_suppl" class="usa-link">mmc1.xlsx</a><sup> (1.6MB, xlsx) </sup>
</div></div></section></section><section id="bi005" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="bi005_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="b0005">
<span class="label">1.</span><cite>ISECG. The Global Exploration Roadmap. In: Group TISEC, editor.: National Aeronautics and Space Administration Headquarters, Washington, DC 20546-0001; 2018.</cite>
</li>
<li id="b0010">
<span class="label">2.</span><cite>Agency CS. Evaluation of Human Space Missions and International Space Station Utilization. In: Directorate AE, editor. 2019.</cite>
</li>
<li id="b0015">
<span class="label">3.</span><cite>Thirsk R, Hart D, Mihailidis A, E. Adams, Christopher A, Joanette Y, et al. Canadian Healthcare in Deep Space. In: Agency CS, editor.: Minister of Innovation, Science and Economic Development; 2019.</cite>
</li>
<li id="b0020">
<span class="label">4.</span><cite>Nadia H. Agha, Satish K. Mehta, Bridgette V. Rooney, Mitzi S. Laughlin, Melissa M. Markofski, Duane L. Pierson, et al. Stress Challenges and Immunity in Space. Choukèr A, editor: Springer International Publishing 2020.</cite>
</li>
<li id="b0025">
<span class="label">5.</span><cite>Perry R.J., Peng L., Barry N.A., Cline G.W., Zhang D., Cardone R.L. Acetate mediates a microbiome–brain–β-cell axis to promote metabolic syndrome. Nature. 2016;534(7606):213. doi: 10.1038/nature18309.</cite> [<a href="https://doi.org/10.1038/nature18309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4922538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27279214/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Acetate%20mediates%20a%20microbiome%E2%80%93brain%E2%80%93%CE%B2-cell%20axis%20to%20promote%20metabolic%20syndrome&amp;author=R.J.%20Perry&amp;author=L.%20Peng&amp;author=N.A.%20Barry&amp;author=G.W.%20Cline&amp;author=D.%20Zhang&amp;volume=534&amp;issue=7606&amp;publication_year=2016&amp;pages=213&amp;pmid=27279214&amp;doi=10.1038/nature18309&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0030">
<span class="label">6.</span><cite>Sarkar A., Harty S., Lehto S.M., Moeller A.H., Dinan T.G., Dunbar R.I. The microbiome in psychology and cognitive neuroscience. Trends Cognitive Sci. 2018;22(7):611–636. doi: 10.1016/j.tics.2018.04.006.</cite> [<a href="https://doi.org/10.1016/j.tics.2018.04.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29907531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Cognitive%20Sci&amp;title=The%20microbiome%20in%20psychology%20and%20cognitive%20neuroscience&amp;author=A.%20Sarkar&amp;author=S.%20Harty&amp;author=S.M.%20Lehto&amp;author=A.H.%20Moeller&amp;author=T.G.%20Dinan&amp;volume=22&amp;issue=7&amp;publication_year=2018&amp;pages=611-636&amp;pmid=29907531&amp;doi=10.1016/j.tics.2018.04.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0035">
<span class="label">7.</span><cite>Erny D., de Angelis A.L.H., Jaitin D., Wieghofer P., Staszewski O., David E. Host microbiota constantly control maturation and function of microglia in the CNS. Nat Neurosci. 2015;18(7):965. doi: 10.1038/nn.4030.</cite> [<a href="https://doi.org/10.1038/nn.4030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5528863/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26030851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Neurosci&amp;title=Host%20microbiota%20constantly%20control%20maturation%20and%20function%20of%20microglia%20in%20the%20CNS&amp;author=D.%20Erny&amp;author=A.L.H.%20de%20Angelis&amp;author=D.%20Jaitin&amp;author=P.%20Wieghofer&amp;author=O.%20Staszewski&amp;volume=18&amp;issue=7&amp;publication_year=2015&amp;pages=965&amp;pmid=26030851&amp;doi=10.1038/nn.4030&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0040">
<span class="label">8.</span><cite>Fung T.C., Olson C.A., Hsiao E.Y. Interactions between the microbiota, immune and nervous systems in health and disease. Nat Neurosci. 2017;20(2):145. doi: 10.1038/nn.4476.</cite> [<a href="https://doi.org/10.1038/nn.4476" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6960010/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28092661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Neurosci&amp;title=Interactions%20between%20the%20microbiota,%20immune%20and%20nervous%20systems%20in%20health%20and%20disease&amp;author=T.C.%20Fung&amp;author=C.A.%20Olson&amp;author=E.Y.%20Hsiao&amp;volume=20&amp;issue=2&amp;publication_year=2017&amp;pages=145&amp;pmid=28092661&amp;doi=10.1038/nn.4476&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0045">
<span class="label">9.</span><cite>Sampson T.R., Mazmanian S.K. Control of brain development, function, and behavior by the microbiome. Cell Host Microbe. 2015;17(5):565–576. doi: 10.1016/j.chom.2015.04.011.</cite> [<a href="https://doi.org/10.1016/j.chom.2015.04.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4442490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25974299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20Microbe&amp;title=Control%20of%20brain%20development,%20function,%20and%20behavior%20by%20the%20microbiome&amp;author=T.R.%20Sampson&amp;author=S.K.%20Mazmanian&amp;volume=17&amp;issue=5&amp;publication_year=2015&amp;pages=565-576&amp;pmid=25974299&amp;doi=10.1016/j.chom.2015.04.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0050">
<span class="label">10.</span><cite>Flint H.J., Scott K.P., Louis P., Duncan S.H. The role of the gut microbiota in nutrition and health. Nat Rev Gastroenterol Hepatol. 2012;9(10):577. doi: 10.1038/nrgastro.2012.156.</cite> [<a href="https://doi.org/10.1038/nrgastro.2012.156" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22945443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Gastroenterol%20Hepatol&amp;title=The%20role%20of%20the%20gut%20microbiota%20in%20nutrition%20and%20health&amp;author=H.J.%20Flint&amp;author=K.P.%20Scott&amp;author=P.%20Louis&amp;author=S.H.%20Duncan&amp;volume=9&amp;issue=10&amp;publication_year=2012&amp;pages=577&amp;pmid=22945443&amp;doi=10.1038/nrgastro.2012.156&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0055">
<span class="label">11.</span><cite>Mora M., Mahnert A., Koskinen K., Pausan M.R., Oberauner-Wappis L., Krause R. Microorganisms in confined habitats: microbial monitoring and control of intensive care units, operating rooms, cleanrooms and the International Space Station. Front Microbiol. 2016;7:1573. doi: 10.3389/fmicb.2016.01573.</cite> [<a href="https://doi.org/10.3389/fmicb.2016.01573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5061736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27790191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Microorganisms%20in%20confined%20habitats:%20microbial%20monitoring%20and%20control%20of%20intensive%20care%20units,%20operating%20rooms,%20cleanrooms%20and%20the%20International%20Space%20Station&amp;author=M.%20Mora&amp;author=A.%20Mahnert&amp;author=K.%20Koskinen&amp;author=M.R.%20Pausan&amp;author=L.%20Oberauner-Wappis&amp;volume=7&amp;publication_year=2016&amp;pages=1573&amp;pmid=27790191&amp;doi=10.3389/fmicb.2016.01573&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0060">
<span class="label">12.</span><cite>Gonzalez E., Pitre F., Brereton N. ANCHOR: A 16S rRNA gene amplicon pipeline for microbial analysis of multiple environmental samples. Environ Microbiol. 2019 doi: 10.1111/1462-2920.14632.</cite> [<a href="https://doi.org/10.1111/1462-2920.14632" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6851558/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30990927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environ%20Microbiol&amp;title=ANCHOR:%20A%2016S%20rRNA%20gene%20amplicon%20pipeline%20for%20microbial%20analysis%20of%20multiple%20environmental%20samples&amp;author=E.%20Gonzalez&amp;author=F.%20Pitre&amp;author=N.%20Brereton&amp;publication_year=2019&amp;pmid=30990927&amp;doi=10.1111/1462-2920.14632&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0065">
<span class="label">13.</span><cite>Singh N.K., Wood J.M., Karouia F., Venkateswaran K. Succession and persistence of microbial communities and antimicrobial resistance genes associated with International Space Station environmental surfaces. Microbiome. 2018;6(1):204. doi: 10.1186/s40168-018-0585-2.</cite> [<a href="https://doi.org/10.1186/s40168-018-0585-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6234677/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30424821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Succession%20and%20persistence%20of%20microbial%20communities%20and%20antimicrobial%20resistance%20genes%20associated%20with%20International%20Space%20Station%20environmental%20surfaces&amp;author=N.K.%20Singh&amp;author=J.M.%20Wood&amp;author=F.%20Karouia&amp;author=K.%20Venkateswaran&amp;volume=6&amp;issue=1&amp;publication_year=2018&amp;pages=204&amp;pmid=30424821&amp;doi=10.1186/s40168-018-0585-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0070">
<span class="label">14.</span><cite>Lang J.M., Coil D.A., Neches R.Y., Brown W.E., Cavalier D., Severance M. A microbial survey of the International Space Station (ISS) PeerJ. 2017;5 doi: 10.7717/peerj.4029.</cite> [<a href="https://doi.org/10.7717/peerj.4029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5827671/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29492330/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=A%20microbial%20survey%20of%20the%20International%20Space%20Station%20(ISS)&amp;author=J.M.%20Lang&amp;author=D.A.%20Coil&amp;author=R.Y.%20Neches&amp;author=W.E.%20Brown&amp;author=D.%20Cavalier&amp;volume=5&amp;publication_year=2017&amp;pmid=29492330&amp;doi=10.7717/peerj.4029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0075">
<span class="label">15.</span><cite>Chen J., Wang Q., Hao Z., Li Z., Sahu S.K., Liu H. Relationship between the Gut Microbiome and Energy/Nutrient intake in a confined bioregenerative life support system. Appl Environ Microbiol. 2020;86(4) doi: 10.1128/AEM.02465-19.</cite> [<a href="https://doi.org/10.1128/AEM.02465-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6997737/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31811045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Relationship%20between%20the%20Gut%20Microbiome%20and%20Energy/Nutrient%20intake%20in%20a%20confined%20bioregenerative%20life%20support%20system&amp;author=J.%20Chen&amp;author=Q.%20Wang&amp;author=Z.%20Hao&amp;author=Z.%20Li&amp;author=S.K.%20Sahu&amp;volume=86&amp;issue=4&amp;publication_year=2020&amp;pmid=31811045&amp;doi=10.1128/AEM.02465-19&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0080">
<span class="label">16.</span><cite>Voorhies A.A., Lorenzi H.A. The challenge of maintaining a healthy microbiome during long-duration space missions. Front Astronomy Space Sci. 2016;3:23.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Astronomy%20Space%20Sci&amp;title=The%20challenge%20of%20maintaining%20a%20healthy%20microbiome%20during%20long-duration%20space%20missions&amp;author=A.A.%20Voorhies&amp;author=H.A.%20Lorenzi&amp;volume=3&amp;publication_year=2016&amp;pages=23&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0085">
<span class="label">17.</span><cite>Turroni S., Rampelli S., Biagi E., Consolandi C., Severgnini M., Peano C. Temporal dynamics of the gut microbiota in people sharing a confined environment, a 520-day ground-based space simulation, MARS500. Microbiome. 2017;5(1):39. doi: 10.1186/s40168-017-0256-8.</cite> [<a href="https://doi.org/10.1186/s40168-017-0256-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5366131/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28340597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Temporal%20dynamics%20of%20the%20gut%20microbiota%20in%20people%20sharing%20a%20confined%20environment,%20a%20520-day%20ground-based%20space%20simulation,%20MARS500&amp;author=S.%20Turroni&amp;author=S.%20Rampelli&amp;author=E.%20Biagi&amp;author=C.%20Consolandi&amp;author=M.%20Severgnini&amp;volume=5&amp;issue=1&amp;publication_year=2017&amp;pages=39&amp;pmid=28340597&amp;doi=10.1186/s40168-017-0256-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0090">
<span class="label">18.</span><cite>Strollo F., Macchi C., Eberini I., Masini M., Botta M., Vassilieva G. Body composition and metabolic changes during a 520-day mission simulation to Mars. J Endocrinol Invest. 2018;41(11):1267–1273. doi: 10.1007/s40618-018-0861-9.</cite> [<a href="https://doi.org/10.1007/s40618-018-0861-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29532423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Endocrinol%20Invest&amp;title=Body%20composition%20and%20metabolic%20changes%20during%20a%20520-day%20mission%20simulation%20to%20Mars&amp;author=F.%20Strollo&amp;author=C.%20Macchi&amp;author=I.%20Eberini&amp;author=M.%20Masini&amp;author=M.%20Botta&amp;volume=41&amp;issue=11&amp;publication_year=2018&amp;pages=1267-1273&amp;pmid=29532423&amp;doi=10.1007/s40618-018-0861-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0095">
<span class="label">19.</span><cite>Mellitus* ECotDCoD. Follow-up report on the diagnosis of diabetes mellitus. Diabetes Care. 2003;26(11):3160-7.</cite> [<a href="https://doi.org/10.2337/diacare.26.11.3160" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14578255/" class="usa-link">PubMed</a>]</li>
<li id="b0100">
<span class="label">20.</span><cite>Gaffney C.J., Fomina E., Babich D., Kitov V., Uskov K., Green D.A. The effect of long-term confinement and the efficacy of exercise countermeasures on muscle strength during a simulated mission to Mars: data from the Mars500 study. Sports Medicine-Open. 2017;3(1):1–11. doi: 10.1186/s40798-017-0107-y.</cite> [<a href="https://doi.org/10.1186/s40798-017-0107-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5684057/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29134470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sports%20Medicine-Open&amp;title=The%20effect%20of%20long-term%20confinement%20and%20the%20efficacy%20of%20exercise%20countermeasures%20on%20muscle%20strength%20during%20a%20simulated%20mission%20to%20Mars:%20data%20from%20the%20Mars500%20study&amp;author=C.J.%20Gaffney&amp;author=E.%20Fomina&amp;author=D.%20Babich&amp;author=V.%20Kitov&amp;author=K.%20Uskov&amp;volume=3&amp;issue=1&amp;publication_year=2017&amp;pages=1-11&amp;pmid=29134470&amp;doi=10.1186/s40798-017-0107-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0105">
<span class="label">21.</span><cite>Roda A., Mirasoli M., Guardigli M., Simoni P., Festi D., Afonin B. Non-invasive panel tests for gastrointestinal motility monitoring within the MARS-500 Project. World Journal of Gastroenterology: WJG. 2013;19(14):2208. doi: 10.3748/wjg.v19.i14.2208.</cite> [<a href="https://doi.org/10.3748/wjg.v19.i14.2208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3627885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23599647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=World%20Journal%20of%20Gastroenterology:%20WJG&amp;title=Non-invasive%20panel%20tests%20for%20gastrointestinal%20motility%20monitoring%20within%20the%20MARS-500%20Project&amp;author=A.%20Roda&amp;author=M.%20Mirasoli&amp;author=M.%20Guardigli&amp;author=P.%20Simoni&amp;author=D.%20Festi&amp;volume=19&amp;issue=14&amp;publication_year=2013&amp;pages=2208&amp;pmid=23599647&amp;doi=10.3748/wjg.v19.i14.2208&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0110">
<span class="label">22.</span><cite>Bjarnason I. The use of fecal calprotectin in inflammatory bowel disease. Gastroenterol Hepatol. 2017;13(1):53.</cite> [<a href="/articles/PMC5390326/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28420947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterol%20Hepatol&amp;title=The%20use%20of%20fecal%20calprotectin%20in%20inflammatory%20bowel%20disease&amp;author=I.%20Bjarnason&amp;volume=13&amp;issue=1&amp;publication_year=2017&amp;pages=53&amp;pmid=28420947&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0115">
<span class="label">23.</span><cite>Yi B., Rykova M., Jäger G., Feuerecker M., Hörl M., Matzel S. Influences of large sets of environmental exposures on immune responses in healthy adult men. Sci Rep. 2015;5:13367. doi: 10.1038/srep13367.</cite> [<a href="https://doi.org/10.1038/srep13367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4549790/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26306804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Influences%20of%20large%20sets%20of%20environmental%20exposures%20on%20immune%20responses%20in%20healthy%20adult%20men&amp;author=B.%20Yi&amp;author=M.%20Rykova&amp;author=G.%20J%C3%A4ger&amp;author=M.%20Feuerecker&amp;author=M.%20H%C3%B6rl&amp;volume=5&amp;publication_year=2015&amp;pages=13367&amp;pmid=26306804&amp;doi=10.1038/srep13367&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0120">
<span class="label">24.</span><cite>Gueguinou N., Huin-Schohn C., Bascove M., Bueb J.L., Tschirhart E., Legrand-Frossi C. Could spaceflight-associated immune system weakening preclude the expansion of human presence beyond Earth's orbit? J Leukoc Biol. 2009;86(5):1027–1038. doi: 10.1189/jlb.0309167.</cite> [<a href="https://doi.org/10.1189/jlb.0309167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19690292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Leukoc%20Biol&amp;title=Could%20spaceflight-associated%20immune%20system%20weakening%20preclude%20the%20expansion%20of%20human%20presence%20beyond%20Earth's%20orbit?&amp;author=N.%20Gueguinou&amp;author=C.%20Huin-Schohn&amp;author=M.%20Bascove&amp;author=J.L.%20Bueb&amp;author=E.%20Tschirhart&amp;volume=86&amp;issue=5&amp;publication_year=2009&amp;pages=1027-1038&amp;pmid=19690292&amp;doi=10.1189/jlb.0309167&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0125">
<span class="label">25.</span><cite>Schwendner P., Mahnert A., Koskinen K., Moissl-Eichinger C., Barczyk S., Wirth R. Preparing for the crewed Mars journey: microbiota dynamics in the confined Mars500 habitat during simulated Mars flight and landing. Microbiome. 2017;5(1):129. doi: 10.1186/s40168-017-0345-8.</cite> [<a href="https://doi.org/10.1186/s40168-017-0345-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5627443/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28974259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Preparing%20for%20the%20crewed%20Mars%20journey:%20microbiota%20dynamics%20in%20the%20confined%20Mars500%20habitat%20during%20simulated%20Mars%20flight%20and%20landing&amp;author=P.%20Schwendner&amp;author=A.%20Mahnert&amp;author=K.%20Koskinen&amp;author=C.%20Moissl-Eichinger&amp;author=S.%20Barczyk&amp;volume=5&amp;issue=1&amp;publication_year=2017&amp;pages=129&amp;pmid=28974259&amp;doi=10.1186/s40168-017-0345-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0130">
<span class="label">26.</span><cite>Minerbi A., Gonzalez E., Brereton N.J., Anjarkouchian A., Dewar K., Fitzcharles M.-A. Altered microbiome composition in individuals with fibromyalgia. Pain. 2019 doi: 10.1097/j.pain.0000000000001640.</cite> [<a href="https://doi.org/10.1097/j.pain.0000000000001640" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31219947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pain&amp;title=Altered%20microbiome%20composition%20in%20individuals%20with%20fibromyalgia&amp;author=A.%20Minerbi&amp;author=E.%20Gonzalez&amp;author=N.J.%20Brereton&amp;author=A.%20Anjarkouchian&amp;author=K.%20Dewar&amp;publication_year=2019&amp;pmid=31219947&amp;doi=10.1097/j.pain.0000000000001640&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0135">
<span class="label">27.</span><cite>Gonzalez E., Brereton N.J., Li C., Leyva L.L., Solomons N.W., Agellon L.B. Distinct changes occur in the human breast milk microbiome between early and established lactation in breastfeeding Guatemalan mothers. Front Microbiol. 2021;12:194. doi: 10.3389/fmicb.2021.557180.</cite> [<a href="https://doi.org/10.3389/fmicb.2021.557180" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7907006/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33643228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Distinct%20changes%20occur%20in%20the%20human%20breast%20milk%20microbiome%20between%20early%20and%20established%20lactation%20in%20breastfeeding%20Guatemalan%20mothers&amp;author=E.%20Gonzalez&amp;author=N.J.%20Brereton&amp;author=C.%20Li&amp;author=L.L.%20Leyva&amp;author=N.W.%20Solomons&amp;volume=12&amp;publication_year=2021&amp;pages=194&amp;pmid=33643228&amp;doi=10.3389/fmicb.2021.557180&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0140">
<span class="label">28.</span><cite>Brereton N., Gonzalez E., Desjardins D., Labrecque M., Pitre F. Co-cropping with three phytoremediation crops influences rhizosphere microbiome community in contaminated soil. Sci Total Environ. 2019;135067 doi: 10.1016/j.scitotenv.2019.135067.</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2019.135067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31818595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Total%20Environ&amp;title=Co-cropping%20with%20three%20phytoremediation%20crops%20influences%20rhizosphere%20microbiome%20community%20in%20contaminated%20soil&amp;author=N.%20Brereton&amp;author=E.%20Gonzalez&amp;author=D.%20Desjardins&amp;author=M.%20Labrecque&amp;author=F.%20Pitre&amp;volume=135067&amp;publication_year=2019&amp;pmid=31818595&amp;doi=10.1016/j.scitotenv.2019.135067&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0145">
<span class="label">29.</span><cite>Porter T.M., Hajibabaei M. Scaling up: A guide to high-throughput genomic approaches for biodiversity analysis. Mol Ecol. 2018;27(2):313–338. doi: 10.1111/mec.14478.</cite> [<a href="https://doi.org/10.1111/mec.14478" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29292539/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Ecol&amp;title=Scaling%20up:%20A%20guide%20to%20high-throughput%20genomic%20approaches%20for%20biodiversity%20analysis&amp;author=T.M.%20Porter&amp;author=M.%20Hajibabaei&amp;volume=27&amp;issue=2&amp;publication_year=2018&amp;pages=313-338&amp;pmid=29292539&amp;doi=10.1111/mec.14478&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0150">
<span class="label">30.</span><cite>Knight R., Vrbanac A., Taylor B.C., Aksenov A., Callewaert C., Debelius J. Best practices for analysing microbiomes. Nat Rev Microbiol. 2018;1 doi: 10.1038/s41579-018-0029-9.</cite> [<a href="https://doi.org/10.1038/s41579-018-0029-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29795328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Microbiol&amp;title=Best%20practices%20for%20analysing%20microbiomes&amp;author=R.%20Knight&amp;author=A.%20Vrbanac&amp;author=B.C.%20Taylor&amp;author=A.%20Aksenov&amp;author=C.%20Callewaert&amp;volume=1&amp;publication_year=2018&amp;pmid=29795328&amp;doi=10.1038/s41579-018-0029-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0155">
<span class="label">31.</span><cite>Callahan B.J., McMurdie P.J., Holmes S.P. Exact sequence variants should replace operational taxonomic units in marker-gene data analysis. ISME J. 2017;11(12):2639. doi: 10.1038/ismej.2017.119.</cite> [<a href="https://doi.org/10.1038/ismej.2017.119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5702726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28731476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISME%20J&amp;title=Exact%20sequence%20variants%20should%20replace%20operational%20taxonomic%20units%20in%20marker-gene%20data%20analysis&amp;author=B.J.%20Callahan&amp;author=P.J.%20McMurdie&amp;author=S.P.%20Holmes&amp;volume=11&amp;issue=12&amp;publication_year=2017&amp;pages=2639&amp;pmid=28731476&amp;doi=10.1038/ismej.2017.119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0160">
<span class="label">32.</span><cite>Edgar R.C. UNOISE2: improved error-correction for Illumina 16S and ITS amplicon sequencing. BioRxiv. 2016;081257</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=BioRxiv&amp;title=UNOISE2:%20improved%20error-correction%20for%20Illumina%2016S%20and%20ITS%20amplicon%20sequencing&amp;author=R.C.%20Edgar&amp;volume=081257&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0165">
<span class="label">33.</span><cite>Schloss P.D., Westcott S.L., Ryabin T., Hall J.R., Hartmann M., Hollister E.B. Introducing mothur: open-source, platform-independent, community-supported software for describing and comparing microbial communities. Appl Environ Microbiol. 2009;75(23):7537–7541. doi: 10.1128/AEM.01541-09.</cite> [<a href="https://doi.org/10.1128/AEM.01541-09" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2786419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19801464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Introducing%20mothur:%20open-source,%20platform-independent,%20community-supported%20software%20for%20describing%20and%20comparing%20microbial%20communities&amp;author=P.D.%20Schloss&amp;author=S.L.%20Westcott&amp;author=T.%20Ryabin&amp;author=J.R.%20Hall&amp;author=M.%20Hartmann&amp;volume=75&amp;issue=23&amp;publication_year=2009&amp;pages=7537-7541&amp;pmid=19801464&amp;doi=10.1128/AEM.01541-09&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0170">
<span class="label">34.</span><cite>David L.A., Materna A.C., Friedman J., Campos-Baptista M.I., Blackburn M.C., Perrotta A. Host lifestyle affects human microbiota on daily timescales. Genome Biol. 2014;15(7):R89. doi: 10.1186/gb-2014-15-7-r89.</cite> [<a href="https://doi.org/10.1186/gb-2014-15-7-r89" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4405912/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25146375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Host%20lifestyle%20affects%20human%20microbiota%20on%20daily%20timescales&amp;author=L.A.%20David&amp;author=A.C.%20Materna&amp;author=J.%20Friedman&amp;author=M.I.%20Campos-Baptista&amp;author=M.C.%20Blackburn&amp;volume=15&amp;issue=7&amp;publication_year=2014&amp;pages=R89&amp;pmid=25146375&amp;doi=10.1186/gb-2014-15-7-r89&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0175">
<span class="label">35.</span><cite>Hernandez-Rodriguez J., Arandjelovic M., Lester J., de Filippo C., Weihmann A., Meyer M. The impact of endogenous content, replicates and pooling on genome capture from faecal samples. Mol Ecol Resour. 2018;18(2):319–333. doi: 10.1111/1755-0998.12728.</cite> [<a href="https://doi.org/10.1111/1755-0998.12728" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5900898/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29058768/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Ecol%20Resour&amp;title=The%20impact%20of%20endogenous%20content,%20replicates%20and%20pooling%20on%20genome%20capture%20from%20faecal%20samples&amp;author=J.%20Hernandez-Rodriguez&amp;author=M.%20Arandjelovic&amp;author=J.%20Lester&amp;author=C.%20de%20Filippo&amp;author=A.%20Weihmann&amp;volume=18&amp;issue=2&amp;publication_year=2018&amp;pages=319-333&amp;pmid=29058768&amp;doi=10.1111/1755-0998.12728&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0180">
<span class="label">36.</span><cite>Wu G.D., Lewis J.D., Hoffmann C., Chen Y.-Y., Knight R., Bittinger K. Sampling and pyrosequencing methods for characterizing bacterial communities in the human gut using 16S sequence tags. BMC Microbiol. 2010;10(1):206. doi: 10.1186/1471-2180-10-206.</cite> [<a href="https://doi.org/10.1186/1471-2180-10-206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2921404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20673359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol&amp;title=Sampling%20and%20pyrosequencing%20methods%20for%20characterizing%20bacterial%20communities%20in%20the%20human%20gut%20using%2016S%20sequence%20tags&amp;author=G.D.%20Wu&amp;author=J.D.%20Lewis&amp;author=C.%20Hoffmann&amp;author=Y.-Y.%20Chen&amp;author=R.%20Knight&amp;volume=10&amp;issue=1&amp;publication_year=2010&amp;pages=206&amp;pmid=20673359&amp;doi=10.1186/1471-2180-10-206&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0185">
<span class="label">37.</span><cite>Ji B.W., Sheth R.U., Dixit P.D., Huang Y., Kaufman A., Wang H.H. Quantifying spatiotemporal variability and noise in absolute microbiota abundances using replicate sampling. Nat Methods. 2019;16(8):731–736. doi: 10.1038/s41592-019-0467-y.</cite> [<a href="https://doi.org/10.1038/s41592-019-0467-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7219825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31308552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Quantifying%20spatiotemporal%20variability%20and%20noise%20in%20absolute%20microbiota%20abundances%20using%20replicate%20sampling&amp;author=B.W.%20Ji&amp;author=R.U.%20Sheth&amp;author=P.D.%20Dixit&amp;author=Y.%20Huang&amp;author=A.%20Kaufman&amp;volume=16&amp;issue=8&amp;publication_year=2019&amp;pages=731-736&amp;pmid=31308552&amp;doi=10.1038/s41592-019-0467-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0190">
<span class="label">38.</span><cite>Love M., Anders S., Huber W. Differential analysis of count data–the DESeq2 package. Genome Biol. 2014;15:550. doi: 10.1186/s13059-014-0550-8.</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Differential%20analysis%20of%20count%20data%E2%80%93the%20DESeq2%20package&amp;author=M.%20Love&amp;author=S.%20Anders&amp;author=W.%20Huber&amp;volume=15&amp;publication_year=2014&amp;pages=550&amp;pmid=25516281&amp;doi=10.1186/s13059-014-0550-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0195">
<span class="label">39.</span><cite>Thorsen J., Brejnrod A., Mortensen M., Rasmussen M.A., Stokholm J., Al-Soud W.A. Large-scale benchmarking reveals false discoveries and count transformation sensitivity in 16S rRNA gene amplicon data analysis methods used in microbiome studies. Microbiome. 2016;4(1):62. doi: 10.1186/s40168-016-0208-8.</cite> [<a href="https://doi.org/10.1186/s40168-016-0208-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5123278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27884206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Large-scale%20benchmarking%20reveals%20false%20discoveries%20and%20count%20transformation%20sensitivity%20in%2016S%20rRNA%20gene%20amplicon%20data%20analysis%20methods%20used%20in%20microbiome%20studies&amp;author=J.%20Thorsen&amp;author=A.%20Brejnrod&amp;author=M.%20Mortensen&amp;author=M.A.%20Rasmussen&amp;author=J.%20Stokholm&amp;volume=4&amp;issue=1&amp;publication_year=2016&amp;pages=62&amp;pmid=27884206&amp;doi=10.1186/s40168-016-0208-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0200">
<span class="label">40.</span><cite>Weiss S., Xu Z.Z., Peddada S., Amir A., Bittinger K., Gonzalez A. Normalization and microbial differential abundance strategies depend upon data characteristics. Microbiome. 2017;5(1):27. doi: 10.1186/s40168-017-0237-y.</cite> [<a href="https://doi.org/10.1186/s40168-017-0237-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5335496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28253908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Normalization%20and%20microbial%20differential%20abundance%20strategies%20depend%20upon%20data%20characteristics&amp;author=S.%20Weiss&amp;author=Z.Z.%20Xu&amp;author=S.%20Peddada&amp;author=A.%20Amir&amp;author=K.%20Bittinger&amp;volume=5&amp;issue=1&amp;publication_year=2017&amp;pages=27&amp;pmid=28253908&amp;doi=10.1186/s40168-017-0237-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0205">
<span class="label">41.</span><cite>Love M.I., Huber W., Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 2014;15(12):550. doi: 10.1186/s13059-014-0550-8.</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&amp;author=M.I.%20Love&amp;author=W.%20Huber&amp;author=S.%20Anders&amp;volume=15&amp;issue=12&amp;publication_year=2014&amp;pages=550&amp;pmid=25516281&amp;doi=10.1186/s13059-014-0550-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0210">
<span class="label">42.</span><cite>Oksanen J, Blanchet FG, Kindt R, Legendre P, Minchin PR, O’hara R, et al. Package ‘vegan’. Community ecology package, version2017.</cite>
</li>
<li id="b0215">
<span class="label">43.</span><cite>Browne H.P., Forster S.C., Anonye B.O., Kumar N., Neville B.A., Stares M.D. Culturing of ‘unculturable’human microbiota reveals novel taxa and extensive sporulation. Nature. 2016;533(7604):543–546. doi: 10.1038/nature17645.</cite> [<a href="https://doi.org/10.1038/nature17645" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4890681/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27144353/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Culturing%20of%20%E2%80%98unculturable%E2%80%99human%20microbiota%20reveals%20novel%20taxa%20and%20extensive%20sporulation&amp;author=H.P.%20Browne&amp;author=S.C.%20Forster&amp;author=B.O.%20Anonye&amp;author=N.%20Kumar&amp;author=B.A.%20Neville&amp;volume=533&amp;issue=7604&amp;publication_year=2016&amp;pages=543-546&amp;pmid=27144353&amp;doi=10.1038/nature17645&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0220">
<span class="label">44.</span><cite>Kassinen A., Krogius-Kurikka L., Mäkivuokko H., Rinttilä T., Paulin L., Corander J. The fecal microbiota of irritable bowel syndrome patients differs significantly from that of healthy subjects. Gastroenterology. 2007;133(1):24–33. doi: 10.1053/j.gastro.2007.04.005.</cite> [<a href="https://doi.org/10.1053/j.gastro.2007.04.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17631127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=The%20fecal%20microbiota%20of%20irritable%20bowel%20syndrome%20patients%20differs%20significantly%20from%20that%20of%20healthy%20subjects&amp;author=A.%20Kassinen&amp;author=L.%20Krogius-Kurikka&amp;author=H.%20M%C3%A4kivuokko&amp;author=T.%20Rinttil%C3%A4&amp;author=L.%20Paulin&amp;volume=133&amp;issue=1&amp;publication_year=2007&amp;pages=24-33&amp;pmid=17631127&amp;doi=10.1053/j.gastro.2007.04.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0225">
<span class="label">45.</span><cite>La Reau A.J., Meier-Kolthoff J.P., Suen G. Sequence-based analysis of the genus Ruminococcus resolves its phylogeny and reveals strong host association. Microb Genomics. 2016;2(12) doi: 10.1099/mgen.0.000099.</cite> [<a href="https://doi.org/10.1099/mgen.0.000099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5359413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28348838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microb%20Genomics&amp;title=Sequence-based%20analysis%20of%20the%20genus%20Ruminococcus%20resolves%20its%20phylogeny%20and%20reveals%20strong%20host%20association&amp;author=A.J.%20La%20Reau&amp;author=J.P.%20Meier-Kolthoff&amp;author=G.%20Suen&amp;volume=2&amp;issue=12&amp;publication_year=2016&amp;pmid=28348838&amp;doi=10.1099/mgen.0.000099&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0230">
<span class="label">46.</span><cite>Li M., Wang B., Zhang M., Rantalainen M., Wang S., Zhou H. Symbiotic gut microbes modulate human metabolic phenotypes. Proc Natl Acad Sci. 2008;105(6):2117–2122. doi: 10.1073/pnas.0712038105.</cite> [<a href="https://doi.org/10.1073/pnas.0712038105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2538887/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18252821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci&amp;title=Symbiotic%20gut%20microbes%20modulate%20human%20metabolic%20phenotypes&amp;author=M.%20Li&amp;author=B.%20Wang&amp;author=M.%20Zhang&amp;author=M.%20Rantalainen&amp;author=S.%20Wang&amp;volume=105&amp;issue=6&amp;publication_year=2008&amp;pages=2117-2122&amp;pmid=18252821&amp;doi=10.1073/pnas.0712038105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0235">
<span class="label">47.</span><cite>Lin A., Bik E.M., Costello E.K., Dethlefsen L., Haque R., Relman D.A. Distinct distal gut microbiome diversity and composition in healthy children from Bangladesh and the United States. PLoS ONE. 2013;8(1) doi: 10.1371/journal.pone.0053838.</cite> [<a href="https://doi.org/10.1371/journal.pone.0053838" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3551965/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23349750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Distinct%20distal%20gut%20microbiome%20diversity%20and%20composition%20in%20healthy%20children%20from%20Bangladesh%20and%20the%20United%20States&amp;author=A.%20Lin&amp;author=E.M.%20Bik&amp;author=E.K.%20Costello&amp;author=L.%20Dethlefsen&amp;author=R.%20Haque&amp;volume=8&amp;issue=1&amp;publication_year=2013&amp;pmid=23349750&amp;doi=10.1371/journal.pone.0053838&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0240">
<span class="label">48.</span><cite>Ley R.E., Turnbaugh P.J., Klein S., Gordon J.I. Human gut microbes associated with obesity. Nature. 2006;444(7122):1022–1023. doi: 10.1038/4441022a.</cite> [<a href="https://doi.org/10.1038/4441022a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17183309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Human%20gut%20microbes%20associated%20with%20obesity&amp;author=R.E.%20Ley&amp;author=P.J.%20Turnbaugh&amp;author=S.%20Klein&amp;author=J.I.%20Gordon&amp;volume=444&amp;issue=7122&amp;publication_year=2006&amp;pages=1022-1023&amp;pmid=17183309&amp;doi=10.1038/4441022a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0245">
<span class="label">49.</span><cite>Ley R.E., Hamady M., Lozupone C., Turnbaugh P.J., Ramey R.R., Bircher J.S. Evolution of mammals and their gut microbes. Science. 2008;320(5883):1647–1651. doi: 10.1126/science.1155725.</cite> [<a href="https://doi.org/10.1126/science.1155725" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2649005/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18497261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Evolution%20of%20mammals%20and%20their%20gut%20microbes&amp;author=R.E.%20Ley&amp;author=M.%20Hamady&amp;author=C.%20Lozupone&amp;author=P.J.%20Turnbaugh&amp;author=R.R.%20Ramey&amp;volume=320&amp;issue=5883&amp;publication_year=2008&amp;pages=1647-1651&amp;pmid=18497261&amp;doi=10.1126/science.1155725&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0250">
<span class="label">50.</span><cite>Harrell L., Wang Y., Antonopoulos D., Young V., Lichtenstein L., Huang Y. Standard colonic lavage alters the natural state of mucosal-associated microbiota in the human colon. PLoS ONE. 2012;7(2) doi: 10.1371/journal.pone.0032545.</cite> [<a href="https://doi.org/10.1371/journal.pone.0032545" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3289660/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22389708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Standard%20colonic%20lavage%20alters%20the%20natural%20state%20of%20mucosal-associated%20microbiota%20in%20the%20human%20colon&amp;author=L.%20Harrell&amp;author=Y.%20Wang&amp;author=D.%20Antonopoulos&amp;author=V.%20Young&amp;author=L.%20Lichtenstein&amp;volume=7&amp;issue=2&amp;publication_year=2012&amp;pmid=22389708&amp;doi=10.1371/journal.pone.0032545&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0255">
<span class="label">51.</span><cite>Ramadass B., Rani B.S., Srinivasan Pugazhendhi K., Ramakrishna B.S. Faecal microbiota of healthy adults in south India: comparison of a tribal &amp; a rural population. Indian J Med Res. 2017;145(2):237. doi: 10.4103/ijmr.IJMR_639_14.</cite> [<a href="https://doi.org/10.4103/ijmr.IJMR_639_14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5501057/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28639601/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Indian%20J%20Med%20Res&amp;title=Faecal%20microbiota%20of%20healthy%20adults%20in%20south%20India:%20comparison%20of%20a%20tribal%20&amp;%20a%20rural%20population&amp;author=B.%20Ramadass&amp;author=B.S.%20Rani&amp;author=K.%20Srinivasan%20Pugazhendhi&amp;author=B.S.%20Ramakrishna&amp;volume=145&amp;issue=2&amp;publication_year=2017&amp;pages=237&amp;pmid=28639601&amp;doi=10.4103/ijmr.IJMR_639_14&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0260">
<span class="label">52.</span><cite>Walker A.W., Ince J., Duncan S.H., Webster L.M., Holtrop G., Ze X. Dominant and diet-responsive groups of bacteria within the human colonic microbiota. ISME J. 2011;5(2):220–230. doi: 10.1038/ismej.2010.118.</cite> [<a href="https://doi.org/10.1038/ismej.2010.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3105703/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20686513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISME%20J&amp;title=Dominant%20and%20diet-responsive%20groups%20of%20bacteria%20within%20the%20human%20colonic%20microbiota&amp;author=A.W.%20Walker&amp;author=J.%20Ince&amp;author=S.H.%20Duncan&amp;author=L.M.%20Webster&amp;author=G.%20Holtrop&amp;volume=5&amp;issue=2&amp;publication_year=2011&amp;pages=220-230&amp;pmid=20686513&amp;doi=10.1038/ismej.2010.118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0265">
<span class="label">53.</span><cite>Almeida A., Mitchell A.L., Boland M., Forster S.C., Gloor G.B., Tarkowska A. A new genomic blueprint of the human gut microbiota. Nature. 2019;568(7753):499. doi: 10.1038/s41586-019-0965-1.</cite> [<a href="https://doi.org/10.1038/s41586-019-0965-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6784870/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30745586/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20new%20genomic%20blueprint%20of%20the%20human%20gut%20microbiota&amp;author=A.%20Almeida&amp;author=A.L.%20Mitchell&amp;author=M.%20Boland&amp;author=S.C.%20Forster&amp;author=G.B.%20Gloor&amp;volume=568&amp;issue=7753&amp;publication_year=2019&amp;pages=499&amp;pmid=30745586&amp;doi=10.1038/s41586-019-0965-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0270">
<span class="label">54.</span><cite>Hynönen U., Zoetendal E.G., Virtala A.-M.-K., Shetty S., Hasan S., Jakava-Viljanen M. Molecular ecology of the yet uncultured bacterial Ct85-cluster in the mammalian gut. Anaerobe. 2019;102104 doi: 10.1016/j.anaerobe.2019.102104.</cite> [<a href="https://doi.org/10.1016/j.anaerobe.2019.102104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31562947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anaerobe&amp;title=Molecular%20ecology%20of%20the%20yet%20uncultured%20bacterial%20Ct85-cluster%20in%20the%20mammalian%20gut&amp;author=U.%20Hyn%C3%B6nen&amp;author=E.G.%20Zoetendal&amp;author=A.-M.-K.%20Virtala&amp;author=S.%20Shetty&amp;author=S.%20Hasan&amp;volume=102104&amp;publication_year=2019&amp;pmid=31562947&amp;doi=10.1016/j.anaerobe.2019.102104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0275">
<span class="label">55.</span><cite>Benevides L., Burman S., Martin R., Robert V., Thomas M., Miquel S. New insights into the diversity of the genus Faecalibacterium. Front Microbiol. 2017;8:1790. doi: 10.3389/fmicb.2017.01790.</cite> [<a href="https://doi.org/10.3389/fmicb.2017.01790" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5609107/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28970823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=New%20insights%20into%20the%20diversity%20of%20the%20genus%20Faecalibacterium&amp;author=L.%20Benevides&amp;author=S.%20Burman&amp;author=R.%20Martin&amp;author=V.%20Robert&amp;author=M.%20Thomas&amp;volume=8&amp;publication_year=2017&amp;pages=1790&amp;pmid=28970823&amp;doi=10.3389/fmicb.2017.01790&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0280">
<span class="label">56.</span><cite>Martín R., Miquel S., Benevides L., Bridonneau C., Robert V., Hudault S. Functional characterization of novel Faecalibacterium prausnitzii strains isolated from healthy volunteers: a step forward in the use of F. prausnitzii as a next-generation probiotic. Front Microbiol. 2017;8:1226. doi: 10.3389/fmicb.2017.01226.</cite> [<a href="https://doi.org/10.3389/fmicb.2017.01226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5492426/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28713353/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Functional%20characterization%20of%20novel%20Faecalibacterium%20prausnitzii%20strains%20isolated%20from%20healthy%20volunteers:%20a%20step%20forward%20in%20the%20use%20of%20F.%20prausnitzii%20as%20a%20next-generation%20probiotic&amp;author=R.%20Mart%C3%ADn&amp;author=S.%20Miquel&amp;author=L.%20Benevides&amp;author=C.%20Bridonneau&amp;author=V.%20Robert&amp;volume=8&amp;publication_year=2017&amp;pages=1226&amp;pmid=28713353&amp;doi=10.3389/fmicb.2017.01226&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0285">
<span class="label">57.</span><cite>Fitzgerald C.B., Shkoporov A.N., Sutton T.D., Chaplin A.V., Velayudhan V., Ross R.P. Comparative analysis of Faecalibacterium prausnitzii genomes shows a high level of genome plasticity and warrants separation into new species-level taxa. BMC Genomics. 2018;19(1):931. doi: 10.1186/s12864-018-5313-6.</cite> [<a href="https://doi.org/10.1186/s12864-018-5313-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6295017/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30547746/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Comparative%20analysis%20of%20Faecalibacterium%20prausnitzii%20genomes%20shows%20a%20high%20level%20of%20genome%20plasticity%20and%20warrants%20separation%20into%20new%20species-level%20taxa&amp;author=C.B.%20Fitzgerald&amp;author=A.N.%20Shkoporov&amp;author=T.D.%20Sutton&amp;author=A.V.%20Chaplin&amp;author=V.%20Velayudhan&amp;volume=19&amp;issue=1&amp;publication_year=2018&amp;pages=931&amp;pmid=30547746&amp;doi=10.1186/s12864-018-5313-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0290">
<span class="label">58.</span><cite>Vesely RC, Giani G, Maiocchi G, inventorsCompletely natural dessert cream comprising fructooligosaccharides. European Patent Office patent 98830393.9. 2008.</cite>
</li>
<li id="b0295">
<span class="label">59.</span><cite>Santos G.d., Nogueira R.I., Rosenthal A. Powdered yoghurt produced by spray drying and freeze drying: a review. Braz J Food Technol. 2018;21</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Braz%20J%20Food%20Technol&amp;title=Powdered%20yoghurt%20produced%20by%20spray%20drying%20and%20freeze%20drying:%20a%20review&amp;author=G.d.%20Santos&amp;author=R.I.%20Nogueira&amp;author=A.%20Rosenthal&amp;volume=21&amp;publication_year=2018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0300">
<span class="label">60.</span><cite>Amann R.I., Ludwig W., Schleifer K.-H. Phylogenetic identification and in situ detection of individual microbial cells without cultivation. Microbiol Rev. 1995;59(1):143–169. doi: 10.1128/mr.59.1.143-169.1995.</cite> [<a href="https://doi.org/10.1128/mr.59.1.143-169.1995" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC239358/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7535888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol%20Rev&amp;title=Phylogenetic%20identification%20and%20in%20situ%20detection%20of%20individual%20microbial%20cells%20without%20cultivation&amp;author=R.I.%20Amann&amp;author=W.%20Ludwig&amp;author=K.-H.%20Schleifer&amp;volume=59&amp;issue=1&amp;publication_year=1995&amp;pages=143-169&amp;pmid=7535888&amp;doi=10.1128/mr.59.1.143-169.1995&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0305">
<span class="label">61.</span><cite>Ueki A., Ohtaki Y., Kaku N., Ueki K. Descriptions of Anaerotaenia torta gen. nov., sp. nov. and Anaerocolumna cellulosilytica gen. nov., sp. nov. isolated from a methanogenic reactor of cattle waste and reclassification of Clostridium aminovalericum, Clostridium jejuense and Clostridium xylanovorans as Anaerocolumna species. Int J Syst Evol Micr. 2016;66(8):2936–2943. doi: 10.1099/ijsem.0.001123.</cite> [<a href="https://doi.org/10.1099/ijsem.0.001123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27126251/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Micr&amp;title=Descriptions%20of%20Anaerotaenia%20torta%20gen.%20nov.,%20sp.%20nov.%20and%20Anaerocolumna%20cellulosilytica%20gen.%20nov.,%20sp.%20nov.%20isolated%20from%20a%20methanogenic%20reactor%20of%20cattle%20waste%20and%20reclassification%20of%20Clostridium%20aminovalericum,%20Clostridium%20jejuense%20and%20Clostridium%20xylanovorans%20as%20Anaerocolumna%20species&amp;author=A.%20Ueki&amp;author=Y.%20Ohtaki&amp;author=N.%20Kaku&amp;author=K.%20Ueki&amp;volume=66&amp;issue=8&amp;publication_year=2016&amp;pages=2936-2943&amp;pmid=27126251&amp;doi=10.1099/ijsem.0.001123&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0310">
<span class="label">62.</span><cite>Kim C.C., Kelly W.J., Patchett M.L., Tannock G.W., Jordens Z., Stoklosinski H.M. Monoglobus pectinilyticus gen. nov., sp. nov., a pectinolytic bacterium isolated from human faeces. Int J Syst Evol Micr. 2017;67(12):4992–4998. doi: 10.1099/ijsem.0.002395.</cite> [<a href="https://doi.org/10.1099/ijsem.0.002395" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29039307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Micr&amp;title=Monoglobus%20pectinilyticus%20gen.%20nov.,%20sp.%20nov.,%20a%20pectinolytic%20bacterium%20isolated%20from%20human%20faeces&amp;author=C.C.%20Kim&amp;author=W.J.%20Kelly&amp;author=M.L.%20Patchett&amp;author=G.W.%20Tannock&amp;author=Z.%20Jordens&amp;volume=67&amp;issue=12&amp;publication_year=2017&amp;pages=4992-4998&amp;pmid=29039307&amp;doi=10.1099/ijsem.0.002395&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0315">
<span class="label">63.</span><cite>Bui T.P.N., Shetty S.A., Lagkouvardos I., Ritari J., Chamlagain B., Douillard F.P. Comparative genomics and physiology of the butyrate-producing bacterium Intestinimonas butyriciproducens. Environ Microbiol Rep. 2016;8(6):1024–1037. doi: 10.1111/1758-2229.12483.</cite> [<a href="https://doi.org/10.1111/1758-2229.12483" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27717172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environ%20Microbiol%20Rep&amp;title=Comparative%20genomics%20and%20physiology%20of%20the%20butyrate-producing%20bacterium%20Intestinimonas%20butyriciproducens&amp;author=T.P.N.%20Bui&amp;author=S.A.%20Shetty&amp;author=I.%20Lagkouvardos&amp;author=J.%20Ritari&amp;author=B.%20Chamlagain&amp;volume=8&amp;issue=6&amp;publication_year=2016&amp;pages=1024-1037&amp;pmid=27717172&amp;doi=10.1111/1758-2229.12483&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0320">
<span class="label">64.</span><cite>Kou S., Vincent G., Gonzalez E., Pitre F.E., Labrecque M., Brereton N.J. The response of a 16S ribosomal RNA gene fragment amplified community to lead, zinc, and copper pollution in a shanghai field trial. Front Microbiol. 2018;9:366. doi: 10.3389/fmicb.2018.00366.</cite> [<a href="https://doi.org/10.3389/fmicb.2018.00366" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5838024/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29545788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=The%20response%20of%20a%2016S%20ribosomal%20RNA%20gene%20fragment%20amplified%20community%20to%20lead,%20zinc,%20and%20copper%20pollution%20in%20a%20shanghai%20field%20trial&amp;author=S.%20Kou&amp;author=G.%20Vincent&amp;author=E.%20Gonzalez&amp;author=F.E.%20Pitre&amp;author=M.%20Labrecque&amp;volume=9&amp;publication_year=2018&amp;pages=366&amp;pmid=29545788&amp;doi=10.3389/fmicb.2018.00366&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0325">
<span class="label">65.</span><cite>Gonzalez E., Pitre F., Pagé A., Marleau J., Nissim W.G., St-Arnaud M. Trees, fungi and bacteria: tripartite metatranscriptomics of a root microbiome responding to soil contamination. Microbiome. 2018;6(1):53. doi: 10.1186/s40168-018-0432-5.</cite> [<a href="https://doi.org/10.1186/s40168-018-0432-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5863371/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29562928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Trees,%20fungi%20and%20bacteria:%20tripartite%20metatranscriptomics%20of%20a%20root%20microbiome%20responding%20to%20soil%20contamination&amp;author=E.%20Gonzalez&amp;author=F.%20Pitre&amp;author=A.%20Pag%C3%A9&amp;author=J.%20Marleau&amp;author=W.G.%20Nissim&amp;volume=6&amp;issue=1&amp;publication_year=2018&amp;pages=53&amp;pmid=29562928&amp;doi=10.1186/s40168-018-0432-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0330">
<span class="label">66.</span><cite>Hugenholtz P., Tyson G.W., Webb R.I., Wagner A.M., Blackall L.L. Investigation of candidate division TM7, a recently recognized major lineage of the domain Bacteria with no known pure-culture representatives. Appl Environ Microbiol. 2001;67(1):411–419. doi: 10.1128/AEM.67.1.411-419.2001.</cite> [<a href="https://doi.org/10.1128/AEM.67.1.411-419.2001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC92593/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11133473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Investigation%20of%20candidate%20division%20TM7,%20a%20recently%20recognized%20major%20lineage%20of%20the%20domain%20Bacteria%20with%20no%20known%20pure-culture%20representatives&amp;author=P.%20Hugenholtz&amp;author=G.W.%20Tyson&amp;author=R.I.%20Webb&amp;author=A.M.%20Wagner&amp;author=L.L.%20Blackall&amp;volume=67&amp;issue=1&amp;publication_year=2001&amp;pages=411-419&amp;pmid=11133473&amp;doi=10.1128/AEM.67.1.411-419.2001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0335">
<span class="label">67.</span><cite>He X., McLean J.S., Edlund A., Yooseph S., Hall A.P., Liu S.-Y. Cultivation of a human-associated TM7 phylotype reveals a reduced genome and epibiotic parasitic lifestyle. Proc Natl Acad Sci. 2015;112(1):244–249. doi: 10.1073/pnas.1419038112.</cite> [<a href="https://doi.org/10.1073/pnas.1419038112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4291631/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25535390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci&amp;title=Cultivation%20of%20a%20human-associated%20TM7%20phylotype%20reveals%20a%20reduced%20genome%20and%20epibiotic%20parasitic%20lifestyle&amp;author=X.%20He&amp;author=J.S.%20McLean&amp;author=A.%20Edlund&amp;author=S.%20Yooseph&amp;author=A.P.%20Hall&amp;volume=112&amp;issue=1&amp;publication_year=2015&amp;pages=244-249&amp;pmid=25535390&amp;doi=10.1073/pnas.1419038112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0340">
<span class="label">68.</span><cite>Quévrain E., Maubert M., Michon C., Chain F., Marquant R., Tailhades J. Identification of an anti-inflammatory protein from Faecalibacterium prausnitzii, a commensal bacterium deficient in Crohn’s disease. Gut. 2016;65(3):415–425. doi: 10.1136/gutjnl-2014-307649.</cite> [<a href="https://doi.org/10.1136/gutjnl-2014-307649" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5136800/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26045134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Identification%20of%20an%20anti-inflammatory%20protein%20from%20Faecalibacterium%20prausnitzii,%20a%20commensal%20bacterium%20deficient%20in%20Crohn%E2%80%99s%20disease&amp;author=E.%20Qu%C3%A9vrain&amp;author=M.%20Maubert&amp;author=C.%20Michon&amp;author=F.%20Chain&amp;author=R.%20Marquant&amp;volume=65&amp;issue=3&amp;publication_year=2016&amp;pages=415-425&amp;pmid=26045134&amp;doi=10.1136/gutjnl-2014-307649&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0345">
<span class="label">69.</span><cite>Lopez-Siles M., Duncan S.H., Garcia-Gil L.J., Martinez-Medina M. Faecalibacterium prausnitzii: from microbiology to diagnostics and prognostics. ISME J. 2017;11(4):841–852. doi: 10.1038/ismej.2016.176.</cite> [<a href="https://doi.org/10.1038/ismej.2016.176" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5364359/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28045459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISME%20J&amp;title=Faecalibacterium%20prausnitzii:%20from%20microbiology%20to%20diagnostics%20and%20prognostics&amp;author=M.%20Lopez-Siles&amp;author=S.H.%20Duncan&amp;author=L.J.%20Garcia-Gil&amp;author=M.%20Martinez-Medina&amp;volume=11&amp;issue=4&amp;publication_year=2017&amp;pages=841-852&amp;pmid=28045459&amp;doi=10.1038/ismej.2016.176&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0350">
<span class="label">70.</span><cite>Klingberg E., Magnusson M.K., Strid H., Deminger A., Ståhl A., Sundin J. A distinct gut microbiota composition in patients with ankylosing spondylitis is associated with increased levels of fecal calprotectin. Arthritis Res Ther. 2019;21(1):248. doi: 10.1186/s13075-019-2018-4.</cite> [<a href="https://doi.org/10.1186/s13075-019-2018-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6880506/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31771630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis%20Res%20Ther&amp;title=A%20distinct%20gut%20microbiota%20composition%20in%20patients%20with%20ankylosing%20spondylitis%20is%20associated%20with%20increased%20levels%20of%20fecal%20calprotectin&amp;author=E.%20Klingberg&amp;author=M.K.%20Magnusson&amp;author=H.%20Strid&amp;author=A.%20Deminger&amp;author=A.%20St%C3%A5hl&amp;volume=21&amp;issue=1&amp;publication_year=2019&amp;pages=248&amp;pmid=31771630&amp;doi=10.1186/s13075-019-2018-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0355">
<span class="label">71.</span><cite>Kowalska-Duplaga K., Gosiewski T., Kapusta P., Sroka-Oleksiak A., Wędrychowicz A., Pieczarkowski S. Differences in the intestinal microbiome of healthy children and patients with newly diagnosed Crohn’s disease. Sci Rep. 2019;9(1):1–11. doi: 10.1038/s41598-019-55290-9.</cite> [<a href="https://doi.org/10.1038/s41598-019-55290-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6906406/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31827191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Differences%20in%20the%20intestinal%20microbiome%20of%20healthy%20children%20and%20patients%20with%20newly%20diagnosed%20Crohn%E2%80%99s%20disease&amp;author=K.%20Kowalska-Duplaga&amp;author=T.%20Gosiewski&amp;author=P.%20Kapusta&amp;author=A.%20Sroka-Oleksiak&amp;author=A.%20W%C4%99drychowicz&amp;volume=9&amp;issue=1&amp;publication_year=2019&amp;pages=1-11&amp;pmid=31827191&amp;doi=10.1038/s41598-019-55290-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0360">
<span class="label">72.</span><cite>Gurung M., Li Z., You H., Rodrigues R., Jump D.B., Morgun A. Role of gut microbiota in type 2 diabetes pathophysiology. EBioMedicine. 2020;51 doi: 10.1016/j.ebiom.2019.11.051.</cite> [<a href="https://doi.org/10.1016/j.ebiom.2019.11.051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6948163/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31901868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EBioMedicine&amp;title=Role%20of%20gut%20microbiota%20in%20type%202%20diabetes%20pathophysiology&amp;author=M.%20Gurung&amp;author=Z.%20Li&amp;author=H.%20You&amp;author=R.%20Rodrigues&amp;author=D.B.%20Jump&amp;volume=51&amp;publication_year=2020&amp;pmid=31901868&amp;doi=10.1016/j.ebiom.2019.11.051&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0365">
<span class="label">73.</span><cite>Lopez-Siles M., Martinez-Medina M., Surís-Valls R., Aldeguer X., Sabat-Mir M., Duncan S.H. Changes in the abundance of Faecalibacterium prausnitzii phylogroups I and II in the intestinal mucosa of inflammatory bowel disease and patients with colorectal cancer. Inflamm Bowel Dis. 2016;22(1):28–41. doi: 10.1097/MIB.0000000000000590.</cite> [<a href="https://doi.org/10.1097/MIB.0000000000000590" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26595550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Inflamm%20Bowel%20Dis&amp;title=Changes%20in%20the%20abundance%20of%20Faecalibacterium%20prausnitzii%20phylogroups%20I%20and%20II%20in%20the%20intestinal%20mucosa%20of%20inflammatory%20bowel%20disease%20and%20patients%20with%20colorectal%20cancer&amp;author=M.%20Lopez-Siles&amp;author=M.%20Martinez-Medina&amp;author=R.%20Sur%C3%ADs-Valls&amp;author=X.%20Aldeguer&amp;author=M.%20Sabat-Mir&amp;volume=22&amp;issue=1&amp;publication_year=2016&amp;pages=28-41&amp;pmid=26595550&amp;doi=10.1097/MIB.0000000000000590&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0370">
<span class="label">74.</span><cite>Mukhopadhya I., Moraïs S., Laverde-Gomez J., Sheridan P.O., Walker A.W., Kelly W. Sporulation capability and amylosome conservation among diverse human colonic and rumen isolates of the keystone starch-degrader Ruminococcus bromii. Environ Microbiol. 2018;20(1):324–336. doi: 10.1111/1462-2920.14000.</cite> [<a href="https://doi.org/10.1111/1462-2920.14000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5814915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29159997/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environ%20Microbiol&amp;title=Sporulation%20capability%20and%20amylosome%20conservation%20among%20diverse%20human%20colonic%20and%20rumen%20isolates%20of%20the%20keystone%20starch-degrader%20Ruminococcus%20bromii&amp;author=I.%20Mukhopadhya&amp;author=S.%20Mora%C3%AFs&amp;author=J.%20Laverde-Gomez&amp;author=P.O.%20Sheridan&amp;author=A.W.%20Walker&amp;volume=20&amp;issue=1&amp;publication_year=2018&amp;pages=324-336&amp;pmid=29159997&amp;doi=10.1111/1462-2920.14000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0375">
<span class="label">75.</span><cite>Ze X., Duncan S.H., Louis P., Flint H.J. Ruminococcus bromii is a keystone species for the degradation of resistant starch in the human colon. ISME J. 2012;6(8):1535–1543. doi: 10.1038/ismej.2012.4.</cite> [<a href="https://doi.org/10.1038/ismej.2012.4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3400402/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22343308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISME%20J&amp;title=Ruminococcus%20bromii%20is%20a%20keystone%20species%20for%20the%20degradation%20of%20resistant%20starch%20in%20the%20human%20colon&amp;author=X.%20Ze&amp;author=S.H.%20Duncan&amp;author=P.%20Louis&amp;author=H.J.%20Flint&amp;volume=6&amp;issue=8&amp;publication_year=2012&amp;pages=1535-1543&amp;pmid=22343308&amp;doi=10.1038/ismej.2012.4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0380">
<span class="label">76.</span><cite>Robertson M.D., Bickerton A.S., Dennis A.L., Vidal H., Frayn K.N. Insulin-sensitizing effects of dietary resistant starch and effects on skeletal muscle and adipose tissue metabolism. Am J Clin Nutr. 2005;82(3):559–567. doi: 10.1093/ajcn.82.3.559.</cite> [<a href="https://doi.org/10.1093/ajcn.82.3.559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16155268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am%20J%20Clin%20Nutr&amp;title=Insulin-sensitizing%20effects%20of%20dietary%20resistant%20starch%20and%20effects%20on%20skeletal%20muscle%20and%20adipose%20tissue%20metabolism&amp;author=M.D.%20Robertson&amp;author=A.S.%20Bickerton&amp;author=A.L.%20Dennis&amp;author=H.%20Vidal&amp;author=K.N.%20Frayn&amp;volume=82&amp;issue=3&amp;publication_year=2005&amp;pages=559-567&amp;pmid=16155268&amp;doi=10.1093/ajcn.82.3.559&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0385">
<span class="label">77.</span><cite>Robertson M.D., Currie J., Morgan L., Jewell D., Frayn K. Prior short-term consumption of resistant starch enhances postprandial insulin sensitivity in healthy subjects. Diabetologia. 2003;46(5):659–665. doi: 10.1007/s00125-003-1081-0.</cite> [<a href="https://doi.org/10.1007/s00125-003-1081-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12712245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=Prior%20short-term%20consumption%20of%20resistant%20starch%20enhances%20postprandial%20insulin%20sensitivity%20in%20healthy%20subjects&amp;author=M.D.%20Robertson&amp;author=J.%20Currie&amp;author=L.%20Morgan&amp;author=D.%20Jewell&amp;author=K.%20Frayn&amp;volume=46&amp;issue=5&amp;publication_year=2003&amp;pages=659-665&amp;pmid=12712245&amp;doi=10.1007/s00125-003-1081-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0390">
<span class="label">78.</span><cite>Bindels L.B., Munoz R.R.S., Gomes-Neto J.C., Mutemberezi V., Martínez I., Salazar N. Resistant starch can improve insulin sensitivity independently of the gut microbiota. Microbiome. 2017;5(1):12. doi: 10.1186/s40168-017-0230-5.</cite> [<a href="https://doi.org/10.1186/s40168-017-0230-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5294823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28166818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Resistant%20starch%20can%20improve%20insulin%20sensitivity%20independently%20of%20the%20gut%20microbiota&amp;author=L.B.%20Bindels&amp;author=R.R.S.%20Munoz&amp;author=J.C.%20Gomes-Neto&amp;author=V.%20Mutemberezi&amp;author=I.%20Mart%C3%ADnez&amp;volume=5&amp;issue=1&amp;publication_year=2017&amp;pages=12&amp;pmid=28166818&amp;doi=10.1186/s40168-017-0230-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0395">
<span class="label">79.</span><cite>Keenan M.J., Zhou J., Hegsted M., Pelkman C., Durham H.A., Coulon D.B. Role of resistant starch in improving gut health, adiposity, and insulin resistance. Adv Nutr. 2015;6(2):198–205. doi: 10.3945/an.114.007419.</cite> [<a href="https://doi.org/10.3945/an.114.007419" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4352178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25770258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv%20Nutr&amp;title=Role%20of%20resistant%20starch%20in%20improving%20gut%20health,%20adiposity,%20and%20insulin%20resistance&amp;author=M.J.%20Keenan&amp;author=J.%20Zhou&amp;author=M.%20Hegsted&amp;author=C.%20Pelkman&amp;author=H.A.%20Durham&amp;volume=6&amp;issue=2&amp;publication_year=2015&amp;pages=198-205&amp;pmid=25770258&amp;doi=10.3945/an.114.007419&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0400">
<span class="label">80.</span><cite>Birt D.F., Boylston T., Hendrich S., Jane J.-L., Hollis J., Li L. Resistant starch: promise for improving human health. Adv Nutr. 2013;4(6):587–601. doi: 10.3945/an.113.004325.</cite> [<a href="https://doi.org/10.3945/an.113.004325" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3823506/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24228189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv%20Nutr&amp;title=Resistant%20starch:%20promise%20for%20improving%20human%20health&amp;author=D.F.%20Birt&amp;author=T.%20Boylston&amp;author=S.%20Hendrich&amp;author=J.-L.%20Jane&amp;author=J.%20Hollis&amp;volume=4&amp;issue=6&amp;publication_year=2013&amp;pages=587-601&amp;pmid=24228189&amp;doi=10.3945/an.113.004325&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0405">
<span class="label">81.</span><cite>Yang X., Darko K.O., Huang Y., He C., Yang H., He S. Resistant starch regulates gut microbiota: structure, biochemistry and cell signalling. Cell Physiol Biochem. 2017;42(1):306–318. doi: 10.1159/000477386.</cite> [<a href="https://doi.org/10.1159/000477386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28535508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Physiol%20Biochem&amp;title=Resistant%20starch%20regulates%20gut%20microbiota:%20structure,%20biochemistry%20and%20cell%20signalling&amp;author=X.%20Yang&amp;author=K.O.%20Darko&amp;author=Y.%20Huang&amp;author=C.%20He&amp;author=H.%20Yang&amp;volume=42&amp;issue=1&amp;publication_year=2017&amp;pages=306-318&amp;pmid=28535508&amp;doi=10.1159/000477386&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0410">
<span class="label">82.</span><cite>Vrieze A., Van Nood E., Holleman F., Salojärvi J., Kootte R.S., Bartelsman J.F. Transfer of intestinal microbiota from lean donors increases insulin sensitivity in individuals with metabolic syndrome. Gastroenterology. 2012;143(4):913–936. doi: 10.1053/j.gastro.2012.06.031.</cite> [<a href="https://doi.org/10.1053/j.gastro.2012.06.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22728514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Transfer%20of%20intestinal%20microbiota%20from%20lean%20donors%20increases%20insulin%20sensitivity%20in%20individuals%20with%20metabolic%20syndrome&amp;author=A.%20Vrieze&amp;author=E.%20Van%20Nood&amp;author=F.%20Holleman&amp;author=J.%20Saloj%C3%A4rvi&amp;author=R.S.%20Kootte&amp;volume=143&amp;issue=4&amp;publication_year=2012&amp;pages=913-936&amp;pmid=22728514&amp;doi=10.1053/j.gastro.2012.06.031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0415">
<span class="label">83.</span><cite>Benítez-Páez A., del Pugar E.M.G., López-Almela I., Moya-Pérez Á., Codoñer-Franch P., Sanz Y. Depletion of Blautia species in the microbiota of obese children relates to intestinal inflammation and metabolic phenotype worsening. mSystems. 2020;5(2) doi: 10.1128/mSystems.00857-19.</cite> [<a href="https://doi.org/10.1128/mSystems.00857-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7093825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32209719/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mSystems&amp;title=Depletion%20of%20Blautia%20species%20in%20the%20microbiota%20of%20obese%20children%20relates%20to%20intestinal%20inflammation%20and%20metabolic%20phenotype%20worsening&amp;author=A.%20Ben%C3%ADtez-P%C3%A1ez&amp;author=E.M.G.%20del%20Pugar&amp;author=I.%20L%C3%B3pez-Almela&amp;author=%C3%81.%20Moya-P%C3%A9rez&amp;author=P.%20Codo%C3%B1er-Franch&amp;volume=5&amp;issue=2&amp;publication_year=2020&amp;pmid=32209719&amp;doi=10.1128/mSystems.00857-19&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0420">
<span class="label">84.</span><cite>de Groot P., Scheithauer T., Bakker G.J., Prodan A., Levin E., Khan M.T. Donor metabolic characteristics drive effects of faecal microbiota transplantation on recipient insulin sensitivity, energy expenditure and intestinal transit time. Gut. 2020;69(3):502–512. doi: 10.1136/gutjnl-2019-318320.</cite> [<a href="https://doi.org/10.1136/gutjnl-2019-318320" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7034343/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31147381/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Donor%20metabolic%20characteristics%20drive%20effects%20of%20faecal%20microbiota%20transplantation%20on%20recipient%20insulin%20sensitivity,%20energy%20expenditure%20and%20intestinal%20transit%20time&amp;author=P.%20de%20Groot&amp;author=T.%20Scheithauer&amp;author=G.J.%20Bakker&amp;author=A.%20Prodan&amp;author=E.%20Levin&amp;volume=69&amp;issue=3&amp;publication_year=2020&amp;pages=502-512&amp;pmid=31147381&amp;doi=10.1136/gutjnl-2019-318320&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0425">
<span class="label">85.</span><cite>Bodogai M., O’Connell J., Kim K., Kim Y., Moritoh K., Chen C. Commensal bacteria contribute to insulin resistance in aging by activating innate B1a cells. Sci Transl Med. 2018;10(467) doi: 10.1126/scitranslmed.aat4271.</cite> [<a href="https://doi.org/10.1126/scitranslmed.aat4271" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6445267/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30429354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&amp;title=Commensal%20bacteria%20contribute%20to%20insulin%20resistance%20in%20aging%20by%20activating%20innate%20B1a%20cells&amp;author=M.%20Bodogai&amp;author=J.%20O%E2%80%99Connell&amp;author=K.%20Kim&amp;author=Y.%20Kim&amp;author=K.%20Moritoh&amp;volume=10&amp;issue=467&amp;publication_year=2018&amp;pmid=30429354&amp;doi=10.1126/scitranslmed.aat4271&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0430">
<span class="label">86.</span><cite>Tindall B. The status of the name Lactobacillus rogosae Holdeman and Moore 1974. Opinion 88. Judicial Commission of the International Committee on Systematics of Prokaryotes. Int J Syst Evol Micr. 2014;64(10):3578-9.</cite> [<a href="https://doi.org/10.1099/ijs.0.069146-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25288658/" class="usa-link">PubMed</a>]</li>
<li id="b0435">
<span class="label">87.</span><cite>Stadlbauer V., Horvath A., Komarova I., Schmerboeck B., Feldbacher N., Klymiuk I. Dysbiosis in early sepsis can be modulated by a multispecies probiotic: a randomised controlled pilot trial. Beneficial Microbes. 2019;10(3):265–278. doi: 10.3920/BM2018.0067.</cite> [<a href="https://doi.org/10.3920/BM2018.0067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30694100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Beneficial%20Microbes&amp;title=Dysbiosis%20in%20early%20sepsis%20can%20be%20modulated%20by%20a%20multispecies%20probiotic:%20a%20randomised%20controlled%20pilot%20trial&amp;author=V.%20Stadlbauer&amp;author=A.%20Horvath&amp;author=I.%20Komarova&amp;author=B.%20Schmerboeck&amp;author=N.%20Feldbacher&amp;volume=10&amp;issue=3&amp;publication_year=2019&amp;pages=265-278&amp;pmid=30694100&amp;doi=10.3920/BM2018.0067&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0440">
<span class="label">88.</span><cite>Zhao J., Nian L., Kwok L., Sun T. Reduction in fecal microbiota diversity and short-chain fatty acid producers in methicillin-resistant Staphylococcus aureus infected individuals as revealed by PacBio single molecule, real-time sequencing technology. Eur J Clin Microbiol Infect Dis. 2017;36(8):1463–1472. doi: 10.1007/s10096-017-2955-2.</cite> [<a href="https://doi.org/10.1007/s10096-017-2955-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28455781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Microbiol%20Infect%20Dis&amp;title=Reduction%20in%20fecal%20microbiota%20diversity%20and%20short-chain%20fatty%20acid%20producers%20in%20methicillin-resistant%20Staphylococcus%20aureus%20infected%20individuals%20as%20revealed%20by%20PacBio%20single%20molecule,%20real-time%20sequencing%20technology&amp;author=J.%20Zhao&amp;author=L.%20Nian&amp;author=L.%20Kwok&amp;author=T.%20Sun&amp;volume=36&amp;issue=8&amp;publication_year=2017&amp;pages=1463-1472&amp;pmid=28455781&amp;doi=10.1007/s10096-017-2955-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0445">
<span class="label">89.</span><cite>Lax S., Smith D.P., Hampton-Marcell J., Owens S.M., Handley K.M., Scott N.M. Longitudinal analysis of microbial interaction between humans and the indoor environment. Science. 2014;345(6200):1048–1052. doi: 10.1126/science.1254529.</cite> [<a href="https://doi.org/10.1126/science.1254529" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4337996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25170151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Longitudinal%20analysis%20of%20microbial%20interaction%20between%20humans%20and%20the%20indoor%20environment&amp;author=S.%20Lax&amp;author=D.P.%20Smith&amp;author=J.%20Hampton-Marcell&amp;author=S.M.%20Owens&amp;author=K.M.%20Handley&amp;volume=345&amp;issue=6200&amp;publication_year=2014&amp;pages=1048-1052&amp;pmid=25170151&amp;doi=10.1126/science.1254529&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0450">
<span class="label">90.</span><cite>Lax S., Sangwan N., Smith D., Larsen P., Handley K.M., Richardson M. Bacterial colonization and succession in a newly opened hospital. Sci Transl Med. 2017;9(391) doi: 10.1126/scitranslmed.aah6500.</cite> [<a href="https://doi.org/10.1126/scitranslmed.aah6500" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5706123/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28539477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&amp;title=Bacterial%20colonization%20and%20succession%20in%20a%20newly%20opened%20hospital&amp;author=S.%20Lax&amp;author=N.%20Sangwan&amp;author=D.%20Smith&amp;author=P.%20Larsen&amp;author=K.M.%20Handley&amp;volume=9&amp;issue=391&amp;publication_year=2017&amp;pmid=28539477&amp;doi=10.1126/scitranslmed.aah6500&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0455">
<span class="label">91.</span><cite>Kawamura Y., Li Y., Liu H., Huang X., Li Z., Ezaki T. Bacterial population in Russian space station “Mir”. Microbiol Immunol. 2001;45(12):819–828. doi: 10.1111/j.1348-0421.2001.tb01321.x.</cite> [<a href="https://doi.org/10.1111/j.1348-0421.2001.tb01321.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11838899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol%20Immunol&amp;title=Bacterial%20population%20in%20Russian%20space%20station%20%E2%80%9CMir%E2%80%9D&amp;author=Y.%20Kawamura&amp;author=Y.%20Li&amp;author=H.%20Liu&amp;author=X.%20Huang&amp;author=Z.%20Li&amp;volume=45&amp;issue=12&amp;publication_year=2001&amp;pages=819-828&amp;pmid=11838899&amp;doi=10.1111/j.1348-0421.2001.tb01321.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0460">
<span class="label">92.</span><cite>Ichijo T, Yamaguchi N, Tanigaki F, Shirakawa M, Nasu M. Four-year bacterial monitoring in the International Space Station—Japanese Experiment Module “Kibo” with culture-independent approach. npj Microgravity. 2016;2:16007.</cite> [<a href="https://doi.org/10.1038/npjmgrav.2016.7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5515537/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28725725/" class="usa-link">PubMed</a>]</li>
<li id="b0465">
<span class="label">93.</span><cite>Venkateswaran K., Vaishampayan P., Cisneros J., Pierson D.L., Rogers S.O., Perry J. International Space Station environmental microbiome—microbial inventories of ISS filter debris. Appl Microbiol Biotechnol. 2014;98(14):6453–6466. doi: 10.1007/s00253-014-5650-6.</cite> [<a href="https://doi.org/10.1007/s00253-014-5650-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24695826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Microbiol%20Biotechnol&amp;title=International%20Space%20Station%20environmental%20microbiome%E2%80%94microbial%20inventories%20of%20ISS%20filter%20debris&amp;author=K.%20Venkateswaran&amp;author=P.%20Vaishampayan&amp;author=J.%20Cisneros&amp;author=D.L.%20Pierson&amp;author=S.O.%20Rogers&amp;volume=98&amp;issue=14&amp;publication_year=2014&amp;pages=6453-6466&amp;pmid=24695826&amp;doi=10.1007/s00253-014-5650-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0470">
<span class="label">94.</span><cite>Sielaff A.C., Urbaniak C., Mohan G.B.M., Stepanov V.G., Tran Q., Wood J.M. Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome. 2019;7(1):1–21. doi: 10.1186/s40168-019-0666-x.</cite> [<a href="https://doi.org/10.1186/s40168-019-0666-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6452512/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30955503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&amp;author=A.C.%20Sielaff&amp;author=C.%20Urbaniak&amp;author=G.B.M.%20Mohan&amp;author=V.G.%20Stepanov&amp;author=Q.%20Tran&amp;volume=7&amp;issue=1&amp;publication_year=2019&amp;pages=1-21&amp;pmid=30955503&amp;doi=10.1186/s40168-019-0666-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0475">
<span class="label">95.</span><cite>Bauza V., Madadi V., Ocharo R.M., Nguyen T.H., Guest J.S. Microbial source tracking using 16S rRNA amplicon sequencing identifies evidence of widespread contamination from young children’s feces in an urban slum of Nairobi, Kenya. Environ Sci Technol. 2019;53(14):8271–8281. doi: 10.1021/acs.est.8b06583.</cite> [<a href="https://doi.org/10.1021/acs.est.8b06583" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31268313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environ%20Sci%20Technol&amp;title=Microbial%20source%20tracking%20using%2016S%20rRNA%20amplicon%20sequencing%20identifies%20evidence%20of%20widespread%20contamination%20from%20young%20children%E2%80%99s%20feces%20in%20an%20urban%20slum%20of%20Nairobi,%20Kenya&amp;author=V.%20Bauza&amp;author=V.%20Madadi&amp;author=R.M.%20Ocharo&amp;author=T.H.%20Nguyen&amp;author=J.S.%20Guest&amp;volume=53&amp;issue=14&amp;publication_year=2019&amp;pages=8271-8281&amp;pmid=31268313&amp;doi=10.1021/acs.est.8b06583&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0480">
<span class="label">96.</span><cite>Knights D., Kuczynski J., Charlson E.S., Zaneveld J., Mozer M.C., Collman R.G. Bayesian community-wide culture-independent microbial source tracking. Nat Methods. 2011;8(9):761–763. doi: 10.1038/nmeth.1650.</cite> [<a href="https://doi.org/10.1038/nmeth.1650" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3791591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21765408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Bayesian%20community-wide%20culture-independent%20microbial%20source%20tracking&amp;author=D.%20Knights&amp;author=J.%20Kuczynski&amp;author=E.S.%20Charlson&amp;author=J.%20Zaneveld&amp;author=M.C.%20Mozer&amp;volume=8&amp;issue=9&amp;publication_year=2011&amp;pages=761-763&amp;pmid=21765408&amp;doi=10.1038/nmeth.1650&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0485">
<span class="label">97.</span><cite>Edgar R.C. UPARSE: highly accurate OTU sequences from microbial amplicon reads. Nat Methods. 2013;10:996. doi: 10.1038/nmeth.2604.</cite> [<a href="https://doi.org/10.1038/nmeth.2604" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23955772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=UPARSE:%20highly%20accurate%20OTU%20sequences%20from%20microbial%20amplicon%20reads&amp;author=R.C.%20Edgar&amp;volume=10&amp;publication_year=2013&amp;pages=996&amp;pmid=23955772&amp;doi=10.1038/nmeth.2604&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0490">
<span class="label">98.</span><cite>Callahan B.J., McMurdie P.J., Rosen M.J., Han A.W., Johnson A.J.A., Holmes S.P. DADA2: high-resolution sample inference from Illumina amplicon data. Nat Methods. 2016;13(7):581–583. doi: 10.1038/nmeth.3869.</cite> [<a href="https://doi.org/10.1038/nmeth.3869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4927377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27214047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=DADA2:%20high-resolution%20sample%20inference%20from%20Illumina%20amplicon%20data&amp;author=B.J.%20Callahan&amp;author=P.J.%20McMurdie&amp;author=M.J.%20Rosen&amp;author=A.W.%20Han&amp;author=A.J.A.%20Johnson&amp;volume=13&amp;issue=7&amp;publication_year=2016&amp;pages=581-583&amp;pmid=27214047&amp;doi=10.1038/nmeth.3869&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0495">
<span class="label">99.</span><cite>Bolyen E., Rideout J.R., Dillon M.R., Bokulich N.A., Abnet C.C., Al-Ghalith G.A. Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2. Nat Biotechnol. 2019;37(8):852–857. doi: 10.1038/s41587-019-0209-9.</cite> [<a href="https://doi.org/10.1038/s41587-019-0209-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7015180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31341288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Reproducible,%20interactive,%20scalable%20and%20extensible%20microbiome%20data%20science%20using%20QIIME%202&amp;author=E.%20Bolyen&amp;author=J.R.%20Rideout&amp;author=M.R.%20Dillon&amp;author=N.A.%20Bokulich&amp;author=C.C.%20Abnet&amp;volume=37&amp;issue=8&amp;publication_year=2019&amp;pages=852-857&amp;pmid=31341288&amp;doi=10.1038/s41587-019-0209-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="b0500">
<span class="label">100.</span><cite>Klindworth A., Pruesse E., Schweer T., Peplies J., Quast C., Horn M. Evaluation of general 16S ribosomal RNA gene PCR primers for classical and next-generation sequencing-based diversity studies. Nucleic Acids Res. 2013;41(1) doi: 10.1093/nar/gks808. e1-e.</cite> [<a href="https://doi.org/10.1093/nar/gks808" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3592464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22933715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Evaluation%20of%20general%2016S%20ribosomal%20RNA%20gene%20PCR%20primers%20for%20classical%20and%20next-generation%20sequencing-based%20diversity%20studies&amp;author=A.%20Klindworth&amp;author=E.%20Pruesse&amp;author=T.%20Schweer&amp;author=J.%20Peplies&amp;author=C.%20Quast&amp;volume=41&amp;issue=1&amp;publication_year=2013&amp;pmid=22933715&amp;doi=10.1093/nar/gks808&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Supplementary Data 1</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/8099722/bin/mmc1.xlsx" data-ga-action="click_feat_suppl" class="usa-link">mmc1.xlsx</a><sup> (1.6MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>ANCHOR code is available at <a href="https://github.com/gonzalezem/ANCHOR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/gonzalezem/ANCHOR</a>. ESV tables including relative abundance, annotation, count distribution, blast statistics, alternative database hits, and sequences are provided in <a href="#s0110" class="usa-link">Supplementary file 1</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Computational and Structural Biotechnology Journal are provided here courtesy of <strong>Research Network of Computational and Structural Biotechnology</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1016/j.csbj.2021.03.040"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/main.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/8099722/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/8099722/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC8099722%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC8099722/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC8099722/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC8099722/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/33995915/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC8099722/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/33995915/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC8099722/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/8099722/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="qV4V6CoqSWI6bnVU77vZO5fC99WwBUrDZqPFehLCXd2d8g2iHfGeNCJH3V21Lcsj">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
